\newpage
\chapter{Appendix}
\appendix
\section{Mutagenesis Primers}\label{App:Primers}

\textbf{\atpdx ~D41N Forward}\\
5'-GGTGGTGTTATCATGAATGTCGTCAACGCCGAGC-3'\\
\\
\textbf{\atpdx ~D41N Reverse}\\
5'-GCTCGGCGTTGACGACATTCATGATAACACCACC-3'\\   
\\
\textbf{\atpdx ~K98A Forward}\\ 
5'-CGATTCCGGTGATGGCTGCAGCTAGGATTGGTCATTTC-3'\\
\\
\textbf{\atpdx ~K98A Reverse}\\ 
5'-GAAATGACCAATCCTAGCTGCAGCCATCACCGGAATCG-3'\\
\\
\textbf{\atpdx ~S121A Forward}\\
5'-GGAATCGATTACATCGATGAGGCCGAGGTTTTGACTC-3'\\ 
\\
\textbf{\atpdx ~S121A Reverse}\\
5'-GAGTCAAAACCTCGGCCTCATCGATGTAATCGATTCC-3'\\ 
\\
\textbf{\atpdx ~H132N Forward}\\
5'-TTTGACTCTTGCTGATGAAGATCATAACATCAACAAGCATAATTTCC-3'\\
\\
\textbf{\atpdx ~H132N Reverse}\\
5'-GGAAATTATGCTTGTTGATGTTATGATCTTCATCAGCAAGAGTCAAA-3'\\
\\
\textbf{\atpdx ~K166R Forward}\\
5'-GATGATTAGGACCAGAGGTGAAGCTGGAAC-3'\\
\\
\textbf{\atpdx ~K166R Reverse}\\
5'-GTTCCAGCTTCACCTCTGGTCCTAATCATC-3'\\
%\newpage
%\section{Residue numbering for Pdx1 Orthologs}\label{App:Primers}
%\begin{table}[!htbp]
%  \centering
%\begin{tabular}{ |P{2.1cm}|P{2.1cm}|P{2.2cm}|P{2.1cm}|P{2.2cm}|P{2.3cm}|}
%\hline
%Arabidopsis&Bacillus&Thermatoga&Geobacillus&Plasmodium&Saccharomyces\\
%Pdx1.3&YaaD (PDB:2NV1)&PdxS (PDB:2ISS)&PdxT (PDB:1ZNN)&Pdx1 (PDB:4ADU)&Pdx1.1 (PDB:3FEM)\\
%\hline
%Asp41&Asp24&&&&\\
%Lys98&Lys81&&&&\\
%Asp119&Asp102&&&&\\
%Ser121&Ser104&&&&\\
%Glu122&Glu105&&&&\\
%Met162&Met145&&&&\\
%Arg164&Arg147&&&&\\
%Thr165&&&&&\\
%Lys166&&&&&\\
%Ser253&&&&&\\
%\hline
%\end{tabular}
%  \caption[Numbering of equivalent residues in Pdx1 orthologs]{Numbering of equivalent residues that have been assigned as participating in catalysis in Pdx1 orthologs for which there are published structures.}
%\end{table}
\newpage
\section{Doses absorbed by each lysozyme crystal} \label{App:lys_xtal_dimensions}

\begin{table}[!htbp]
  \centering
\begin{tabular}{ |P{2cm}||P{3.5cm}|P{2.5cm}|P{3cm}|}
\hline
Crystal Name&Crystal Dimensions (\si{\micro\meter})&Datasets Collected&Dose per Dataset (\si{\kilo\gray})\\
\hline
Lys\_2 &300 300 200&2&460\\
Lys\_6 &100 60  60 &1&1153\\
Lys\_10 &450 220 130&8&569\\
Lys\_11 &150 100 80 &2&930\\
Lys\_12 &150 100 80&3&930\\
\hline
\end{tabular}
  \caption[Dose Calculations for Lysozyme Crystals used in Multi-Crystal Analysis]{Dose Calculations for lysozyme crystals used in multi-crystal analysis. 16 datasets were collected from 5 crystals, using the same data collection protocol. The diffraction weighted dose for each dataset was calculated using RADDOSE-3D (see Table \ref{table:RADINP_320} for input parameters). The average DWD for all lysozyme datasets was calculated to be 705 \si{\kilo\gray} by adding the doses for each datasets and dividing by the number of datasets. The average DWD for all datasets was divided by the number of images and multiplied by 100 to find the dose for each 100 image sweep to be 19.6 kGy.}
\end{table}
\clearpage
\section{Crystallographic Statistics for Multi-Crystal Lysozyme Data} \label{App:lys_stats}
\begin{landscape}
\begin{table}[!htbp]
 \centering
\begin{tabular}{ |p{4cm}||p{3.5cm}|p{3.5cm}|p{3.5cm}|p{3.5cm}|}
 \hline
 \multicolumn{5}{|c|}{Crystallographic Statistics for Lysozyme Composite Datasets 1-4} \\
 \hline
 \multicolumn{1}{|l|}{Sweep Number (Dose)} &Sweep 1 (20 kGy)&Sweep 2 (40 kGy)&Sweep 3 (60 kGy)&Sweep 4 (80 kGy)\\%&Sweep 5 (65 kGy)&Sweep 6 (78 kGy)&Sweep 7 (91 kGy)&Sweep 8 (104 kGy)&Sweep 9 (117 kGy)&Sweep 10 (130 kGy)&Sweep 11 (143 kGy)&Sweep 12 (156 kGy)&Sweep 13 (169 kGy)&Sweep 14 (182 kGy)&Sweep 15 (195 kGy)&Sweep 16 (208 kGy)&Sweep 17 (221 kGy)&Sweep 18 (234 kGy)&Sweep 19 (247 kGy)&Sweep 20 (260 kGy)&Sweep 21 (273 kGy)&Sweep 22 (286 kGy)&Sweep 23 (299 kGy)&Sweep 24 (312 kGy)&Sweep 25 (325 kGy)&Sweep 26 (338 kGy)&Sweep 27 (351 kGy)&Sweep 28 (364 kGy)&Sweep 29 (377 kGy)&Sweep 30 (390 kGy)&Sweep 31 (403 kGy)&Sweep 32 (416 kGy)&Sweep 33 (429 kGy)&Sweep 34 (442 kGy)&Sweep 35 (455 kGy)&Sweep 36 (468 kGy)
 \hline
 Data Collection   &I04 (DLS) & I04 (DLS) & I04 (DLS) &I04 (DLS) \\
 Space group &P 4$_3$ 2$_1$ 2&P 4$_3$ 2$_1$ 2&P 4$_3$ 2$_1$ 2&P 4$_3$ 2$_1$ 2\\
 Unit cell (\textit{a, b, c}) &77.4, 77.4, 37.5&77.3, 77.3, 37.4&77.4, 77.4, 37.5&77.4, 77.4, 37.5\\
 Resolution    &38.65-1.50(1.53-1.50)&37.43-1.50(1.53-1.50)&37.47-1.50(1.53-1.50)&38.71-1.50(1.53-1.50)\\
 R\textsubscript{merge}&15.4 (81.9)&31.1 (329.6)&36.1 (512.4)&13.4 (80.3)\\
 CC\sfrac{1}{2}&0.993 (0.753)&0.991 (0.808)&0.963 (0.009)&0.993 (0.966)\\
 \sfrac{I}{$\sigma$(I)}&19.3 (10.4)&19.9 (12.2)&21.4 (12.7)&13.4 (6.6)\\
 Completeness (\%)   &99.9 (100.0)&99.9 (100.0)&99.8 (99.6)&99.6 (99.8)\\
 Multiplicity    &9.6 (9.9)&11.0 (11.2)&10.3 (10.6)&9.1 (9.3)\\
 Unique Reflections    &18683 (895)&18722 (892)&18755 (891)&18660 (898)\\
 Wilson B-factor    &9.6&9.8&10.0&9.6\\
 \hline 
 \hline 
 \multicolumn{5}{|c|}{Crystallographic Statistics for Lysozyme Composite Datasets 5-8} \\
 \hline
 \multicolumn{1}{|l|}{Sweep Number (Dose)} &Sweep 5 (100 kGy)&Sweep 6 (120 kGy)&Sweep 7 (140 kGy)&Sweep 8 (160 kGy)\\
 \hline
 Data Collection   &I04 (DLS) & I04 (DLS) & I04 (DLS) &I04 (DLS) \\
 Space group &P 4$_3$ 2$_1$ 2&P 4$_3$ 2$_1$ 2&P 4$_3$ 2$_1$ 2&P 4$_3$ 2$_1$ 2\\
 Unit cell (\textit{a, b, c}) &77.4, 77.4, 37.5&77.4, 77.4, 37.4&77.4, 77.4, 37.4&77.4, 77.4, 37.4\\
 Resolution    &37.46-1.50(1.53-1.50)&37.43-1.50(1.53-1.50)&38.69-1.50(1.53-1.50)&38.70-1.50(1.53-1.50)\\
 R\textsubscript{merge}&38.7 (481.8)&14.6 (79.6)&14.5 (78.8)&15.8 (94.1)\\
 CC\sfrac{1}{2}&0.970 (0.472)&0.994 (0.900)&0.995 (0.961)&0.994 (0.962)\\
 \sfrac{I}{$\sigma$(I)}&24.3 (13.2)&19.4 (12.2)&20.1 (12.9)&17.4 (10.2)\\
 Completeness (\%)   &99.9 (99.8)&99.8 (99.6)&99.8 (99.8)&99.8 (100.0)\\
 Multiplicity    &10.2 (10.4)&9.0 (9.4)&8.9 (9.2)&8.3 (8.5)\\
 Unique Reflections    &18794 (897)&18723 (892)&18715 (890)&18702 (892)\\
 Wilson B-factor    &9.4&9.8&9.7&9.8\\
 \hline
\end{tabular}
\caption[Crystallographic Statistics for Composite Lysozyme Datasets 1-8]{Table of crystallographic statistics for composite lysozyme datasets 1-8.} 
\end{table}
\end{landscape}

\clearpage

\begin{landscape}
\begin{table}[p]
 \centering
\begin{tabular}{ |p{4cm}||p{3.5cm}|p{3.5cm}|p{3.5cm}|p{3.5cm}|}
\hline
 \multicolumn{5}{|c|}{Crystallographic Statistics for Lysozyme Composite Datasets 9-12} \\
 \hline
 \multicolumn{1}{|l|}{Sweep Number (Dose)} &Sweep 9 (180 kGy)&Sweep 10 (200 kGy)&Sweep 11 (220 kGy)&Sweep 12 (240 kGy)\\
 \hline
 Data Collection   &I04 (DLS) & I04 (DLS) & I04 (DLS) &I04 (DLS) \\
 Space group &P 4$_3$ 2$_1$ 2&P 4$_3$ 2$_1$ 2&P 4$_3$ 2$_1$ 2&P 4$_3$ 2$_1$ 2\\
 Unit cell (\textit{a, b, c}) &77.5, 77.5, 37.5&77.4, 77.4, 37.4&77.3, 77.3, 37.4&77.4, 77.4, 37.4 \\
 Resolution    &37.46-1.50(1.53-1.50)&38.68-1.50(1.53-1.50)&38.66-1.50(1.53-1.50)&38.69-1.50(1.53-1.50)\\
 R\textsubscript{merge}&32.2 (391.5)&10.5 (25.0)&24.1 (173.4)&19.7 (139.0)\\
 CC\sfrac{1}{2}&0.990 (0.641)&0.994 (0.970)&0.988 (0.924)&0.993 (0.937)\\
 \sfrac{I}{$\sigma$(I)}&18.2 (9.5)&21.5 (9.9)&9.0 (4.4)&11.5 (5.6)\\
 Completeness (\%)   &98.7 (99.6)&99.7 (99.8)&99.9 (100.0)&99.0 (100.0)\\
 Multiplicity    &8.9 (9.1)&7.5 (7.9)&10.3 (10.5)&10.5 (10.7)\\
 Unique Reflections    &18776 (902)&18649 (889)&18732 (885)&18532 (892)\\
 Wilson B-factor    &9.5&9.7&9.8&9.8\\
 \hline 
 \hline 
 \multicolumn{5}{|c|}{Crystallographic Statistics for Lysozyme Composite Datasets 13-16} \\
 \hline
 \multicolumn{1}{|l|}{Sweep Number (Dose)} &Sweep 13 (260 kGy)&Sweep 14 (280 kGy)&Sweep 15 (300 kGy)&Sweep 16 (320 kGy)\\
 \hline
 Data Collection   &I04 (DLS) & I04 (DLS) & I04 (DLS) &I04 (DLS) \\
 Space group &P 4$_3$ 2$_1$ 2&P 4$_3$ 2$_1$ 2&P 4$_3$ 2$_1$ 2&P 4$_3$ 2$_1$ 2\\
 Unit cell (\textit{a, b, c}) &77.3, 77.3, 37.4&77.3, 77.3, 37.4&77.3, 77.3, 37.5&77.4, 77.4, 37.5\\
 Resolution    &38.66-1.50(1.53-1.50)&38.67-1.50(1.53-1.50)&37.49-1.50(1.53-1.50)&38.70-1.50(1.53-1.50)\\
 R\textsubscript{merge}&21.6 (157.0)&23.9 (198.8)&27.6 (267.7)&40.0 (460.9)\\
 CC\sfrac{1}{2}&0.991 (0.230)&0.990 (0.860)&0.979 (0.236)&0.956 (0.023)\\
 \sfrac{I}{$\sigma$(I)}&19.1 (10.1)&19.1 (10.4)&9.9 (3.9)&19.7 (10.2)\\
 Completeness (\%)   &99.9 (99.9)&99.9 (100.0)&99.7 (99.9)&99.9 (99.9)\\
 Multiplicity    &10.2 (10.6)&10.1 (10.5)&10.3 (10.5)&9.4 (9.6)\\
 Unique Reflections    &18720 (890)&18747 (902)&18684 (893)&18786 (897)\\
 Wilson B-factor    &10.0&9.8&10.1&10.1\\
 \hline 
\end{tabular}
\caption[Crystallographic Statistics for Composite Lysozyme Datasets 9-16]{Table of crystallographic statistics for composite lysozyme datasets 9-16.} 
\end{table}
\end{landscape}
\clearpage

\begin{landscape}
\begin{table}[p]
 \centering
\begin{tabular}{ |p{4cm}||p{3.5cm}|p{3.5cm}|p{3.5cm}|p{3.5cm}|}
\hline
 \multicolumn{5}{|c|}{Crystallographic Statistics for Lysozyme Composite Datasets 17-20} \\
 \hline
 \multicolumn{1}{|l|}{Sweep Number (Dose)} &Sweep 17 (340 kGy)&Sweep 18 (360 kGy)&Sweep 19 (380 kGy)&Sweep 20 (400 kGy)\\
 \hline
 Data Collection   &I04 (DLS) & I04 (DLS) & I04 (DLS) &I04 (DLS) \\
 Space group &P 4$_3$ 2$_1$ 2&P 4$_3$ 2$_1$ 2&P 4$_3$ 2$_1$ 2&P 4$_3$ 2$_1$ 2\\
 Unit cell (\textit{a, b, c}) &77.4, 77.4, 37.5&77.4, 77.4, 37.4&77.4, 77.4, 37.4&77.4, 77.4, 37.5 \\
 Resolution    &37.46-1.50(1.53-1.50)&37.42-1.50(1.53-1.50)&38.67-1.50(1.53-1.50)&37.46-1.50(1.53-1.50)\\
 R\textsubscript{merge}&43.4 (521.6)&14.6 (51.0)&34.7 (329.0)&32.0 (355.4)\\
 CC\sfrac{1}{2}&0.978 (0.901)&0.992 (0.925)&0.986 (0.899)&0.990 (0.074)\\
 \sfrac{I}{$\sigma$(I)}&20.7 (11.7)&18.8 (11.0)&19.2 (11.3)&20.3 (11.7)\\
 Completeness (\%)   &99.9 (100.0)&99.7 (100.0)&99.9 (100.0)&99.9 (100.0)\\
 Multiplicity    &10.0 (10.5)&8.9 (9.1)&10.1 (10.2)&9.6 (9.7)\\
 Unique Reflections    &18795 (897)&18653 (895)&18716 (898)&18782 (897)\\
 Wilson B-factor    &9.5&9.9&9.8&10.2\\
 \hline 
 \hline 
 \multicolumn{5}{|c|}{Crystallographic Statistics for Lysozyme Composite Datasets 21-24} \\
 \hline
 \multicolumn{1}{|l|}{Sweep Number (Dose)} &Sweep 21 (420 kGy)&Sweep 22 (440 kGy)&Sweep 23 (460 kGy)&Sweep 24 (480 kGy)\\
 \hline
 Data Collection   &I04 (DLS) & I04 (DLS) & I04 (DLS) &I04 (DLS) \\
 Space group &P 4$_3$ 2$_1$ 2&P 4$_3$ 2$_1$ 2&P 4$_3$ 2$_1$ 2&P 4$_3$ 2$_1$ 2\\
 Unit cell (\textit{a, b, c}) &77.4, 77.4, 37.5&77.5, 77.5, 37.5&77.5, 77.5, 37.5&77.4, 77.4, 37.4\\
 Resolution    &37.45-1.50(1.53-1.50)&38.73-1.50(1.53-1.50)&37.45-1.50(1.53-1.50)&37.44-1.50(1.53-1.50)\\
 R\textsubscript{merge}&15.0 (83.1)&16.2 (109.9)&17.2 (124.4)&17.9 (129.6)\\
 CC\sfrac{1}{2}&0.994 (0.882)&0.994 (0.777)&0.995 (0.810)&0.994 (0.741)\\
 \sfrac{I}{$\sigma$(I)}&21.1 (11.9)&21.4 (11.1)&21.1 (11.1)&22.1 (11.1)\\
 Completeness (\%)   &99.8 (99.5)&99.6 (99.9)&99.9 (99.6)&99.7 (99.3)\\
 Multiplicity    &9.6 (10.1)&9.7 (9.8)&9.6 (9.8)&9.6 (10.1)\\
 Unique Reflections    &18726 (891)&18691 (901)&18793 (901)&18731 (889)\\
 Wilson B-factor    &10.4&10.4&10.3&10.6\\
 \hline 
\end{tabular}
\caption[Crystallographic Statistics for Composite Lysozyme Datasets 17-24]{Table of crystallographic statistics for composite lysozyme datasets 17-24.} 
\end{table}
\end{landscape}
\clearpage

\begin{landscape}
\begin{table}[p]
 \centering
\begin{tabular}{ |p{4cm}||p{3.5cm}|p{3.5cm}|p{3.5cm}|p{3.5cm}|}
\hline
 \multicolumn{5}{|c|}{Crystallographic Statistics for Lysozyme Composite Datasets 25-28} \\
 \hline
 \multicolumn{1}{|l|}{Sweep Number (Dose)} &Sweep 25 (500 kGy)&Sweep 26 (520 kGy)&Sweep 27 (540 kGy)&Sweep 28 (560 kGy)\\
 \hline
 Data Collection   &I04 (DLS) & I04 (DLS) & I04 (DLS) &I04 (DLS) \\
 Space group &P 4$_3$ 2$_1$ 2&P 4$_3$ 2$_1$ 2&P 4$_3$ 2$_1$ 2&P 4$_3$ 2$_1$ 2\\
 Unit cell (\textit{a, b, c}) &77.4, 77.4, 37.4&77.5, 77.5, 37.5&77.5, 77.5, 37.5&77.5, 77.5, 37.5 \\
 Resolution    &38.72-1.50(1.53-1.50)&38.75-1.50(1.53-1.50)&37.46-1.50(1.53-1.50)&38.74-1.50(1.53-1.50)\\
 R\textsubscript{merge}&18.8 (146.6)&24.2 (215.8)&21.2 (165.9)&19.3 (139.2)\\
 CC\sfrac{1}{2}&0.993 (0.392)&0.991 (0.324)&0.993 (0.916)&0.994 (0.877)\\
 \sfrac{I}{$\sigma$(I)}&23.4 (11.7)&20.9 (11.5)&17.8 (9.8)&17.3 (8.8)\\
 Completeness (\%)   &99.8 (99.6)&99.9 (100.0)&99.8 (100.0)&100.0 (100.0)\\
 Multiplicity    &9.5 (9.9)&10.2 (10.4)&10.3 (10.6)&10.3 (10.6)\\
 Unique Reflections    &18759 (899)&18837 (910)&18786 (901)& 18828 (904)\\
 Wilson B-factor    &10.5&10.5&10.3&10.5\\
 \hline 
 \hline 
 \multicolumn{5}{|c|}{Crystallographic Statistics for Lysozyme Composite Datasets 29-32} \\
 \hline
 \multicolumn{1}{|l|}{Sweep Number (Dose)} &Sweep 29 (580 kGy)&Sweep 30 (600 kGy)&Sweep 31 (620 kGy)&Sweep 32 (640 kGy)\\
 \hline
 Data Collection   &I04 (DLS) & I04 (DLS) & I04 (DLS) &I04 (DLS) \\
 Space group &P 4$_3$ 2$_1$ 2&P 4$_3$ 2$_1$ 2&P 4$_3$ 2$_1$ 2&P 4$_3$ 2$_1$ 2\\
 Unit cell (\textit{a, b, c}) &77.5, 77.5, 37.5&77.3, 77.3, 37.4&77.3, 77.3, 37.5&77.4, 77.4, 37.5\\
 Resolution    &38.74-1.50(1.53-1.50)&38.67-1.50(1.53-1.50)&38.67-1.50(1.53-1.50)&38.69-1.50(1.53-1.50)\\
 R\textsubscript{merge}&19.3 (153.6)&21.0 (156.0)&23.4 (199.2)&31.9 (349.6)\\
 CC\sfrac{1}{2}&0.993 (0.905)&0.992 (0.529)&0.992 (0.503)&0.970 (0.197)\\
 \sfrac{I}{$\sigma$(I)}&15.6 (7.5)&9.7 (4.5)&16.6 (8.2)&18.5 (8.1)\\
 Completeness (\%)   &100.0 (100.0)&99.0 (99.8)&99.9 (99.8)&99.9 (100.0)\\
 Multiplicity    &9.6 (10.0)&10.5 (10.8)&10.1 (10.5)&10.0 (10.4)\\
 Unique Reflections    &18846 (906)&18491 (888)&18751 (897)&18774 (894)\\
 Wilson B-factor    &10.0&10.5&10.1&10.1\\
 \hline 
\end{tabular}
\caption[Crystallographic Statistics for Composite Lysozyme Datasets 25-32]{Table of crystallographic statistics for composite lysozyme datasets 25-32.} 
\end{table}
\end{landscape}
\clearpage



\begin{landscape}
\begin{table}[p]
 \centering
\begin{tabular}{ |p{4cm}||p{3.5cm}|p{3.5cm}|p{3.5cm}|p{3.5cm}|}
\hline
 \multicolumn{5}{|c|}{Crystallographic Statistics for Lysozyme Composite Datasets 33-36} \\
 \hline
 \multicolumn{1}{|l|}{Sweep Number (Dose)} &Sweep 33 (660 kGy)&Sweep 34 (680 kGy)&Sweep 35 (700 kGy)&Sweep 36 (720 kGy)\\
 \hline
 Data Collection   &I04 (DLS) & I04 (DLS) & I04 (DLS) &I04 (DLS) \\
 Space group &P 4$_3$ 2$_1$ 2&P 4$_3$ 2$_1$ 2&P 4$_3$ 2$_1$ 2&P 4$_3$ 2$_1$ 2\\
 Unit cell (\textit{a, b, c}) &77.3, 77.3, 37.4&77.4, 77.4, 37.4&77.4, 77.4, 37.5&77.4, 77.4, 37.5\\
 Resolution    &37.44-1.50(1.53-1.50)&38.72-1.50(1.53-1.50)&37.49-1.50(1.53-1.50)&37.47-1.50(1.53-1.50)\\
 R\textsubscript{merge}&39.5 (480.1)&15.4 (85.0)&39.1 (509.1)&36.3 (410.0)\\
 CC\sfrac{1}{2}&0.934 (0.017)&0.993 (0.903)&0.985 (0.772)&0.988 (0.325)\\
 \sfrac{I}{$\sigma$(I)}&22.3 (9.0)&18.7 (9.3)&18.7 (9.9)&19.2 (9.4)\\
 Completeness (\%)   &99.7 (100.0)&99.6 (99.5)&99.9 (100.0)&99.9 (99.9)\\
 Multiplicity    &10.0 (10.4)&8.1 (8.4)&10.2 (10.6)&10.2 (10.4)\\
 Unique Reflections    &18661 (895)&18709 (895)&18792 (892)&18784 (895)\\
 Wilson B-factor    &10.1&10.9&10.3&10.1\\
 \hline 
\end{tabular}
\caption[Crystallographic Statistics for Composite Lysozyme Datasets 33-36]{Table of crystallographic statistics for composite lysozyme datasets 33-36.} 
\end{table}
\end{landscape}
\newpage

\section{Isomorphous Difference Density Maps for Multi-Crystal Lysozyme Data}\label{App:Lys_fofo}
\subsection*{Fo-Fo Maps for Multi-Crystal Lysozyme Data Bond 1}


\begin{minipage}{\linewidth}
	\makebox[\linewidth]{
		
	\includegraphics[width=16cm, height=16cm, keepaspectratio]{/Users/matt/Dropbox/ThesisPrep/Thesis/fig/lysozyme/lysozyme_structure/LysozymeBond_1fofo_2}}	

	\captionof{figure}[Lysozyme Cys 6 - Cys 127 Disulphide Bond Fo$_n$-Fo$_1$ Maps]{Isomorphous Difference Density Maps at 20 kGy intervals for the Cys 6 (right) - Cys 127 (left) disulphide bond. Red peaks indicate loss of electrons, green peaks indicate areas with increased electron density, contoured at 3.0 $\sigma$ and -3.0 $\sigma$ respectively.}	
\end{minipage}

\subsection*{Fo-Fo Maps for Multi-Crystal Lysozyme Data Bond 2}


\begin{minipage}{\linewidth}
	\makebox[\linewidth]{
		
	\includegraphics[width=16cm, height=16cm, keepaspectratio]{/Users/matt/Dropbox/ThesisPrep/Thesis/fig/lysozyme/lysozyme_structure/LysozymeBond_2fofo_2}}	

	\captionof{figure}[Lysozyme Cys 30 - Cys 115 Disulphide Bond Fo$_n$-Fo$_1$ Maps]{Isomorphous Difference Density Maps at 20 kGy intervals for the Cys 30 (left) - Cys 115 (right) disulphide bond. Red peaks indicate loss of electrons, green peaks indicate areas with increased electron density, contoured at 3.0 $\sigma$ and -3.0 $\sigma$ respectively.}	
\end{minipage}

\subsection*{Fo-Fo Maps for Multi-Crystal Lysozyme Data Bond 3}


\begin{minipage}{\linewidth}
	\makebox[\linewidth]{
		
	\includegraphics[width=16cm, height=16cm, keepaspectratio]{/Users/matt/Dropbox/ThesisPrep/Thesis/fig/lysozyme/lysozyme_structure/LysozymeBond_3fofo_2}}	

	\captionof{figure}[Lysozyme Cys 64 - Cys 80 Disulphide Bond Fo$_n$-Fo$_1$ Maps]{Isomorphous Difference Density Maps at 20 kGy intervals for the Cys 64 (left) - Cys 80 (right) disulphide bond. Red peaks indicate loss of electrons, green peaks indicate areas with increased electron density, contoured at 3.0 $\sigma$ and -3.0 $\sigma$ respectively.}	
\end{minipage} 

\subsection*{Fo-Fo Maps for Multi-Crystal Lysozyme Data Bond 4}


\begin{minipage}{\linewidth}
	\makebox[\linewidth]{
		
	\includegraphics[width=16cm, height=16cm, keepaspectratio]{/Users/matt/Dropbox/ThesisPrep/Thesis/fig/lysozyme/lysozyme_structure/LysozymeBond_4fofo_2}}	

	\captionof{figure}[Lysozyme Cys 76 - Cys 94 Disulphide Bond Fo$_n$-Fo$_1$ Maps]{Isomorphous Difference Density Maps at 20 kGy intervals for the Cys 76 (left) - Cys 94 (right) disulphide bond. Red peaks indicate loss of electrons, green peaks indicate areas with increased electron density, contoured at 3.0 $\sigma$ and -3.0 $\sigma$ respectively.}	
\end{minipage}
\clearpage
\section{Crystallisation conditions for \atpdx -I320 experiment} 
\begin{table}[!hbp]
 \centering
%\begin{tabular}{|p{6cm}|p{4cm}|}
\begin{tabular}{|l|p{8cm}|}
%\begin{tabular}{|l|P{23mm}|p{4cm}|}
 %\hline
 %\multicolumn{2}{|c|}{\textbf{}} \\
 %\hline
 %\multicolumn{2}{|c|}{Crystal Parameters}\\
 \hline
 %Crystal Name(s) &Protein:Buffer Mix&Crystallisation Conditions\\
 Crystal Name(s)&Crystallisation Conditions\\
 \hline
 %Ma92-1, Ma93-4&1 \si{\micro\litre} : 1 \si{\micro\litre}&100 \mM Tris pH 7.8, 350 \mM Sodium Acetate pH 5.5, 5\% PEG 4000 (w/v))\\
 Ma92-1, Ma93-4&100 \mM Tris pH 7.8, 350 \mM  Sodium Acetate pH 5.5, 5\% PEG 4000 (w/v))\\
 Ma99-4&100 \mM Tris pH 7.8, 300 \mM  Sodium Acetate pH 5.5, 10.2\% PEG 4000 (w/v))\\
 Ma101-5&100 \mM Tris pH 7.8, 200 \mM  Sodium Acetate pH 5.5, 15\% PEG 4000 (w/v))\\
 Ma102-1, Ma102-3&100 \mM Tris pH 7.8, 200 \mM  Sodium Acetate pH 5.5, 8.6\% PEG 4000 (w/v))\\
 Ma103-3, Ma103-5, Ma104-1&100 \mM Tris pH 7.8, 200 \mM  Sodium Acetate pH 5.5, 11.8\% PEG 4000 (w/v))\\
 Ma104-4, Ma104-5, Ma105-1&100 \mM Tris pH 7.8, 400 \mM  Sodium Acetate pH 5.5, 11.8\% PEG 4000 (w/v))\\
 Ma105-2, Ma105-4, Ma105-5&100 \mM Tris pH 7.8, 400 \mM  Sodium Acetate pH 5.5, 10.2\% PEG 4000 (w/v))\\
 Ma106-2, Ma106-3&100 \mM Tris pH 7.8, 400 \mM  Sodium Acetate pH 5.5, 7\% PEG 4000 (w/v))\\
 Ma109-3, Ma109-4&100 \mM Tris pH 7.8, 200 \mM  Sodium Acetate pH 5.5, 8.6\% PEG 4000 (w/v))\\
 Ma109-5&100 \mM Tris pH 7.8, 200 \mM  Sodium Acetate pH 5.5, 10.2\% PEG 4000 (w/v))\\
 \hline 
\end{tabular}
\caption[Crystallisation conditions for crystals used in multi-crystal \atpdx -I320 experiment]{Crystallisation conditions for crystals used in multi-crystal \atpdx -I320 experiment. With the exception of Ma92-1 and Ma93-4 all crystals were grown in 96 well sitting drop crystallisation plates with a 1 \si{\micro\litre} : 0.5 \si{\micro\litre} ratio of protein to buffer, protein concentration 22.5 \si{\milli\gram\per\milli\litre}. Ma92-1 and Ma93-4 were grown in a 24 well hanging drop vapour diffusion plate, with a 1 \si{\micro\litre} : 1 \si{\micro\litre} ratio of protein to buffer mix, protein concentration 55 \si{\milli\gram\per\milli\litre}.}\label{table:I320multi-crystallisation} 
\end{table} 
\clearpage
\section{Calculated Doses for \atpdx -I320 Datasets}\label{App:I320Dose}  
\begin{table}[!htbp]
  \centering
\begin{tabular}{ |P{2cm}||P{3.5cm}|P{2.5cm}|P{2.5cm}|P{2.5cm}|}
\hline
Dataset Name&Crystal Dimensions (\si{\micro\meter})&Flux ~~(photons \si{\per\second})&Number of Images&Dose (\si{\mega\gray})\\
\hline
Ma92-1{\_}1&80 20 20&8.0 x 10$^{10}$&450&2.18\\%145.3
Ma92-1{\_}2&80 20 20&8.0 x 10$^{10}$&300&1.62\\%162.0
Ma92-2{\_}1&100 50 50&9.6 x 10$^{10}$&300&1.73\\%173
Ma92-2{\_}2&100 50 50&9.6 x 10$^{10}$&300&1.73\\%173
Ma92-2{\_}3&100 50 50&9.6 x 10$^{10}$&300&1.73\\%173
Ma92-3{\_}1&80 80 40&1.4 x 10$^{11}$&300&2.65\\%265
Ma93-4{\_}1&250 100 30&8.2 x 10$^{10}$&450&1.71\\%114
Ma99-4{\_}1&60 60 30&4.0 x 10$^{10}$&450&1.13\\%75.33
Ma101-5{\_}1&80 50 30&9.0 x 10$^{10}$&450&2.27\\%151.3
Ma102-1{\_}1&60 60 30&9.0 x 10$^{10}$&450&2.55\\%170
Ma102-3{\_}1&50 50 40&3.2 x 10$^{10}$&450&0.95\\%63.3
Ma102-3{\_}2&50 50 40&3.2 x 10$^{10}$&450&0.95\\%63.3
Ma103-3{\_}1&80 80 50&5.9 x 10$^{10}$&450&1.50\\%100
Ma103-3{\_}2&80 80 50&5.9 x 10$^{10}$&450&1.50\\%100
Ma103-5{\_}1&80 50 50&8.6 x 10$^{10}$&450&2.15\\%143.3
Ma104-1{\_}1&60 50 50&8.3 x 10$^{10}$&450&2.31\\%154
Ma104-4{\_}1&80 50 50&4.5 x 10$^{10}$&450&1.13\\%75.3
Ma104-5{\_}1&80 60 50&6.5 x 10$^{10}$&450&1.65\\%110
Ma105-1{\_}1&60 40 20&8.3 x 10$^{10}$&450&2.37\\ %158
Ma105-2{\_}1&100 80 30&7.8 x 10$^{10}$&450&1.87\\%124.7
Ma105-4{\_}1&80 80 40&7.8 x 10$^{10}$&450&1.98\\%132
Ma105-5{\_}1&80 60 40&7.9 x 10$^{10}$&450&2.01\\%134
Ma106-2{\_}1&200 150 50&9.6 x 10$^{10}$&450&1.89\\%126
Ma106-3{\_}1&150 120 50&7.6 x 10$^{10}$&450&1.60\\%106.7
Ma109-3{\_}1&80 40 40&7.9 x 10$^{10}$&450&2.01\\ %134
Ma109-4{\_}1&80 80 20&7.2 x 10$^{10}$&450&1.85\\%123.3
Ma109-5{\_}1&100 100 50&6.9 x 10$^{10}$&450&1.62\\%108 (All 3557.8 / 27 =131.8)
%450 images sum = 2,611.8 / 22 = 118.7 
%11=1306, 12=1424.4 13=1543 14=1662 15=1781
\hline
\end{tabular}
  \caption[Dose Calculations for \atpdx -I320 Crystals used in Multi-Crystal Analysis]{Dose calculations for \atpdx -I320 crystals used in the multi-crystal analysis. 27 datasets were collected from 22 crystals, using the same data collection protocol, aside from five of the datasets where 300 images were collected, rather than 450. The diffraction weighted dose was calculated for each dataset using RADDOSE-3D (see Table \ref{table:RADINP_320} for input parameters), divided by the number of images in the dataset, and multiplied by 30 to determine the dose for a 30 image wedge for each dataset. The mean dose for the first 10 sweeps was calculated using the doses for all datasets while the mean doses for sweeps 10-15 were calculated using only the doses for the 450 image datasets. The dose for each of the first 10 sweeps is 132 kGy, the dose for each of sweeps 11-15 is 119 kGy.}\label{table:I320DoseCalculations} 
\end{table}

\clearpage
\section{Crystallographic Statistics for Multi-Crystal \atpdx-I320 Data} \label{App:multi_I320stats}

\begin{landscape}
\begin{table}[!htbp]
 \centering
\begin{tabular}{ |p{4cm}||p{4cm}|p{4cm}|p{4cm}|}
 \hline
 \multicolumn{4}{|c|}{Crystallographic Statistics for \atpdx-I320 Composite Datasets 1-3} \\
 \hline
 \multicolumn{1}{|l|}{Sweep Number (Dose)} &Sweep 1 (132 kGy)&Sweep 2 (264 kGy)&Sweep 3 (396 kGy)\\
 \hline
 Space group &R3&R3&R3\\
 Unit cell (\textit{a, b, c}) &178.5, 178.5, 116.3&178.5, 178.5, 116.3&178.5, 178.5, 116.3\\
 Resolution  &46.46-2.30 (2.36-2.30)&44.46-2.30 (2.36-2.30)&44.46-2.30 (2.36-2.30)\\
 R\textsubscript{merge}&33.9 (204.2)&38.0 (272.9)&40.9 (251.0)\\
 R\textsubscript{pim}&18.6 (107.5)&26.2 (193.2)&28.1 (176.0)\\	 
 CC\sfrac{1}{2}&92.3 (22.0)&91.4 (16.0)&88.1 (17.2)\\
 \sfrac{I}{$\sigma$(I)}&3.5 (1.3)&3.6 (1.3)&3.7 (1.4)\\
 Completeness (\%)   &98.8 (98.8)&98.4 (98.3)&98.1 (98.0)\\
 Multiplicity    &4.6 (4.5)&4.6 (4.4)&4.6 (4.4)\\
 Unique Reflections    &60658 (4504)&60392 (4479)&60217 (4467)\\
 Wilson B-factor    &24.5&24.5&23.7\\
 \hline 
 \hline 
 \multicolumn{4}{|c|}{Crystallographic Statistics for \atpdx-I320 Composite Datasets 4-6} \\
 \hline
 \multicolumn{1}{|l|}{Sweep Number (Dose)} &Sweep 4 (528 kGy)&Sweep 5 (660 kGy)&Sweep 6 (792 kGy)\\
 \hline
 Space group &R3&R3&R3\\
 Unit cell (\textit{a, b, c}) &178.5, 178.5, 116.3&178.5, 178.5, 116.3&178.5, 178.5, 116.3\\
 Resolution    &46.46-2.30 (2.36-2.30)&46.46-2.30 (2.36-2.30)&41.21-2.30 (2.36-2.30)\\
 R\textsubscript{merge}&34.1 (269.4)&38.3 (323.0)&32.9 (293.3)\\
 R\textsubscript{pim}&23.3 (189.8)&26.2 (225.5)&22.7 (208.7)\\ 
 CC\sfrac{1}{2}&92.2 (15.6)&88.0 (6.2)&93.1 (10.7)\\
 \sfrac{I}{$\sigma$(I)}&3.4 (1.2)&3.4 (1.2)&3.9 (1.3)\\
 Completeness (\%)   &98.4 (98.4)&97.9 (97.9)&98.5 (98.6)\\
 Multiplicity    &4.6 (4.5)&4.6 (4.5)&4.6 (4.5)\\
 Unique Reflections    &60402 (4483)&60091 (4463)&60433 (4488)\\
 Wilson B-factor    &25.9&26.7&26.7\\
 \hline
\end{tabular}
\caption[Crystallographic Statistics for Composite \atpdx-I320 Datasets 1-6]{Table of crystallographic statistics for composite \atpdx-I320 datasets 1-6.} 
\end{table}
\end{landscape}
\clearpage
\begin{landscape}
\begin{table}[!htbp]
 \centering
\begin{tabular}{ |p{4cm}||p{4cm}|p{4cm}|p{4cm}|}
 \hline
 \multicolumn{4}{|c|}{Crystallographic Statistics for \atpdx-I320 Composite Datasets 7-9} \\
 \hline
 \multicolumn{1}{|l|}{Sweep Number (Dose)} &Sweep 7 (924 kGy)&Sweep 8 (1.056 MGy)&Sweep 9 (1.188 MGy)\\
 \hline
 Space group &R3&R3&R3\\
 Unit cell (\textit{a, b, c})&178.50 178.50 116.3&178.50 178.50 116.3&178.50 178.50 116.3\\
 Resolution  &46.46-2.30 (2.36-2.30)&44.62-2.30 (2.36-2.30)&44.62-2.30 (2.36-2.30)\\
 R\textsubscript{merge}&36.9 (332.1)&43.3 (382.9)&46.7 (267.6)\\
 R\textsubscript{pim}&25.8 (237.6)&30.8 (279.6)&32.9 (194.5)\\	 
 CC\sfrac{1}{2}&90.8 (7.8)&86.1 (7.4)&80.2 (6.7)\\
 \sfrac{I}{$\sigma$(I)}&3.6 (1.2)&3.4 (1.2)&3.2 (1.1)\\
 Completeness (\%)   &99.0 (99.1)&99.3 (99.0)&99.0 (98.6)\\
 Multiplicity    &4.5 (4.4)&4.5 (4.4)&4.5 (4.4)\\
 Unique Reflections    &60754 (4518)&60937 (4512)&60768 (4492)\\
 Wilson B-factor    &25.2&23.3&25.2\\
 \hline 
 \hline 
 \multicolumn{4}{|c|}{Crystallographic Statistics for \atpdx-I320 Composite Datasets 10-12} \\
 \hline
 \multicolumn{1}{|l|}{Sweep Number (Dose)} &Sweep 10(1.320 MGy)&Sweep 11 (1.306 MGy)&Sweep 12 (1.424 MGy)\\
 \hline
 Space group &R3&R3&R3\\
 Unit cell (\textit{a, b, c}) &178.50 178.50 116.3&178.7, 178.7, 116.3&178.7, 178.7, 116.3\\
 Resolution    &44.62-2.30 (2.36-2.30)&44.66-2.30 (2.36-2.30)&44.66-2.30 (2.36-2.30)\\
 R\textsubscript{merge}&30.3 (223.7)&39.5 (355.0)&37.6 (378.4)\\
 R\textsubscript{pim}&20.9 (158.9)&29.2 (268.4)&27.5 (281.6)\\ 
 CC\sfrac{1}{2}&94.7 (13.2)&82.3 (7.5)&89.3 (3.5)\\
 \sfrac{I}{$\sigma$(I)}&3.5 (1.1)&3.3 (1.1)&3.1 (0.8)\\
 Completeness (\%)   &99.3 (99.3)&97.4 (97.5)&96.9 (96.9)\\
 Multiplicity    &4.5 (4.5)&3.8 (3.4)&3.8 (3.7)\\
 Unique Reflections    &60940 (4526)&59930 (4676)&59603 (4647)\\
 Wilson B-factor    &26.1&21.0&23.3\\
 \hline
\end{tabular}
\caption[Crystallographic Statistics for Composite \atpdx-I320 Datasets 7-12]{Table of crystallographic statistics for composite \atpdx-I320 datasets 7-12.} 
\end{table}
\end{landscape}
\clearpage
\begin{landscape}
\begin{table}[!htbp]
 \centering
\begin{tabular}{ |p{4cm}||p{4cm}|p{4cm}|p{4cm}|}
 \hline
 \multicolumn{4}{|c|}{Crystallographic Statistics for \atpdx-I320 Composite Datasets 13-15} \\
 \hline
 \multicolumn{1}{|l|}{Sweep Number (Dose)} &Sweep 13 (1.543 MGy)&Sweep 14 (1.662 MGy)&Sweep 15 (1.781 MGy)\\
 \hline
 Space group &R3&R3&R3\\
 Unit cell (\textit{a, b, c}) &178.7, 178.7, 116.3&178.7, 178.7, 116.3&178.7, 178.7, 116.3\\
 Resolution  &44.66-2.30 (2.36-2.30)&46.48-2.30 (2.36-2.30)&44.66-2.30 (2.30-2.36)\\
 R\textsubscript{merge}&34.1 (3.127)&39.2 (407.1)&47.8 (586.4)\\
 R\textsubscript{pim}&25.0 (232.3)&29.5 (314.5)&34.4 (434.4)\\	 
 CC\sfrac{1}{2}&91.6 (5.2)&84.5 (3.6)&78.5 (3.2)\\
 \sfrac{I}{$\sigma$(I)}&3.3 (1.0)&3.4 (1.0)&3.1 (0.9)\\
 Completeness (\%)   &96.8 (96.5)&97.8 (97.9)&96.7 (97.1)\\
 Multiplicity    &3.8 (3.7)&3.8 (3.6)&3.8 (3.7)\\
 Unique Reflections    &59516 (4627)&60144 (4474)&59493 (4659)\\
 Wilson B-factor    &22.9&25.3&27.2\\
 \hline 
\end{tabular}
\caption[Crystallographic Statistics for Composite \atpdx-I320 Datasets 13-15]{Table of crystallographic statistics for composite \atpdx-I320 datasets 13-15.} 
\end{table}
\end{landscape}
\clearpage

\section{Isomorphous Difference Density Maps for Multi-Crystal \atpdx -I320 Data}\label{App:I320ALL_fofo}
\subsection*{Fo-Fo Maps for Multi-Crystal \atpdx -I320 Chain A}

\begin{minipage}{\linewidth}
	\makebox[\linewidth]{
		
	\includegraphics[width=16cm, height=16cm, keepaspectratio]{/Users/matt/Dropbox/ThesisPrep/Thesis/fig/blend/I320ChainA.png}}	

	\captionof{figure}[\atpdx -I320 Chain A Isomorphous Difference Density Maps]{Isomorphous Difference Density Maps for the I320 intermediate in chain A of \atpdx -I320. Lysine 98 (left) is linked by the carbon atoms 1-5 (left to right, orange) of I320 to lysine 166 (right), the C2 nitrogen is shown in blue and the C3 oxygen is shown in red. Red peaks indicate loss of electrons, green peaks indicate areas with increased electron density, contoured at 3.0 $\sigma$ and -3.0 $\sigma$ respectively.}	
\end{minipage}

\subsection*{Fo-Fo Maps for Multi-Crystal \atpdx -I320 Chain B}

\begin{minipage}{\linewidth}
	\makebox[\linewidth]{
		
	\includegraphics[width=16cm, height=16cm, keepaspectratio]{/Users/matt/Dropbox/ThesisPrep/Thesis/fig/blend/I320ChainB.png}}	

	\captionof{figure}[\atpdx -I320 Chain B Isomorphous Difference Density Maps]{Isomorphous Difference Density Maps for the I320 intermediate in chain B of \atpdx -I320. Lysine 98 (left) is linked by the carbon atoms 1-5 (left to right, orange) of I320 to lysine 166 (right), the C2 nitrogen is shown in blue and the C3 oxygen is shown in red. Red peaks indicate loss of electrons, green peaks indicate areas with increased electron density, contoured at 3.0 $\sigma$ and -3.0 $\sigma$ respectively.}	
\end{minipage}

\subsection*{Fo-Fo Maps for Multi-Crystal \atpdx -I320 Chain C}

\begin{minipage}{\linewidth}
	\makebox[\linewidth]{
		
	\includegraphics[width=16cm, height=16cm, keepaspectratio]{/Users/matt/Dropbox/ThesisPrep/Thesis/fig/blend/I320ChainC.png}}	

	\captionof{figure}[\atpdx -I320 Chain C Isomorphous Difference Density Maps]{Isomorphous Difference Density Maps for the I320 intermediate in chain C of \atpdx -I320. Lysine 98 (left) is linked by the carbon atoms 1-5 (left to right, orange) of I320 to lysine 166 (right), the C2 nitrogen is shown in blue and the C3 oxygen is shown in red. Red peaks indicate loss of electrons, green peaks indicate areas with increased electron density, contoured at 3.0 $\sigma$ and -3.0 $\sigma$ respectively.}	
\end{minipage}

\subsection*{Fo-Fo Maps for Multi-Crystal \atpdx -I320 Chain D}

\begin{minipage}{\linewidth}
	\makebox[\linewidth]{
		
	\includegraphics[width=16cm, height=16cm, keepaspectratio]{/Users/matt/Dropbox/ThesisPrep/Thesis/fig/blend/I320ChainD.png}}	

	\captionof{figure}[\atpdx -I320 Chain D Isomorphous Difference Density Maps]{Isomorphous Difference Density Maps for the I320 intermediate in chain D of \atpdx -I320. Lysine 98 (left) is linked by the carbon atoms 1-5 (left to right, orange) of I320 to lysine 166 (right), the C2 nitrogen is shown in blue and the C3 oxygen is shown in red. Red peaks indicate loss of electrons, green peaks indicate areas with increased electron density, contoured at 3.0 $\sigma$ and -3.0 $\sigma$ respectively.}	
\end{minipage}

\clearpage
\section{Bash Scripts for Processing of Multi-Crystal X-ray Diffraction Data} \label{App:Multi-XtalScripts}
\subsection*{Bash Script using XDS to Index and Integrate Multi-Crystal Data}
 \begin{lstlisting}[language=bash]
#Script to run XDS on all of the Pdx1 I320 datasets

#Search for all files ending with 0001.cbf in the 151213 folder and the paths to the ori1.dat text file. Search through ori1 for test images and remove them from the list of files

find /Volumes/SOTONTEWS/ESRF/151213/ -name "*0001.cbf" > ori1.dat
sed -i.bak '/test/d' ./ori1.dat #remove test images

#Make xds directory
mkdir xds

#Count the number of different datasets that are listed in ori1 and save as variable b
b=`wc ori1.dat | awk '{print $1}'`
echo There are $b datasets

#k is the number of images per sweep, can be changed depending on acceptable dose
k=30

#c will be the variable used as a counter that increases with each loop iteration
c=1

while [ $c -le $b ]
	do
   	#d is the directory that each dataset is in
        d=$(dirname `awk "FNR == $c" ori1.dat`)
   	
	#ca is c written as a three digit number, if c is 12, ca is 012.
	ca=`printf "%03d" $c`
	
	#e is the file pattern for the dataset with the image number cut off 
	e=`awk "FNR == $c" ori1.dat | rev | cut -c 9- | rev`
	
	#Keep track of which dataset becomes which wedge in xds_I320 log file 
	echo Wedge_$ca is $d >> xds_I320.log 

	####### GREP Data Collection Parameters form Image Header#######	
	head -40 $e"0001.cbf" > header.txt
	###$WL is wavelength variable
	WL=$(head -40 header.txt | grep "Wavelength" | awk -F" " '{print $3}')
	
	###$DIS is detector distance variable
	DIS=`grep -e 'Detector_distance' header.txt | cut -c 21- | rev | cut -c 4- |rev`
	DISa=$(echo "scale=5; $DIS*1000" | bc -l)
	
	mkdir xds/wedge_$ca
	
	###$BMX and $BMY define the beam origin for the detector
	BMX=`grep -e 'Beam_xy' header.txt | cut -c 12- | rev | cut -c 19-| rev` 
	BMY=`grep -e 'Beam_xy' header.txt  | cut -c 21- | rev | cut -c 10- |rev`
	
###List the file names of all images in dataset in ori2.dat text file
	ls $e*cbf > ori2.dat
	
#Count how many are in the current dataset
	f=`wc ori2.dat | awk '{print $1}'`
	echo There are $f images in dataset $c >> xds_I320.log	
	echo There are $f images in dataset $c 

#Divide the number of images in the dataset by the number of images per sweep to calculate the number of sweeps that can be obtained from the dataset, save as variable g
	g=$(echo "$f/$k" | bc) 
	echo Wedge_$ca will have $g sweeps	

	#Sweep number counter will be $h
	h=1
	while [ $h -le $g ]
   	do
		ha=`printf "%03d" $h`
		mkdir xds/wedge_$ca/sweep_$ha
		touch xds/wedge_$ca/sweep_$ha/XDS.INP
		
		i=$(( ($k * ( $h - 1 )+1) ))
		echo Start image is number $i

      	j=$(( $k * $h ))
      	echo Final image number is $j

#If this is the first sweep of the dataset first run XDS indexing using multiple spot ranges. Make sure the x_geo_corr.cbf and y_geo_corr.cbf are in the correct directory. If data is collected using a detector that is not a PILATUS 6M-F the trusted regions, and other detector parameters will need to be changed. Change unit cell parameters and resolution shell depending on protein / estimated quality of data.    

if [ $h -eq 1 ]
then
################## START WRITING XDS.INP INDEXING #####################
echo JOB= XYCORR COLSPOT INIT IDXREF>> xds/wedge_$ca/sweep_$ha/XDS.INP
echo DATA_RANGE= 1 300			   >> xds/wedge_$ca/sweep_$ha/XDS.INP
echo SPOT_RANGE= 1 20 	           >> xds/wedge_$ca/sweep_$ha/XDS.INP
echo SPOT_RANGE= 100 120 		   >> xds/wedge_$ca/sweep_$ha/XDS.INP
echo SPOT_RANGE= 200 220 		   >> xds/wedge_$ca/sweep_$ha/XDS.INP
echo   							   >> xds/wedge_$ca/sweep_$ha/XDS.INP
echo BACKGROUND_RANGE= 1 20		   >> xds/wedge_$ca/sweep_$ha/XDS.INP
echo   							   >> xds/wedge_$ca/sweep_$ha/XDS.INP
echo							  >> xds/wedge_$ca/sweep_$ha/XDS.INP
echo !masking non sensitive area of Pilatus>> xds/wedge_$ca/sweep_$ha/XDS.INP
echo UNTRUSTED_RECTANGLE=487 495 0 2528>>xds/wedge_$ca/sweep_$ha/XDS.INP
echo UNTRUSTED_RECTANGLE=981 989 0 2528>>xds/wedge_$ca/sweep_$ha/XDS.INP
echo UNTRUSTED_RECTANGLE=1475 1483 0 2528>>xds/wedge_$ca/sweep_$ha/XDS.INP
echo UNTRUSTED_RECTANGLE=1969 1977 0 2528>>xds/wedge_$ca/sweep_$ha/XDS.INP
echo UNTRUSTED_RECTANGLE=0 2464 195 213>>xds/wedge_$ca/sweep_$ha/XDS.INP
echo UNTRUSTED_RECTANGLE=0 2464 407 425>>xds/wedge_$ca/sweep_$ha/XDS.INP
echo UNTRUSTED_RECTANGLE=0 2464 619 637>>xds/wedge_$ca/sweep_$ha/XDS.INP
echo UNTRUSTED_RECTANGLE=0 2464 831 849>>xds/wedge_$ca/sweep_$ha/XDS.INP
echo UNTRUSTED_RECTANGLE=0 2464 1043 1061>>xds/wedge_$ca/sweep_$ha/XDS.INP
echo UNTRUSTED_RECTANGLE=0 2464 1255 1273>>xds/wedge_$ca/sweep_$ha/XDS.INP
echo UNTRUSTED_RECTANGLE=0 2464 1467 1485>>xds/wedge_$ca/sweep_$ha/XDS.INP
echo UNTRUSTED_RECTANGLE=0 2464 1679 1697>>xds/wedge_$ca/sweep_$ha/XDS.INP
echo UNTRUSTED_RECTANGLE=0 2464 1891 1909>>xds/wedge_$ca/sweep_$ha/XDS.INP
echo UNTRUSTED_RECTANGLE=0 2464 2103 2121>>xds/wedge_$ca/sweep_$ha/XDS.INP
echo UNTRUSTED_RECTANGLE=0 2464 2315 2333>>xds/wedge_$ca/sweep_$ha/XDS.INP
echo TRUSTED_REGION=0.0 1.41 !Relative radii limiting trusted	 >> xds/wedge_$ca/sweep_$ha/XDS.INP
echo							   >> xds/wedge_$ca/sweep_$ha/XDS.INP
echo   !correction tables to compensate the misorientations of the modules >> xds/wedge_$ca/sweep_$ha/XDS.INP
echo							   >> xds/wedge_$ca/sweep_$ha/XDS.INP
echo   X-GEO_CORR= /Volumes/SOTONTEWS/ESRF/151213/RAW_DATA/ma94-5/w2/process//x_geo_corr.cbf >> xds/wedge_$ca/sweep_$ha/XDS.INP
echo   Y-GEO_CORR= /Volumes/SOTONTEWS/ESRF/151213/RAW_DATA/ma94-5/w2/process//y_geo_corr.cbf >> xds/wedge_$ca/sweep_$ha/XDS.INP
echo							   >> xds/wedge_$ca/sweep_$ha/XDS.INP
echo SECONDS=600				   >> xds/wedge_$ca/sweep_$ha/XDS.INP
echo MINIMUM_NUMBER_OF_PIXELS_IN_A_SPOT=3 >> xds/wedge_$ca/sweep_$ha/XDS.INP
echo							   >> xds/wedge_$ca/sweep_$ha/XDS.INP
echo !STRONG_PIXEL= 3.0  	     >> xds/wedge_$ca/sweep_$ha/XDS.INP
echo							   >> xds/wedge_$ca/sweep_$ha/XDS.INP
echo OSCILLATION_RANGE= 0.2000  >> xds/wedge_$ca/sweep_$ha/XDS.INP
echo X-RAY_WAVELENGTH=  $WL	   >> xds/wedge_$ca/sweep_$ha/XDS.INP
echo NAME_TEMPLATE_OF_DATA_FRAMES= $e"????.cbf !CBF" >> xds/wedge_$ca/sweep_$ha/XDS.INP
echo								   >> xds/wedge_$ca/sweep_$ha/XDS.INP
echo !STARTING_ANGLES_OF_SPINDLE_ROTATION=0 180 10>> xds/wedge_$ca/sweep_$ha/XDS.INP
echo !TOTAL_SPINDLE_ROTATION_RANGES=60 180 10>>xds/wedge_$ca/sweep_$ha/XDS.INP
echo						     	   >>xds/wedge_$ca/sweep_$ha/XDS.INP
echo DETECTOR_DISTANCE= $DISa      >>xds/wedge_$ca/sweep_$ha/XDS.INP
echo DETECTOR= PILATUS MINIMUM_VALID_PIXEL_VALUE=0.0 OVERLOAD=1048500>>xds/wedge_$ca/sweep_$ha/XDS.INP
echo								   >>xds/wedge_$ca/sweep_$ha/XDS.INP
echo SENSOR_THICKNESS=0.32		   >>xds/wedge_$ca/sweep_$ha/XDS.INP
echo ORGX= $BMX ORGY= $BMY   	   >>xds/wedge_$ca/sweep_$ha/XDS.INP
echo NX= 2463 NY= 2527		      >>xds/wedge_$ca/sweep_$ha/XDS.INP
echo QX= 0.1720  QY= 0.1720		  >> xds/wedge_$ca/sweep_$ha/XDS.INP
echo VALUE_RANGE_FOR_TRUSTED_DETECTOR_PIXELS= 7000 30000		   >> xds/wedge_$ca/sweep_$ha/XDS.INP
echo								>> xds/wedge_$ca/sweep_$ha/XDS.INP
echo DIRECTION_OF_DETECTOR_X-AXIS= 1.0 0.0 0.0>>xds/wedge_$ca/sweep_$ha/XDS.INP
echo DIRECTION_OF_DETECTOR_Y-AXIS= 0.0 1.0 0.0>>xds/wedge_$ca/sweep_$ha/XDS.INP
echo ROTATION_AXIS= 1.0 0.0 0.0>>xds/wedge_$ca/sweep_$ha/XDS.INP
echo INCIDENT_BEAM_DIRECTION= 0.0 0.0 1.0>>xds/wedge_$ca/sweep_$ha/XDS.INP
echo FRACTION_OF_POLARIZATION= 0.99>>xds/wedge_$ca/sweep_$ha/XDS.INP
echo POLARIZATION_PLANE_NORMAL= 0.0 1.0 0.0>>xds/wedge_$ca/sweep_$ha/XDS.INP
echo !AIR= %.8f					    >> xds/wedge_$ca/sweep_$ha/XDS.INP
echo								    >> xds/wedge_$ca/sweep_$ha/XDS.INP
echo SPACE_GROUP_NUMBER= 146	        >> xds/wedge_$ca/sweep_$ha/XDS.INP
echo UNIT_CELL_CONSTANTS= 178.64 178.64 116.63 90 90 120		   >> xds/wedge_$ca/sweep_$ha/XDS.INP
echo INCLUDE_RESOLUTION_RANGE= 50.0 2>> xds/wedge_$ca/sweep_$ha/XDS.INP
echo !STRICT_ABSORPTION_CORRECTION=TRUE>>xds/wedge_$ca/sweep_$ha/XDS.INP
echo								>> xds/wedge_$ca/sweep_$ha/XDS.INP
echo "REFINE(INTEGRATE)= BEAM ORIENTATION CELL"			   >> xds/wedge_$ca/sweep_$ha/XDS.INP
echo   !== Default value recommended	   >> xds/wedge_$ca/sweep_$ha/XDS.INP
echo   !DELPHI= %.3f					   >> xds/wedge_$ca/sweep_$ha/XDS.INP
echo   MAXIMUM_NUMBER_OF_PROCESSORS= 8  >> xds/wedge_$ca/sweep_$ha/XDS.INP
echo   !MAXIMUM_NUMBER_OF_JOBS= 16	   >> xds/wedge_$ca/sweep_$ha/XDS.INP
###########################END XDS INDEXING FILE###########################

#Change directory to same directory as XDS input file
		cd xds/wedge_$ca/sweep_$ha/
#Run XDS indexing procedures 
		xds_par
#Return to starting directory
		cd -
fi

if [ $h -gt 1 ]
	then 
#If this is not the first sweep in the dataset then copy the output files from the indexing run to the folder for the new sweep rather than re-running the indexing for each sweep.
	cp xds/wedge_$ca/sweep_001/*cbf         xds/wedge_$ca/sweep_$ha/
	cp xds/wedge_$ca/sweep_001/SPOT.XDS     xds/wedge_$ca/sweep_$ha/
	cp xds/wedge_$ca/sweep_001/GXPARM.XDS   xds/wedge_$ca/sweep_$ha/ 
	cp xds/wedge_$ca/sweep_001/XPARM.XDS    xds/wedge_$ca/sweep_$ha/ 
fi

#See above indexing procedure, run the integration procedure in XDS 
#################### START WRITING XDS.INP INTEGRATION ####################
echo   JOB= DEFPIX INTEGRATE > xds/wedge_$ca/sweep_$ha/XDS.INP
echo   DATA_RANGE= $i $j		>> xds/wedge_$ca/sweep_$ha/XDS.INP
echo   						>> xds/wedge_$ca/sweep_$ha/XDS.INP
echo   SPOT_RANGE= $i $j     >> xds/wedge_$ca/sweep_$ha/XDS.INP
echo   						>> xds/wedge_$ca/sweep_$ha/XDS.INP
echo   BACKGROUND_RANGE= $i $j	   >> xds/wedge_$ca/sweep_$ha/XDS.INP
echo   							   >> xds/wedge_$ca/sweep_$ha/XDS.INP
echo								   >> xds/wedge_$ca/sweep_$ha/XDS.INP
echo UNTRUSTED_RECTANGLE=487 495 0 2528>>xds/wedge_$ca/sweep_$ha/XDS.INP
echo UNTRUSTED_RECTANGLE=981 989 0 2528>>xds/wedge_$ca/sweep_$ha/XDS.INP
echo UNTRUSTED_RECTANGLE=1475 1483 0 2528>>xds/wedge_$ca/sweep_$ha/XDS.INP
echo UNTRUSTED_RECTANGLE=1969 1977 0 2528>>xds/wedge_$ca/sweep_$ha/XDS.INP
echo UNTRUSTED_RECTANGLE=0 2464 195 213>>xds/wedge_$ca/sweep_$ha/XDS.INP
echo UNTRUSTED_RECTANGLE=0 2464 407 425>>xds/wedge_$ca/sweep_$ha/XDS.INP
echo UNTRUSTED_RECTANGLE=0 2464 619 637>>xds/wedge_$ca/sweep_$ha/XDS.INP
echo UNTRUSTED_RECTANGLE=0 2464 831 849>>xds/wedge_$ca/sweep_$ha/XDS.INP
echo UNTRUSTED_RECTANGLE=0 2464 1043 1061>>xds/wedge_$ca/sweep_$ha/XDS.INP
echo UNTRUSTED_RECTANGLE=0 2464 1255 1273>>xds/wedge_$ca/sweep_$ha/XDS.INP
echo UNTRUSTED_RECTANGLE=0 2464 1467 1485>>xds/wedge_$ca/sweep_$ha/XDS.INP
echo UNTRUSTED_RECTANGLE=0 2464 1679 1697>>xds/wedge_$ca/sweep_$ha/XDS.INP
echo UNTRUSTED_RECTANGLE=0 2464 1891 1909>>xds/wedge_$ca/sweep_$ha/XDS.INP
echo UNTRUSTED_RECTANGLE=0 2464 2103 2121>>xds/wedge_$ca/sweep_$ha/XDS.INP
echo UNTRUSTED_RECTANGLE=0 2464 2315 2333>>xds/wedge_$ca/sweep_$ha/XDS.INP
echo TRUSTED_REGION=0.0 1.41 !Relative radii limiting trusted >>xds/wedge_$ca/sweep_$ha/XDS.INP
echo	        					   >> xds/wedge_$ca/sweep_$ha/XDS.INP
echo   !correction tables to compensate the misorientations of the modules >> xds/wedge_$ca/sweep_$ha/XDS.INP
echo							   >> xds/wedge_$ca/sweep_$ha/XDS.INP
echo   X-GEO_CORR= /Volumes/SOTONTEWS/ESRF/151213/RAW_DATA/ma94-5/w2/process//x_geo_corr.cbf >> xds/wedge_$ca/sweep_$ha/XDS.INP
echo   Y-GEO_CORR= /Volumes/SOTONTEWS/ESRF/151213/RAW_DATA/ma94-5/w2/process//y_geo_corr.cbf >> xds/wedge_$ca/sweep_$ha/XDS.INP
echo  						   >> xds/wedge_$ca/sweep_$ha/XDS.INP
echo SECONDS=600				   >> xds/wedge_$ca/sweep_$ha/XDS.INP
echo MINIMUM_NUMBER_OF_PIXELS_IN_A_SPOT= 3>> xds/wedge_$ca/sweep_$ha/XDS.INP
echo							   >> xds/wedge_$ca/sweep_$ha/XDS.INP
echo !STRONG_PIXEL= 3.0	       >> xds/wedge_$ca/sweep_$ha/XDS.INP
echo							   >> xds/wedge_$ca/sweep_$ha/XDS.INP
echo OSCILLATION_RANGE= 0.200   >> xds/wedge_$ca/sweep_$ha/XDS.INP
echo X-RAY_WAVELENGTH=  $WL	   >> xds/wedge_$ca/sweep_$ha/XDS.INP
echo NAME_TEMPLATE_OF_DATA_FRAMES= $e"????.cbf !CBF">> xds/wedge_$ca/sweep_$ha/XDS.INP
echo							   >> xds/wedge_$ca/sweep_$ha/XDS.INP
echo !STARTING_ANGLES_OF_SPINDLE_ROTATION= 0 180 10>> xds/wedge_$ca/sweep_$ha/XDS.INP
echo !TOTAL_SPINDLE_ROTATION_RANGES= 60 180 10>> xds/wedge_$ca/sweep_$ha/XDS.INP
echo								>> xds/wedge_$ca/sweep_$ha/XDS.INP
echo   DETECTOR_DISTANCE= $DISa  >> xds/wedge_$ca/sweep_$ha/XDS.INP
echo   DETECTOR= PILATUS MINIMUM_VALID_PIXEL_VALUE= 0.0 OVERLOAD= 1048500  >> xds/wedge_$ca/sweep_$ha/XDS.INP
echo							    >> xds/wedge_$ca/sweep_$ha/XDS.INP
echo SENSOR_THICKNESS=0.32			   >> xds/wedge_$ca/sweep_$ha/XDS.INP
echo ORGX= $BMX ORGY= $BMY			   >> xds/wedge_$ca/sweep_$ha/XDS.INP
echo NX= 2463 NY= 2527				   >> xds/wedge_$ca/sweep_$ha/XDS.INP
echo QX= 0.1720  QY= 0.1720		       >> xds/wedge_$ca/sweep_$ha/XDS.INP
echo VALUE_RANGE_FOR_TRUSTED_DETECTOR_PIXELS= 7000 30000		   >> xds/wedge_$ca/sweep_$ha/XDS.INP
echo								>> xds/wedge_$ca/sweep_$ha/XDS.INP
echo DIRECTION_OF_DETECTOR_X-AXIS= 1.0 0.0 0.0 >> xds/wedge_$ca/sweep_$ha/XDS.INP
echo DIRECTION_OF_DETECTOR_Y-AXIS= 0.0 1.0 0.0 >> xds/wedge_$ca/sweep_$ha/XDS.INP
echo ROTATION_AXIS= 1.0 0.0 0.0		   >> xds/wedge_$ca/sweep_$ha/XDS.INP
echo INCIDENT_BEAM_DIRECTION=0.0 0.0 1.0>>xds/wedge_$ca/sweep_$ha/XDS.INP
echo FRACTION_OF_POLARIZATION= 0.99	>> xds/wedge_$ca/sweep_$ha/XDS.INP
echo POLARIZATION_PLANE_NORMAL=0 1 0 >> xds/wedge_$ca/sweep_$ha/XDS.INP
echo !AIR= %.8f					   >> xds/wedge_$ca/sweep_$ha/XDS.INP
echo								   >> xds/wedge_$ca/sweep_$ha/XDS.INP
echo SPACE_GROUP_NUMBER= 146		   >> xds/wedge_$ca/sweep_$ha/XDS.INP
echo UNIT_CELL_CONSTANTS=178.6 178.6 116.6 90 90 120>>xds/wedge_$ca/sweep_$ha/XDS.INP
echo INCLUDE_RESOLUTION_RANGE= 50 2 >> xds/wedge_$ca/sweep_$ha/XDS.INP
echo !STRICT_ABSORPTION_CORRECTION=TRUE>> xds/wedge_$ca/sweep_$ha/XDS.INP
echo								 >> xds/wedge_$ca/sweep_$ha/XDS.INP
echo "REFINE(INTEGRATE)= BEAM ORIENTATION CELL">> xds/wedge_$ca/sweep_$ha/XDS.INP
echo !== Default value recommended	   >> xds/wedge_$ca/sweep_$ha/XDS.INP
echo !DELPHI= %.3f					   >> xds/wedge_$ca/sweep_$ha/XDS.INP
echo MAXIMUM_NUMBER_OF_PROCESSORS= 8	   >> xds/wedge_$ca/sweep_$ha/XDS.INP
echo   !MAXIMUM_NUMBER_OF_JOBS= 16	   >> xds/wedge_$ca/sweep_$ha/XDS.INP
######################END XDS INTEGRATION FILE##############################
                #Change directory to location of XDS input file
                cd xds/wedge_$ca/sweep_$ha/	
		#Run XDS
		xds_par
		#Return to original directory 
		cd -
		#Increment sweep counter 
		(( h ++ ))
		done
	#Increment wedge / dataset number
	(( c ++ ))
  	done  
 \end{lstlisting}
 \newpage
\subsection*{Bash Script Using BLEND to Scale and Merge Multi-Crystal Data}\label{App:Multi_XtalBlend}
\begin{lstlisting}[language=bash]
#Script to run all sweeps through blend in analysis and synthesis mode
#Load modules for blend R and ccp4 before running script
#Counter for the sweep number
a=1

#Total number of sweeps is b
b=15

while [ $a -le $b ]
do 	
	echo Working on sweep $a
	aa=`printf "%03d" $a`
	mkdir blendsweep_$aa
	cd blendsweep_$aa
	#Find all folders named sweep_$aa, if a=1 then find sweep_001	
	find /dls/mx-scratch/matt/Pdx1BLEND/xds_160718/integratedfiles -name "*sweep_$aa" | sort > ori1.dat

	echo All sweep $a folders found
	
	sed -e 's/$/\/INTEGRATE.HKL/' -i ori1.dat
	
	echo BLEND analysis mode initiated for sweep $a
	#Analysis Keyword file contains the commands RESO HIGH =2.3, RADFRAC=0. See BLEND documentation for more info.
	blend -a ori1.dat < /dls/mx-scratch/matt/Pdx1BLEND/xds_160718/blend/analysis_keywords.dat > blend_analysis.log
		
	echo BLEND analysis mode complete for sweep $a

	echo BLEND synthesis mode initiated for sweep $a
    #Synthesis keyword file contains the command RESO HIGH =2.3. See BLEND documentation for more info.
	blend -s 1000 1.0 < /dls/mx-scratch/matt/Pdx1BLEND/xds_160718/blend/synthesis_keywords.dat > blend_synthesis.log	
		
	echo BLEND synthesis mode complete for sweep $a

	cd ../
	
	#Increment to the next sweep
	(( a ++ ))
done
\end{lstlisting}
\clearpage
%\section{Library files for for refinement of Pdx1 complexes}
%\clearpage
\section{Buffer constituents for Radiation Damage Investigations}\label{App:buffer_comp} 
\begin{table}[!hbp]
\centering
\begin{tabular}{|p{2.5cm}|p{6cm}|P{2.4cm}|P{1.8cm}|}
 \hline
Buffer Name& Buffer Constituents&Flux (photons \si{\per\second})&Dose Rate (\si{\gray\per\second})\\
 \hline
Water&Distilled Water&2.7x10$^{10}$&130\\
40\% Glycerol&40\% Glycerol (v/v)&2.1x10$^{10}$&101\\
100\% Glycerol&100\% Glycerol (v/v)&3.1x10$^{10}$&149\\
40\% MPD&40\% MPD (v/v)&2.3x10$^{10}$&111\\
100\% MPD&100\% MPD (v/v)&2.9x10$^{10}$&139\\
Pdx1 Glycerol Buffer&5.5\% PEG 4000 (w/v),400 mM Sodium Acetate, 100 mM Tris (pH 8.0), 20\% Glycerol (v/v)&2.8x10$^{10}$&156\\
WT \atpdx~ protein film&150 \si{\micro\molar} WT \atpdx, 150 mM Potassium Chloride, 37.5 \si{\milli\molar} Tris pH 8.0, 20\% Glycerol (v/v)&2.5x10$^{10}$&140\\
\atpdx ~D41N protein film&222 \si{\micro\molar} \atpdx~ D41N, 150 mM Potassium Chloride, 37.5 \si{\milli\molar} Tris pH 8.0, 20\% Glycerol (v/v)&2.5x10$^{10}$&140\\
GB3 protein film&1.13 mM GB3 protein in GB3 Buffer&2.4x10$^{10}$&115\\
GB3 buffer&25 mM Bis Tris (pH 6.5), 100mM Sodium Chloride, 0.05\% Sodium Azide, 0.5 mM Tris(2-carboxyethyl)phosphine (TCEP)&2.3x10$^{10}$&120\\
\hline 
\end{tabular}
\caption[Constituents of buffers exposed to X-rays]{Constituents of the buffer solutions exposed to X-rays as UV-Vis spectra were collected. The average dose in the exposed area is used when referring to spectroscopy data. The beam parameters for the dose calculation are listed in Table \ref{table:RADINP_Buffer}}.\label{table:BufferSpec} 
\end{table} 
\clearpage
\section{MATLAB Code for Processing UV-Vis Spectroscopy Data} \label{App:MATLABSPEC}
\subsection*{Script for Importing Spectra as a 3D Array}\label{App:MATLABSPEC_IMPORT}
\begin{lstlisting} 
%This script will import all UV-Vis spectra with the ".txt" extension and 
%file pattern"Glycerol_R2" and load them in order into a 3-D array called
%"Glycerol_R2".

%Tell the script which folder to look for the spectra in
myFolder = '/Users/matt/Dropbox/Pdx1project/PhD/Spectroscopy/ESRFSpec/mx1733/Moritz/Matt/glycerol100run2';

%Error message will show if the myFolder directory cannot be found
if ~isdir(myFolder);
  errorMessage = sprintf('Error: The following folder does not exist:\n%s', myFolder);
  uiwait(warndlg(errorMessage));
  return;
end

%Reads the pattern for the files with .txt extension
filepattern = fullfile(myFolder, 'glycerol100r2_1800s*.txt');
specfiles = dir(filepattern);

for k = 1:length(specfiles);
    baseFilename = specfiles(k).name;
    fullFilename = fullfile(myFolder, baseFilename);
    fprintf (1, 'Now reading %s\n', fullFilename);
   
   %Import data from text file
   delimiter = '\t';
   startRow = 20;
   formatSpec = '%s%s%[^\n\r]';
   fileID = fopen(fullFilename,'r');
   dataArray = textscan(fileID, formatSpec, 'Delimiter', delimiter, 'HeaderLines' ,startRow-1, 'ReturnOnError', false); 
   fclose(fileID);
        %% Convert the contents of columns containing numeric strings to numbers.
% Replace non-numeric strings with NaN.
raw = repmat({''},length(dataArray{1}),length(dataArray)-1);
for col=1:length(dataArray)-1
    raw(1:length(dataArray{col}),col) = dataArray{col};
end
numericData = NaN(size(dataArray{1},1),size(dataArray,2));

for col=[1,2]
    % Converts strings in the input cell array to numbers. Replaced non-numeric
    % strings with NaN.
    rawData = dataArray{col};
    for row=1:size(rawData, 1);
        % Create a regular expression to detect and remove non-numeric prefixes and
        % suffixes.
        regexstr = '(?<prefix>.*?)(?<numbers>([-]*(\d+[\,]*)+[\.]{0,1}\d*[eEdD]{0,1}[-+]*\d*[i]{0,1})|([-]*(\d+[\,]*)*[\.]{1,1}\d+[eEdD]{0,1}[-+]*\d*[i]{0,1}))(?<suffix>.*)';
        try
            result = regexp(rawData{row}, regexstr, 'names');
            numbers = result.numbers;
            
            % Detected commas in non-thousand locations.
            invalidThousandsSeparator = false;
            if any(numbers==',');
                thousandsRegExp = '^\d+?(\,\d{3})*\.{0,1}\d*$';
                if isempty(regexp(thousandsRegExp, ',', 'once'));
                    numbers = NaN;
                    invalidThousandsSeparator = true;
                end
            end
            % Convert numeric strings to numbers.
            if ~invalidThousandsSeparator;
                numbers = textscan(strrep(numbers, ',', ''), '%f');
                numericData(row, col) = numbers{1};
                raw{row, col} = numbers{1};
            end
        catch me
        end
    end
end
%% Replace non-numeric cells with NaN
R = cellfun(@(x) ~isnumeric(x) && ~islogical(x),raw); % Find non-numeric cells
raw(R) = {NaN}; % Replace non-numeric cells

%% Create output variable
Glycerol_R2(:,:,k)= cell2mat(raw);

%% Clear temporary variables
clearvars i k filename delimiter startRow formatSpec fileID dataArray ans raw col numericData rawData row regexstr result numbers invalidThousandsSeparator thousandsRegExp me R;
end
\end{lstlisting}
\newpage
\subsection*{Script for Smoothing Spectra using a Savitzky-Golay Filter}\label{App:MATLABSPEC_SGOLAY}
 \begin{lstlisting} 
%Script to smooth all absorbance values in the 3D array Glycerol_R2 using
%a Savitzky-Golay Filter

%'a' is a variable that will increase until it is equal to the number of 
%pages in the Glycerol_R2 3D array, at that point the loop will exit
a=1;
while a<=size(Glycerol_R2, 3)
    %Write absorbance values to temporary array 'y' column 1
    y(:,1)=Glycerol_R2(:,2,a);    
    %Perform smoothing with a window of 20 and a polynomial order of 2
    %write smoothed values to 'y' column 2
    y(:,2)=smooth(y(:,1),20,'sgolay',2);
    %Write smoothed values to 3D array, column 3, page 'a'
    Glycerol_R2(:,3,a)=y(:,2);
    %Loop
    a=a+1;
    %Clear temporary variables
    clearvars a y;
end
 
 \end{lstlisting}
 
\subsection*{Script for Normalising all Spectra in Series at 900 nm}\label{App:MATLABSPEC_NORM}
\begin{lstlisting}
%Script to internally normalise the spectra at 900 nm where no signal is
%expected.
%'a' is a variable that will increase until it is equal to the number of 
%pages in the Glycerol_R2 3D array
a=1;
while a<=size(Glycerol_R2, 3)
    %Subtract the smoothed absorbance value at 899.53 nm spectrum 'a' from
    %every smoothed absorbance value in spectrum 'a' and write values in to
    %column forur of the 3D array.
    Glycerol_R2(:,4,a)=(Glycerol_R2(:,3,a)-Glycerol_R2(912,3,a));    
    a=a+1;
end
%Clear temporary variables
clearvars a
\end{lstlisting}
\newpage
\subsection*{Script to Produce Difference Spectra}\label{App:MATLABSPEC_DIFFSSPEC}
\begin{lstlisting}
%Script to subtract initial spectrum in a series from every subsequent 
%spectrum.
%'a' is a variable that will increase until it is equal to the number of 
%pages in the Glycerol_R2 3D array
a=1;
while a <= size(Glycerol_R2, 3)
    %Subtract the first smoothed and normalised spectrum collected 
    %(in column 4) from every other smoothed and normailsed specrum in the
    %array
    Glycerol_R2(:,5,a)=Glycerol_R2(:,4,a)-Glycerol_R2(:,4,1);
    a=a+1;
end
%Clear temporary variables
clearvars a;
\end{lstlisting}
\newpage
\subsection*{Script Fit Two Gaussians to Difference Spectra}\label{App:MATLABSPEC_LSQ_GAUSS}
\subsubsection{Function to Least Square Fit Two Gaussians}
\begin{lstlisting}
%This Fit2Gaussians function can be called by a script to fit two
%Gaussians to a UV-Vis spectrum. 
function [fitresult, gof] = createFit(Ma150_1_0wav, Ma150_1_0abs)
%CREATEFIT(MA150_1_0WAV,MA150_1_0ABS)
%  Create a fit.
%
%  Data for 'untitled fit 1' fit:
%      X Input : Ma150_1_0wav
%      Y Output: Ma150_1_0abs
%  Output:
%      fitresult : a fit object representing the fit.
%      gof : structure with goodness-of fit info.
%
%  See also FIT, CFIT, SFIT.

%  Auto-generated by MATLAB on 26-Nov-2014 11:42:04  Edited since, see Matts lab book 3, 27/11/14.


%% Fit: 'untitled fit 1'.
[xData, yData] = prepareCurveData( Ma150_1_0wav, Ma150_1_0abs );

% Set up fittype and options.
ft = fittype( '(d/(e*(sqrt(pi/2))))*exp(-2*(x-f)*(x-f)/(e*e))+(h/(k*(sqrt(pi/2))))*exp(-2*(x-l)*(x-l)/(k*k))+n', 'independent', 'x', 'dependent', 'y' );
opts = fitoptions( 'Method', 'NonlinearLeastSquares' );
opts.Display = 'Off';
%Restrain upper and lower bounds for each of the variables
opts.Lower = [0 70 220 0 225 508 0];
opts.StartPoint = [2 23.4 220 1 233 510 0];
opts.Upper = [Inf 80 Inf 240 235 512 0];
% Fit model to data.
[fitresult, gof] = fit( xData, yData, ft, opts )

% Plot fit with data.
%figure( 'Name', 'Ma151-1' );
%h = plot( fitresult, xData, yData );
%legend( h, 'Ma150_1_0abs vs. Ma150_1_0wav', 'untitled fit 1', 'Location', 'NorthEast' );
% Label axes
%xlabel( 'Wavelength (nm)', 'Interpreter','none' );
%ylabel( 'Absorbance', 'Interpreter','none' );
%title( 'Ma151-1_0 fit of Gaussians on Exponential Background','Interpreter','none' )
%grid on
\end{lstlisting}

\subsubsection{Script to Least Square Fit Two Gaussians}
\begin{lstlisting}
%This script calls the Fit2Gaussians function to fit two Gaussians
%to all of the spectra in a time-resolved series of difference spectra.

a=1;
GaussCoeff_glycerol_r2(1:6,1:7,1:size(glycerol100r2, 3))=zeros;
while a<=size(glycerol100r2, 3);
    %while a<=100;    
    testx=glycerol100r2(1:982,1,a);
    testy=glycerol100r2(1:982,5,a);

    Fit2gaussians(testx, testy);
    %Writes coefficient parameters to Coeff 3d matrix line 1
    
    GaussCoeff_glycerol_r2(1,:,a)=coeffvalues(ans);
    
    %Writes 95% confidence lower and upper bounds to lines 2 and 3 of Coeff
    GaussCoeff_glycerol_r2(2:3,:,a)=confint(ans);
    
    %Converts confidence bounds to standard error and writes to row Coeff 4
    GaussCoeff_glycerol_r2(4,:,a)=(((GaussCoeff_glycerol_r2(3,:,a))-(GaussCoeff_glycerol_r2(2,:,a)))/3.92);
    
    %Makes row 5 the co-efficient minus the Standard Error and row 6 the 
    %coefficient plus the Standard Error
    GaussCoeff_glycerol_r2(5,:,a)=(GaussCoeff_glycerol_r2(1,:,a))-((GaussCoeff_glycerol_r2(4,:,a)));
    GaussCoeff_glycerol_r2(6,:,a)=(GaussCoeff_glycerol_r2(1,:,a))+((GaussCoeff_glycerol_r2(4,:,a)));
    %FitOutPut(a,:)=coeffvalues(ans);
a=a+1;
end;

figure
clearvars r
r(:,1)=GaussCoeff_glycerol_r2(1,4,:);
plot((1:size(GaussCoeff_glycerol_r2, 3))/2,r(:,1))
r(:,1)=GaussCoeff_glycerol_r2(1,1,:);
hold on
plot((1:size(GaussCoeff_glycerol_r2, 3))/2,r(:,1))
\end{lstlisting}

%\subsection*{Script to Perform Gaussian Mixture Modelling on UV-Vis Difference Spectra}\label{App:MATLAB_GMM}
%\begin{lstlisting}  
%%Script to convert UV-Vis spectrum to a 1 dimensional normal distribution 
%%of normal values.
%
%%The weighted average function may need to be in the same folder as this
%%script for the Gaussian Mixture Modelling to run correctly.
%
%%Modified the UV1D script in a few ways:
%
%%1) Added loop to go through all spectra in a 3D array
%%2) All gaussian parmeters are stored in a single array
%%3) Fit a scale factor between each set of gaussians and the difference 
%%   spectra. (Will be done in a subsequent script).  
%%4) Added a smarter way to decide when the EM algorithm has converged.
%
%a=2;
%Tolerance= 0.001;
%% Set 'k' to the number of clusters to find. AKA number of gaussians
%k = 2;
%mu(1 : k) = zeros;
%conv(1,1:k) = zeros; 
%
%c=1; %Counter for writing output matrix
%d=1; %Spectra increment
%%preallocate output matrices
%Gaussians_Glycerol_ALL(1:5,1:k,1:(floor((size(glycerol100r2,3)-a)/d))) = zeros;
%Glycerol_All(1:982,k+5,1:(floor((size(glycerol100r2,3)-a)/d)))=zeros;
%while a <= size(glycerol100r2, 3)
%
%%DIFFSPEC_IN contains the Wavelength values in column 1, Absorbance values
%%in column 2, Absorbance values multiplied by 1000 so that they're all
%%integer values in column 3. Column 4 contains the frequencies (cm-1) 
%%conversion of the wavelengths
%DIFFSPEC_IN(:,1)=glycerol100r2(1:982,1,a);
%DIFFSPEC_IN(:,2)=glycerol100r2(1:982,5,a);
%DIFFSPEC_IN(:,3)=glycerol100r2(1:982,5,a)*1000;
%%DIFFSPEC_IN(:,4)=(1./glycerol100r2(1:982,1,a))*10000000;
%
%%Convert spectrum to 1D distrubution (D1) of absorbance values, negative 
%%absorbance values treated as 0.
%
%auv=1;
%cuv=1;
%duv=0;
%
%while auv <= size(DIFFSPEC_IN, 1)
%    buv=DIFFSPEC_IN(auv,3)+duv;
%    D1(cuv:buv,1)=DIFFSPEC_IN(auv,1);
%    cuv=size(D1, 1) +1;
%    duv=size(D1, 1);
%
%    auv = auv + 1;
%end
%%Comment/uncomment to swith D1 between WL and WN
%%D1=(1./D1)*10000000;
%
%% Set 'm' to the number of data points.
%m = size(D1, 1);
%
%% Randomly select k data points to serve as the means.
%% indices = randperm(m);
%% mu = zeros(1, k);
%% for i = 1 : k
%%     mu(i) = D1(indices(i));
%% end
%
%%Place the gaussians at even intervals through the spectrum
%minD1 = min(D1);
%maxD1 = max(D1);
%D1diff = maxD1 - minD1;
%Interval = D1diff / k;
%
%for i = 1 : k
%    mu(i) = minD1 + (( ( i - 1 ) * Interval) + (0.5 * Interval));    
%end
%
%% Use the overal variance of the dataset as the initial variance 
%% for each cluster.
%sigma = ones(1, k) * sqrt(var(D1));
%
%% Assign equal prior probabilities to each cluster.
%phi = ones(1, k) * (1 / k);
%
%%%%%%%%%%%%%%%%%%%%%%%Run Expectation Maximization%%%%%%%%%%%%%%%%%%%%%%%%
%
%% W holds the probability that each data point belongs to each cluster.
%% One row per data point, one column per cluster.
%W = zeros(m, k);
%%For convergence vs co-incidence detector
%b=1; 
%% Loop until convergence.
%for iter = 1:10000   
%    fprintf('  EM Iteration %d\n', iter);
%    %% Calculate the probability for each data point for each distribution.
%     % Matrix to hold the pdf value for each every data point for every cluster.
%    % One row per data point, one column per cluster.
%    pdf = zeros(m, k);
%    
%    % For each cluster...
%    for j = 1 : k
%        
%        % Evaluate the Gaussian for all data points for cluster 'j'.
%        pdf(:, j) = gaussian1D(D1, mu(j), sigma(j));
%    end
%    
%    % Multiply each pdf value by the prior probability for each cluster.
%    %    pdf  [m  x  k]
%    %    phi  [1  x  k]   
%    %  pdf_w  [m  x  k]
%    pdf_w = bsxfun(@times, pdf, phi);   
%    % Divide the weighted probabilities by the sum of weighted probabilities for each cluster.
%    %   sum(pdf_w, 2) -- sum over the clusters.
%    W = bsxfun(@rdivide, pdf_w, sum(pdf_w, 2));
%    % Store the previous means so we can check for convergence.
%    prevMu = mu;    
%    
%    % For each of the clusters...
%    for j = 1 : k
%    
%        % Calculate the prior probability for cluster 'j'.
%        phi(j) = mean(W(:, j));
%        
%        % Calculate the new mean for cluster 'j' by taking the weighted
%        % average of *all* data points.
%        mu(j) = weightedAverage(W(:, j), D1);
%    
%        % Calculate the variance for cluster 'j' by taking the weighted
%        % average of the squared differences from the mean for all data
%        % points.
%        variance = weightedAverage(W(:, j), (D1 - mu(j)).^2);
%        % Calculate sigma by taking the square root of the variance.
%        sigma(j) = sqrt(variance);
%    end
%    
%    % Check for convergence. The 
%    for j = 1 : k
%       conv(1,j) = prevMu(j) - mu(j); 
%    end
%    conv(2,:) = sqrt(conv(1,:).^2);
%    convdiff(1,1) = sum(conv(2,:))/k;
%    if (convdiff > Tolerance)
%        b = 1;    
%    end
%    if (convdiff < Tolerance)
%        b = b + 1;
%    end
%    if b > 4
%        break
%    end
%% End of Expectation Maximization loop.   
%end    
%
%%Write Absorbance values and wavelengths of analysed spectra to a new
%%matrix. Then evaluate the Gaussians against the x-values
%Glycerol_All(:,1,c)=DIFFSPEC_IN(:,1);
%Glycerol_All(:,2,c)=DIFFSPEC_IN(:,2);
%Glycerol_All(:,3,c)=(1./DIFFSPEC_IN(:,1))*10000000;
%
%%Evaluation of the Guassians, detect if D1 is in WL or WN automatically
%%by seeing if dmax is greater than 1000.
%if max(D1) < 1200
%for j = 1 : k
%    Glycerol_All(:,j+3,c)=phi(j)*(gaussian1D(Glycerol_All(:,1,c),mu(j),sigma(j)));
%end
%else
%for j = 1 : k
%    Glycerol_All(:,j+3,c)=phi(j)*(gaussian1D(Glycerol_All(:,3,c),mu(j),sigma(j)));
%end    
%end
%Glycerol_All(:,k+4,c)=sum(Glycerol_All(:,4:k+3,c),2);
%
%%Fit a scale factor between the GMM and the actual data 
%scale_gauss = fminunc(@(e) sqrError(e,Glycerol_All(:,2,c), Glycerol_All(:,k+4,c)),1);
%%Store the variables describing the Gaussian distributions for each
%%spectrum. 
%for j = 1 : k
%       Gaussians_Glycerol_ALL(1,j,c) = mu(j); 
%       Gaussians_Glycerol_ALL(2,j,c) = sigma(j);
%       Gaussians_Glycerol_ALL(3,j,c) = phi(j); 
%       Gaussians_Glycerol_ALL(4,j,c) = scale_gauss;
%       Gaussians_Glycerol_ALL(5,j,c) = scale_gauss*phi(j);
%end
%
%Glycerol_All(:,k+5,c)=(sum(Glycerol_All(:,4:k+3,c),2))*scale_gauss;
%  c=c+1;
%  clearvars  auv cuv duv D1diff convdiff Interval iter j m mu prevMu sigma W phi D1
%  a=a+d;
%end
%clearvars a  minD1 maxD1  Tolerance variance ans b buv conv DIFFSPEC_IN i indices d a c
%\end{lstlisting}

\newpage
\subsection*{Performing Rayleigh Scattering Background Correction}\label{App:MATLABSPEC_RAYLEIGH}

\subsubsection{Rayleigh Fit Function}
 \begin{lstlisting} 
 %This function is called by the Rayleigh Fit Script to perform the 
 %actual fitting.
 
 %Read in array "WL" as xData and array "Abs" as yData
[xData, yData] = prepareCurveData( WL, Abs );

% Set up fittype and options.
%Fit a polynomial to the data using a non linear least squares method.
ft = fittype( 'power2' );
opts = fitoptions( 'Method', 'NonlinearLeastSquares' );

%Variable 'a' is allowed to float, corresponds to c1 in Equation 2.2 
%Power (variable 'b') is fixed as -4, as in Rayleigh Equation 2.2
%Variable 'c' is allowed to float give the linear offset of the curve on 
%the y-axis. Variable 'c' corresponds to c2 in Equation 2.2.

opts.Display = 'Off';
opts.Lower = [-Inf -4 -Inf];
opts.StartPoint = [1 -4 0];
opts.Upper = [Inf -4 Inf];

% Fit model to data.

[fitresult, gof] = fit( xData, yData, ft, opts );
 \end{lstlisting}  
 \newpage
\subsubsection{Rayleigh Fit Script}
\begin{lstlisting}
%This script will call the Rayleigh Fit Function to fit the Rayleigh 
%equation to a region of the spectrum where only background scattering
%is expected, evaluate the equation across the entire spectrum and 
%substract the contribution from Rayleigh scattering from all observed
%absorbance values  

%'y' is a variable that will increase until it is equal to the number of 
%pages in the WT_I320_1 array
y=1;

v=1; %Upper wavelength bound (1 is 900nm)
w=301; %Lower wavelength bound (301 is 600nm)

while y<=size(WT_I320_1, 3)

%Write one matrix for the wavelength range to fit the Rayleigh equation to 
%and one matrix for the absorbance values.

	WL=WT_I320_1(v:w,1,y);
	Abs=WT_I320_1(v:w,2,y);

%Call Rayleigh Fit function to fit the absorbance data

	Rayleigh_Fit(WL,Abs);
	
%Save the fitted co-efficients as variables a, b and c. 
	
	Coeff=coeffvalues(ans);
	a=Coeff(1,1);
	b=Coeff(1,2);
	c=Coeff(1,3);
	
%Evaluate the curve describing the contribution of Rayleigh Scattering
%to the observed absorbance across the entire spectrum and write to 
%column 3 of 'WT_I320_1'  

	WT_I320_1(:,3,y)=((a*(WT_I320_1(:,1,1).^b))+c);

%Subtract the calculated Rayleigh scattering contribution from the 
%observed absorbance across the entire wavelength range and write
%to column 4.

	WT_I320_1(:,4,y)=WT_I320_1(:,2,y)-WT_I320_1(:,3,y);
	
	y=(y+1);
	
end 
\end{lstlisting} 
\newpage
\subsection*{Calculate Initial Rate of Enzyme Activity}\label{App:Enzyme_Activity}
\begin{lstlisting}
%Script to convert gradient from linear fit of spectroscopy data to an 
%enzyme activity in nanomoles per milligram of protein per minute.
%%%%%%%%%%%%%%%%%%%%%%%%%%%USER INPUT%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%Tell me the extinction co-efficient of the chromophore being monitored
%I320 =16200
%PLP=5380
ExtinctionCoeff=16200;
%Tell me the protein concentration in the assay (micromolar) This is
%reduced from 20 micromolar in the I320 assay to 16 micromolar in the PLP
%assay due to addition of G3P
ProteinConc=20;
%What is the molecular weight of the protein (Daltons)
MW=33216;
%Tell me what the assay volume was (microlitres). 300 microlitres in I320,
%375 in PLP assay
AssayVolume=300;
%%%%%%%%%%%%%%%%%%%%%%%%%%%%END OF USER INPUT%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%First order polynomial (linear) fit to data in y1 to x1 independent 
%variable.
p= polyfit (x1,y1,1);
absec=p(1,1);
abmin=absec*60;

%The absorbance change per minute divided by the the extinction
%co-efficient gives the change in the molar concentration of the
%chromophore per minute
ConcChangeMin=abmin/ExtinctionCoeff;

%(Moles = conc * vol), change assay volume to litres  
AssayVolume(1,2)=(AssayVolume(1,1)*(10^-6));

%Calculate number of moles of chromophore produced per minute
MolesperMin=ConcChangeMin*AssayVolume(1,2);

%Convert protein concentration in micromolar to mg per ml
ProteinConc(1,2)=(ProteinConc(1,1))*10^-6; %Molar
ProteinConc(1,3)=(ProteinConc(1,2))*MW; %mgperml or g/l
ProteinConc(1,4)=(ProteinConc(1,3))*AssayVolume(1,2); %Grams of protein  
                                                      %per assay
ProteinConc(1,5)=(ProteinConc(1,4)*1000); %mg of protein per assay
%Divide the moles of choromophore produced in the assay by the mg of
%protein in the assay to get the enzyme activity in terms of
%nanomoles of chromophore produced per mg of protein per minute.
%Multiplying by 10^9 converts from moles to nanomoles.

%Determine the r-square for the fit
yfit = polyval(p,x1);
yresid = y1 - yfit;
SSresid = sum(yresid.^2);
SStotal = (length(y1)-1) * var(y1);
rsq = 1 - SSresid/SStotal;

Activity=((MolesperMin/ProteinConc(1,5))*(10^9));
%Clear up variables generated by script
clearvars p abmin absec AssayVolume ConcChangeMin 
clearvars ExtinctionCoeff MolesperMin MW ProteinConc
clearvars yfit yresid SSresid SStotal
\end{lstlisting}