\chapter{Introduction}\label{ch:Intro}
\pagenumbering{arabic}
Enzymes are biological catalysts that accelerate the rate of the biochemical reactions that are essential for life to occur. At the cellular level enzymes play vital roles in metabolism, signalling pathways, cellular replication and cell death. The importance of enzymes is reflected by the fact that mutation of a single amino acid in an enzyme can have a profoundly detrimental effect on the health of an organism. An improved understanding of how enzymes perform their catalytic functions and how their activity is regulated is therefore of significant biomedical interest. 

The ability of enzymes to increase the rate of specific chemical reactions has also been applied to the industrial production of food, textiles, chemicals and many other products. The advantages of using enzymes rather than chemical catalysts include improved reaction specificity, the ability to catalyse reactions close to room temperature, which reduces energy usage, and that they are biodegradable. However, a significant effort is often necessary to improve the stability of an enzyme or to adapt naturally occurring enzymes to perform reactions that no known enzyme catalyses. Rational engineering of enzymes is dependent on knowledge of their structure and how that structure can be altered to improve catalytic performance. %examples    http://www.sciencedirect.com/science/article/pii/S095965261200594X

X-ray crystallography is a technique that makes it possible to determine the three-dimensional structure of an enzyme in atomic detail. The location of substrate binding sites, the relative positions of catalytic residues and the degree of mobility in particular regions of the protein can all be determined from a crystal structure and used to understand how the protein performs its function. Given that enzymes are flexible molecules that often undergo significant conformational changes as a result of substrate binding and during catalysis, it is often necessary to crystallise and solve the structure of the protein in several intermediate states to gain a complete understanding of how an enzyme catalyses its reaction. 

One of the drawbacks of X-ray crystallography is that the X-rays used during data collection have a damaging effect on protein crystals. This damage can cause a change in the structure of the protein, possibly leading to a misinterpretation of the protein structure and mechanism by which it performs its function. Understanding the reactions that take place in protein crystals during X-ray irradiation allow the investigator to assess whether their crystallography data may be affected by radiation damage.

The primary aim of this thesis has been to understand how the pyridoxal 5'-phosphate (PLP) synthase enzyme complex catalyses the biosynthesis of PLP by trapping the enzyme in several intermediate states and determining their structure using X-ray crystallography. This chapter provides an introduction to the biological roles of PLP and the pathways for PLP biosynthesis. UV-Vis spectra were collected from the protein crystals to monitor the effects irradiation on the crystals during data collection. This chapter also introduces the X-ray diffraction experiment and explains some of the damaging effects that X-rays can have on protein crystals during collection of diffraction data.      
\newpage
\subsection{Pyridoxal 5'-phosphate (PLP)}
Enzymes often require organic and inorganic cofactors to perform their function. Enzyme cofactors must either be absorbed from the external environment or biosynthesised within the cell; organic cofactors that must be absorbed from the external environment are referred to as vitamins.   

Pyridoxal 5'-phosphate (PLP) is one of six related compounds referred to as `vitamers' of vitamin B6, a vitamer is any compound that may be supplemented to the diet of an organism to alleviate the symptoms of vitamin deficiency \cite{Burk1943}. Humans and other animals must absorb vitamin B6 through their diet while plants and microorganisms are capable of synthesising vitamin B6 \textit{de novo}. Pyridoxal 5'-phosphate is used by all cellular organisms, functioning as an enzyme cofactor and antioxidant. The importance of PLP as an enzyme cofactor is highlighted by the fact that 1.5\% of prokaryotic genes encode PLP-dependent enzymes, and 4\% of all classified enzymes are PLP dependent \cite{Percudani2003}. \par 
\begin{figure}[!htbp]
\begin{minipage}{\linewidth}
	\makebox[\linewidth]{
	\includegraphics[width=12cm, height=8cm, keepaspectratio]{/Users/matt/Dropbox/ThesisPrep/Thesis/fig/Vitamers/SixVitamers3.pdf}}	

	\captionof{figure}[Chemical structures of vitamin B6 vitamers]{Above, left to right: Pyridoxine (PN\glsadd{PN}), Pyridoxamine (PM\glsadd{PM}), Pyridoxal (PL\glsadd{PL}). Below, left to right: Pyridoxine 5'-phosphate (PNP\glsadd{PNP}), Pyridoxamine 5'-phosphate (PMP\glsadd{PMP}), Pyridoxal 5'-phosphate (PLP).\label{fig:vitamers}}	
\end{minipage}
\end{figure}
As shown in Figure \ref{fig:vitamers}, all six vitamers of vitamin B6 have a pyridine ring, differing groups attached to the 4' carbon and exist in both phosphorylated and non-phosphorylated forms. The reason that supplementing the diet with any of the vitamers alleviates symptoms of deficiency is that each of the vitamers can be converted to any of the other vitamers by salvage pathways. In mammals, only the non-phosphorylated vitamers are absorbed by cells \cite{Clayton2006}. The non-phosphorylated vitamers are substrates of pyridoxal kinase enzymes which catalyse the transfer of a phosphate group from ATP\glsadd{ATP} to pyridoxal, pyridoxamine or pyridoxine to convert it to the phosphorylated form \cite{diSalvo2004}.%PLP is covalently bound to the lysine residue of albumin in the bloodstream, once the PLP is released it is dephosphorylated by membrane associated phosphorylases and it can diffuse across the cell membrane. After entering the cell it is phosphorylated to the active form by pyridoxal kinase.  

While PLP is the vitamer most commonly used as an enzyme cofactor, all of the vitamers have a utility within the cell. As is described in Section \ref{sec:anti-oxidant} all of the vitamers function as antioxidants \cite{Bilski2000}. The PNP/PMP oxidase enzyme converts pyridoxine 5'-phosphate and pyridoxamine 5'-phosphate to pyridoxal 5'-phosphate \cite{diSalvo2011,Zhao1995}. As well as PLP, PMP is also utilised as an enzyme cofactor by some enzymes, such as CDP-6-deoxy-L-\textit{threo}-D-\textit{glycero}-4-hexulose-3-dehydrase is the biosynthetic pathway for the sugar ascarylose \cite{Burns1996}.

The importance of the vitamin B6 salvage pathways to health is highlighted by the effects of inborn errors of metabolism in the enzymes that interconvert the vitamers. Mutations in the non-specific alkaline phosphatase enzyme cause accumulation of PLP in the blood that cannot cross the blood-brain barrier. The reduced concentration of PLP in the brain deprives PLP dependent enzymes of the cofactor; this interferes with synthesis of neurotransmitters such as GABA ($\gamma$-amino butyric acid), leading to seizures \cite{Waymire1995}. Similarly, mutations in the  human PNP/PMP oxidase gene results in seizures that can be treated by supplementation of PLP \cite{Mills2014}. 

The vitamin B6 salvage pathways ensure that regardless of the vitamer that is absorbed by the cell it can be converted to the vitamer of greatest use, usually PLP.\par


\subsection{Pyridoxal 5'-Phosphate as an Enzyme Cofactor}
The use of PLP as an enzymatic cofactor is conserved throughout the three domains of life; PLP dependent enzymes primarily catalyse reactions required for amino acid metabolism including decarboxylation, elimination and transamination reactions \cite{Percudani2009}. The Enzyme Commission classifies all enzymes into one of six enzyme families depending on the reaction that the enzyme catalyses \cite{Percudani2003}. Five of the six enzyme families contain PLP dependent enzymes, highlighting the versatility of PLP as an enzyme cofactor. PLP dependent enzymes belonging to four of the enzyme families catalyse amino acid based reactions while enzymes in a fifth family of enzymes function as glycogen phosphorylases \cite{Percudani2009}. \par
 
Cofactor-dependent enzymes are referred to as holoenzymes when the cofactor is bound in the active site and as apoenzymes in the absence of the cofactor. For PLP dependent enzymes the apoenzyme typically has no catalytic activity, while isolated PLP may catalyse a particular reaction, although with a lower reaction specificity and rate than the holoenzyme. 

The PLP dependent enzymes that catalyse reactions with amino acid substrates share a common mechanism that has been described by the Dunathan stereo-electronic hypothesis \cite{Dunathan1966}. In the native state, the 4' carbon of PLP is covalently bound to $\varepsilon$-NH$_2$ of a lysine side chain in the PLP dependent enzyme; this complex is termed the internal aldimine (Figure \ref{fig:transimination}). Binding of the amino acid substrate leads to cleavage of the internal aldimine and formation of a covalent bond between 4' carbon of PLP and the amino group of the substrate; this complex is the external aldimine (Figure \ref{fig:transimination}).

%The role of the protein is to provide a scaffold that limits the number of conformations that substrates may bind in thereby reducing the number of off-pathway side reactions that may occur and improving reaction specificity. The protein also enhances the rate of the reaction by stabilising transition states during catalysis. 



\begin{figure}[!htbp]
\begin{minipage}{\linewidth}
	\makebox[\linewidth]{
	\includegraphics[width=8cm, height=8cm, keepaspectratio]{/Users/matt/Dropbox/ThesisPrep/Thesis/fig/transimination/Transimination.png}}	

	\captionof{figure}[Structure of the Internal and External Aldimines]{The chemical structure of the internal aldimine (left), the 4' carbon of PLP is covalently bound to the $\varepsilon$-NH$_2$ of a lysine side chain (green) in the enzyme active site in the absence of substrate. Substrate binding causes a transimination reaction resulting in the release of PLP from the side chain of the protein and formation of the external aldimine (right), the 4' carbon is covalently bound to the amino group of the substrate (A, orange). Scheme derived from Cerqueria \textit{et al} \cite{Cerqueria2011}.\label{fig:transimination}}	
\end{minipage}    
\end{figure}

The first step in all PLP catalysed reactions involving amino acids is the transimination reaction. After this step, the reaction pathway is controlled by the orientation of the substrate relative to the plane of the PLP ring, which is in turn determined by the shape of the active site. The pyridine ring of PLP has a delocalised $\pi$ system which extends to the amino group of the substrate once the external aldimine is formed (Figure \ref{fig:transimination}). The Dunathan stereo-electronic hypothesis states that after formation of the external aldimine, the next step in the reaction is cleavage of the bond from the C$\alpha$ atom that is perpendicular to the plane of the pyridine ring, and therefore parallel to the p-orbitals of the delocalised system (Figure \ref{fig:dunathan}) \cite{Dunathan1966}. By controlling the orientation that the substrates bind in the active site enzymes using PLP can ensure reaction specificity. After the cleavage of the first bond, a negative charge is present on the C$\alpha$ atom, which is stabilised by the delocalised system on the pyridine ring at this point the reaction mechanisms of the various PLP-dependent enzymes diverge \cite{Toney2011,Percudani2009}. Although the hypothesis was proposed before the structure of any PLP dependent enzyme was solved, all structures solved since have supported the hypothesis \cite{Toney2011}.      

\begin{figure}[!htbp]
\begin{minipage}{\linewidth}
	\makebox[\linewidth]{
	\includegraphics[width=16cm, height=8cm, keepaspectratio]{/Users/matt/Dropbox/ThesisPrep/Thesis/fig/dunathan/Dunathan.png}}	

	\captionof{figure}[Dunathan Stereo-electronic Hypothesis]{The pyridine ring of PLP has a delocalised $\pi$ system that extends to the amino group of the substrate (orange) in the external aldimine complex. The bond between the C$\alpha$ and the group that is perpendicular to the plane of the pyridine ring, and therefore parallel to the p-orbitals is always broken as the first step of the reaction. The three examples above show the first step in a decarboxylation reaction (left), a racemization reaction (centre) and a C$\alpha$-C$\beta$ bond cleavage (right).\label{fig:dunathan}}	
\end{minipage} 
\end{figure}
\newpage
\subsection{Pyridoxal 5'-Phosphate as an Anti-Oxidant}\label{sec:anti-oxidant}
As mentioned previously, in addition to their catalytic function, the vitamers of vitamin B6 play a significant role in removing toxic reactive oxygen species (ROS) from the cell. \textit{Cercospora nicotianae}~ is a species of fungal plant parasite that has evolved to synthesise the chemical cercosporin which catalyses the generation of ROS when exposed to light. The fungus secretes cercosporin and the ROS that are produced damage plant cell membranes, causing the release of nutrients that the parasitic fungi may consume \cite{Daub1983}. The resistance of \textit{Cercospora nicotianae}~to singlet oxygen generated by cercosporin was shown to be lost when the SOR1\glsadd{SOR1} gene was knocked out, this gene was later found to be responsible for biosynthesis of PLP and re-named Pdx1 \cite{Ehrenshaft1998}.\par

ROS\glsadd{ROS} such as singlet oxygen (O$_2$), hydrogen peroxide (H$_2$O$_2$), superoxide radicals (\ch{O_2^{-.}}) and hydroxyl radicals (\ch{^{.}OH}) can react with biological molecules including protein, DNA and lipids causing damage to cells. \par
Macrophages and neutrophils of the innate immune system deliberately generate ROS using the enzyme NADPH (nicotinamide adenine dinucleotide phosphate reduced) oxidase as a means of degrading encapsulated bacteria \cite{Forman2002}. Radiolysis of singlet oxygen present in cells, the interaction of singlet oxygen with proteins with bound metal ions in their reduced forms and transfer of an electron from molecules carrying electrons in the process of oxidative phosphorylation (ubiquinol) to singlet oxygen all also produce ROS \cite{Bartosz2003}. The damage that reactive oxygen species can cause to macromolecules in the cell has been linked to the development of atherosclerosis and cancer in humans, and several mechanisms have evolved to detoxify cells of ROS \cite{Diplock1997}. 

Vitamins A ($\alpha$/$\beta$ carotene), C (ascorbic acid) and E ($\alpha$-tocopherol) have long been understood to have antioxidant activity \cite{Miller1996,Diplock1997,Burton1982,Traber2011}. However, the antioxidant activity of PLP only emerged recently after the discovery of the role it plays in resistance of \textit{Cercospora nicotianae} to high concentrations of singlet oxygen \cite{Ehrenshaft1998}. The antioxidants in the cell can be divided into two categories, lipid soluble and water soluble. Vitamins A and E are lipid soluble and are localised to the membranes of the cell to prevent lipid peroxidation \cite{Asensi2010}. In the leaves of \textit{Arabidopsis thaliana} plants, vitamin C tends to be the soluble antioxidant present at the highest concentrations ($\sim$20 \si{\micro\gram\per\centi\metre\squared}) while vitamin B6 is present at lower concentrations ($\sim$2 \si{\micro\gram\per\centi\metre\squared}) \cite{Triantaphylides2009}. Vitamins B6 and C have different specificities, vitamin B6 is a relatively potent quencher of singlet oxygen while vitamin C is a strong quencher of free radicals \cite{Ehrenshaft1999a,Triantaphylides2009,Asensi2010}.                           

\textit{In vitro}~ studies have shown that PLP acts as an antioxidant by reacting with singlet oxygen, all of the vitamin B6 vitamers displayed similar efficacies as antioxidants suggesting that the pyridine ring is responsible for their antioxidant qualities \cite{Bilski2000}. Within plant cells the Pdx1 protein is primarily localised in association with the cell membrane and organelle membranes; knockout of Pdx1 in \textit{A. thaliana} increased the susceptibility of the lipid membranes to damage by ROS \cite{Chen2005}. Pyridoxine and pyridoxamine have also both been shown to have a protective effect against lipid peroxidation by the superoxide radical (\ch{O_2^{-.}}) in erythrocytes \cite{Jain2001}.

The observation that protozoa, yeast and plants are all observed to increase transcription of genes responsible for PLP biosynthesis when exposed to stress likely to cause an increase in concentrations of ROS supports the theory that vitamin B6 is an important antioxidant in most eukaryotic organisms \cite{Knockel2012,Chen2003,Denslow2005}. However, PLP is also required for the biosynthesis of glutathione, an important antioxidant \cite{Denslow2005}. It is, therefore, difficult to separate the effect of PLP directly scavenging ROS, from its function as a cofactor in the biosynthesis of other antioxidants, when evaluating its role in the oxidative stress response.  
\newpage
\subsection{PLP Synthase as a Drug Target}
The absence of a PLP biosynthetic pathway in humans has lead to interest in the development of inhibitors of PLP synthase with the aim of developing therapeutics against pathogens such as \textit{Mycobacterium tuberculosis} and \textit{Plasmodium falciparum} \cite{Kronenburger2013}. 

Knockout of Pdx1 in \textit{M. tuberculosis} results in the pathogen becoming auxotrophic for vitamin B6 in culture and unable to survive in \textit{in vivo} mouse models \cite{Dick2010}. While this demonstrates the potential for inhibiting the enzyme pathway as a means of  developing an anti-tuberculosis drug, several barriers prevent this. Any inhibitor must be selective for Pdx1 and not affect the function of human proteins; this needs to be considered when developing competitive inhibitors of ribose 5-phosphate and glyceraldehyde 3-phosphate binding to Pdx1 as both compounds are intermediates in the process of respiration. An inhibitor must also be able to cross the two permeability barriers between the cytosol of the pathogen and the surrounding environment. The two barriers being the cell membrane  and the hydrophobic cell wall of \textit{M. tuberculosis} that is formed from a complex of covalently linked peptidoglycan, arabinogalactan and mycolic acid \cite{Niederweis2008}.

The potential to target PLP biosynthesis in \textit{Plasmodium falciparum} has also been investigated \cite{Kronenburger2013}. Part of the \textit{P. falciparum} life cycle is spent within red blood cells where they digest up to 80\% of the available haemoglobin. The amino acids released by this process may be used for protein synthesis; it has also been suggested that haemoglobin degradation is necessary to create space for the replicating parasite without lysing the cell membrane of the erythrocyte \cite{Lew2004}. The digestion of haemoglobin leads to the release of free heme which catalyses the generation of reactive oxygen species including singlet oxygen and hydrogen peroxide \cite{Atamna1993}. These reactive oxygen species increase the oxidative stress within the parasite; inhibiting biosynthesis of PLP, an anti-oxidant may, therefore, reduce the ability of the parasite to deal with oxidative stress and impair its ability to proliferate.\par 

Inhibitors of the \textit{Plasmodium} Pdx1 protein have been designed using homology modelling and \textit{in silico} docking \cite{Reeksting2012}. \textit{In silico} docking identified that the sugar erythrose 4-phosphate (E4P) may bind in the Pdx1 P1 site leading to the screening of E4P\glsadd{E4P} analogues to identify a Pdx1 inhibitor. The analogue 4PEHz\glsadd{4PEHz} (4-phospho-D-erythronohydrazide) had an IC$_{50}$ of 43 \si{\micro\molar}~ \textit{in vitro}, and when the parasite was grown in culture in the presence of 1 \si{\micro\molar}~ there was a reduction in proliferation \cite{Reeksting2012}. This effect was not seen in parasites complemented with extra copies of the Pdx1 and Pdx2 genes \cite{Reeksting2012}. The authors state that further development of the lead compounds is required to increase the potency. It is also unclear if off-target effects would occur when using E4P analogues as a Pdx1 inhibitor in humans as E4P is an intermediate in the pentose phosphate pathway. Before the further development of E4P analogues to improve the strength of binding it may be informative to perform \textit{in vitro} tests to determine their effect on the enzymes of the pentose phosphate pathway.

Aspects of the PLP synthase mechanism that may be targetted but have not yet been investigated include inhibiting the formation of the Pdx1/Pdx2 interface, blocking the site of product formation and inhibition of product release. Given that our knowledge of the reaction mechanism is incomplete, a structural characterisation of the enzyme in different intermediate states may yield new strategies for inhibiting Pdx1.          

\newpage
\section{Biosynthesis of Pyridoxal 5'-Phosphate}
Two distinct and mutually exclusive routes for the \textit{de novo} bio-synthesis of vitamin B6 have been discovered. The 1-Deoxy-D-xylulose-5-phosphate (DXP)\glsadd{DXP} dependent pathway is utilised by several $\gamma$-proteobacteria, including \textit{E. coli}, and is formed from six separate enzymes \cite{Mittenhuber2001}. 

The second to be discovered is dependent on ribose 5-phosphate and is well conserved throughout plants, archaea and most bacteria \cite{Mittenhuber2001}. The Singlet Oxygen Resistance (SOR1) gene had been identified as being responsible for the resistance of \textit{Cercospora nicotianae}~ to compounds that generate high concentrations of singlet oxygen, which causes damage to lipids, proteins and DNA resulting in cell death (Section \ref{sec:anti-oxidant}) \cite{Ehrenshaft1998}. A subsequent investigation identified five \textit{C. nicotianae} mutants that were sensitive to cercosporin \cite{Ehrenshaft1999a}. Complementing the mutants with the SOR1 gene restored cercosporin resistance; however, two mutants remained sensitive to cercosporin suggesting, that a second gene may be required for cercosporin resistance \cite{Ehrenshaft1999a}. 

The \textit{Saccharomyces cerevisiae} SOR1 homologue Snz1 was shown to interact with a second protein named Sno1 using a yeast two-hybrid assay \cite{Padilla1998}. Sequencing of the two \textit{C. nicotianae} mutants that remained susceptible to cercosporin after complementation with the Pdx1 gene showed that the mutations occurred in the \textit{C. nicotianae} homologue of the Sno1 gene which was re-named Pdx2 \cite{Ehrenshaft2001}. Complementing these mutants with the wild-type Pdx2 gene enabled PLP biosynthesis \cite{Ehrenshaft2001}. \par

The substrates for Pdx1 dependent biosynthesis were identified using isotopic labelling experiments. Isotopic labelling of  \ce{^{15}NH4Cl} in the growth media of \textit{Saccharomyces cerevisiae} revealed that the nitrogen atom of pyridoxine could be incorporated from ammonia and that the ammonia was produced by glutamine hydrolysis \cite{Tazuya1995}. The same study showed that this was not observed in \ecoli and suggested that the two species use different pathways for biosynthesis of pyridoxine \cite{Tazuya1995}. Isotopic labelling of potential metabolic precursors of PLP was used to identify that carbons 2, 2', 3, 4 and 4' originated from a single pentose and that carbons 5, 5', 6 of PLP originated from a triose, likely to be glyceraldehyde 3-phosphate \cite{Zeidler2003,Gupta2001}.   

Biosynthesis of PLP has been reconstituted \textit{in vitro} in the presence of Pdx1 using R5P, glutamine and G3P and Pdx2 \cite{Burns2005,Hanes2008a}. Pdx2 performs the glutamine hydrolysis reaction, and the ammonia is passed to Pdx1, which combines R5P, G3P and ammonia to form PLP (Figure \ref{fig:reaction_scheme}). 
\par 

\begin{figure}[!htbp]
\begin{minipage}{\linewidth}
	\makebox[\linewidth]{
		
	\includegraphics[width=13cm, height=7cm, keepaspectratio]{/Users/matt/Dropbox/ThesisPrep/Thesis/fig/reactionsceme/150818Fig1a.png}}	

	\captionof{figure}[Outline of the PLP Synthase Reaction Scheme]{The reaction scheme of PLP synthase, Pdx2 hydrolyses glutamine to glutamate and releases ammonia which is channelled to Pdx1 where it reacts with ribose 5-phosphate and glyceraldehyde 3-phosphate to form pyridoxal 5'-phosphate.\label{fig:reaction_scheme}}	
\end{minipage}
\end{figure}

\clearpage

\subsection{The PLP Synthase Enzyme Complex}
%See /Users/matt/Dropbox/ThesisPrep/notes/PLPSynthase.txt for notes
%The observation that Pdx1 and Pdx2 interact was confirmed in a pull-down assay showing that non-tagged bacterial Pdx2 would co-purify with His-tagged Pdx1 on a nickel affinity chromatography column\cite{Belitsky2004}. In addition to the Pdx1/Pdx2 interaction, analytical ultracentrifugation of purified Pdx1 showed that Pdx1 oligomerised into hexamers and dodecamers in solution\cite{Zhu2005,Strohmeier2006}. Crystal structures of Pdx1 showed that the dodecameric complex was formed from two interlocking hexameric rings\cite{Zhu2005}. 

The PLP synthase enzyme complex is formed from twelve Pdx1 subunits arranged as two hexameric rings; a single Pdx2 subunit transiently binds to each Pdx1 subunit to form the complete 24 subunit complex \cite{Strohmeier2006}. Figure \ref{fig:Dodecamer} shows the crystal structure of the PLP synthase complex for \textit{Bacillus subtilis}. The PLP synthase complex is classified as a glutamine amidotransferase; the Pdx2 subunit hydrolyses glutamine releasing ammonia which is passed to Pdx1 where it reacts with R5P and G3P to form PLP (Figure \ref{fig:reaction_scheme}). Although the Pdx1/Pdx2 interaction is transient the 24-subunit complex was stabilised by mutating Pdx2 to inactivate its glutaminase activity, this allowed the structure of the complete PLP synthase enzyme complex to be solved using X-ray crystallography (Figure \ref{fig:Dodecamer}) \cite{Strohmeier2006}. Interaction of Pdx2 with Pdx1 has been shown to be essential for the catalytic activity of Pdx2; Pdx1 also depends on interaction with Pdx2 to function, unless ammonia is present in the environment \cite{Raschle2005}.   

\begin{figure}[!htbp]	
\begin{minipage}{\linewidth}
	\makebox[\linewidth]{
		
	\includegraphics[width=13cm, height=7cm, keepaspectratio]{/Users/matt/Dropbox/ThesisPrep/Thesis/fig/2NV2/dodecamer.png}}	

	\captionof{figure}[Structure of the PLP synthase enzyme complex]{Left: Structure of the \textit{Bacillus subtilis} PLP synthase enzyme complex, Pdx1 subunits blue, Pdx2 subunits white\cite{Strohmeier2006}. Six Pdx1 subunits are visible with Pdx2 subunits bound in a 1:1 ratio. Right: Structure of the PLP synthase enzyme complex rotated 90 degrees to show the interface between the interlocking Pdx1 hexameric rings.\label{fig:Dodecamer}}	
\end{minipage} 
\end{figure}

Pdx1 has been observed to exist in a dynamic equilibrium between the hexameric and dodecameric states using analytical ultracentrifugation \cite{Strohmeier2006,Zhu2005}. Pdx1 is always observed as a dodecamer when crystallised, except the \textit{Saccharomyces cerevisiae} protein \textit{Sc}Pdx1.1, which is observed as a hexamer in solution \cite{Neuwirth2009}. \textit{Sc}Pdx1.1 is capable of synthesising PLP suggesting that dodecamer formation is not essential for interaction with Pdx2 \cite{Neuwirth2009}.  	

The Pdx1 subunit has a \TIM~ barrel fold with eight alpha helices alternating with eight parallel beta strands(Figure \ref{fig:TIMTopology}). More than 10\% of proteins contain a domain with a \TIM~ barrel fold and \TIM ~barrel enzymes catalyse a diverse range of reactions \cite{Sterner2005}. Residues contributing to the active site in \TIM~ barrel enzymes tend to be located at the C-termini of the $\beta$ strands and on the $\beta\alpha$ loops while residues at the N-termini and on the $\alpha\beta$ loops contribute to the stability of the protein \cite{Sterner2005}. 

\begin{figure}[!htbp]
\begin{minipage}{\linewidth}
	\makebox[\linewidth]{
		
	\includegraphics[width=18cm, height=7cm, keepaspectratio]{/Users/matt/Dropbox/ThesisPrep/Thesis/fig/2NV2/TIMBarrelToppology3.png}}	

	\captionof{figure}[Topology of the Pdx1 \TIM~ Barrel]{The secondary structure of Pdx1.\cite{Zein2006,Strohmeier2006}. \label{fig:TIMTopology}}	
\end{minipage}  
\end{figure}

The Pdx1 protein has two phosphate binding sites (P1 and P2) that have been suggested as binding sites for the phosphorylated substrates, R5P and G3P, and the product, PLP (Figure \ref{fig:TMPdx1}) \cite{Zein2006}. Residues contributing to both sites have been assigned as participating in catalysis as a result of mutagenesis studies \cite{Moccand2011}. The P1 site of Pdx1 is located in a similar to position to the phosphate binding site in the prototypical \TIM~ barrel protein, Triose Phosphate Isomerase, at the $\beta\alpha$ end of the barrel \cite{Zhu2005,Noble1991}. The P2 site of Pdx1 is formed by residues on helix $\alpha$5 and the $\beta$5-$\alpha$5, $\beta$6-$\alpha$6 and $\beta$7-$\alpha$7 loops which are located at the interior of the dodecameric cylinder and close to the interface between the hexameric rings \cite{Zein2006}.   

Each Pdx1 subunit has a total surface area of $\sim$ 11,490 \si{\angstrom}$^2$ with 1909 \si{\angstrom}$^2$ buried in hydrophobic contacts with the adjacent subunits in the hexamer and 1255 \si{\angstrom}$^2$ forming the interface with the opposing hexamer \cite{Zhu2005}. In addition to the eight $\alpha$-helices and $\beta$-sheets that form the \TIM~ barrel Pdx1 contains additional secondary structures in the $\alpha\beta$ and $\beta\alpha$ loops (Figure \ref{fig:TIMTopology}). Helices $\alpha$6, $\alpha$6' and $\alpha$6'' on each Pdx1 subunit interact with the same region on the Pdx1 subunits of the opposite hexamer to form the hexamer - hexamer interface \cite{Zhu2005,Strohmeier2006}. The hydrophobic contacts between adjacent subunits within each hexamer are primarily formed by helices $\alpha$7 and $\alpha$8 on one subunit and helix $\alpha$3 on the adjacent subunit \cite{Strohmeier2006}.        
		
\begin{figure}[!htbp]
\begin{minipage}{\linewidth}
	\makebox[\linewidth]{
		
	\includegraphics[width=8cm, height=8cm, keepaspectratio]{/Users/matt/Dropbox/ThesisPrep/Thesis/fig/2NV2/2ISS.png}}	

	\captionof{figure}[The Structure of Pdx1]{The secondary structure of \textit{Thermatoga maritima} Pdx1. The side chains of two lysine residues critical for the catalytic activity of Pdx1 are shown in stick format, lysine residues 82 and 150 in \textit{B. subtilis} are numbered residues 98 and 166 respectively in \textit{A. thaliana}. Two phosphate-binding sites have been identified in Pdx1 and are named P1 and P2 \cite{Zein2006}.\linebreak \label{fig:TMPdx1}}	
\end{minipage} 
\end{figure}

The Pdx2 subunit of PLP synthase has a three-layered $\alpha\beta\alpha$ sandwich fold and uses a Glutamine-Histidine-Cysteine catalytic triad to catalyse glutamine hydrolysis \cite{Bauer2004,Gengenbacher2006}. Pdx2 glutaminase activity is dependent on interaction with Pdx1, crystal structures of the complete PLP synthase complex show that interaction with Pdx1 is necessary for Pdx2 to adopt an active conformation\cite{Wrenger2005,Strohmeier2006}. The $\alpha\beta$ loops on the stability face of Pdx1 are at the interface between Pdx1 and Pdx2, residues 1-20 of Pdx1 are typically disordered, however, upon complex formation residues 1-4 form a short $\beta$-strand ($\beta$-N) that hydrogen bonds to $\beta$7 of Pdx2 (Figure \ref{fig:Pdx1_2Interface}). Pdx1 residues 6-16 also form an N-terminal $\alpha$-helix ($\alpha$-N) at the interface with Pdx2 that mutagenesis experiments have shown to be essential for activation of the Pdx2 glutaminase activity \cite{Wallner2009}. The requirement for Pdx1 binding before activation of the Pdx2 glutaminase activity prevents inappropriate glutamine hydrolysis.

\begin{figure}[!htbp]
\begin{minipage}{\linewidth}
	\makebox[\linewidth]{
		
	\includegraphics[width=13cm, height=9cm, keepaspectratio]{/Users/matt/Dropbox/ThesisPrep/Thesis/fig/Pdx1_2/2NV2Interfacered.png}}	

	\captionof{figure}[The Pdx1/Pdx2 Interace]{The interface between \textit{Bacillus subtilis} Pdx1 (blue) and Pdx2 (white) \cite{Strohmeier2006}. A Pdx1 N-terminal $\beta$-strand ($\beta$N, orange) becomes ordered upon complex formation and hydrogen bonds to strand $\beta$7 of the Pdx2 $\beta$-sheet, the N-terminal $\alpha$-helix of Pdx1 ($\alpha-N$, orange) also becomes ordered \cite{Strohmeier2006}. The interaction between $\alpha-N$ and Pdx2 is essential for activation of Pdx2 glutaminase activity \cite{Wallner2009}. The glutamine molecule is shown in stick format (carbon orange, oxygen red, nitrogen blue) close to the Pdx1/2 interface. The central barrel of Pdx1 is shown in red \cite{Strohmeier2006,Guedez2012}. \linebreak \label{fig:Pdx1_2Interface}}	
\end{minipage} 
\end{figure}
		
The ammonia produced by the glutaminase reaction is highly reactive. To prevent the ammonia diffusing away from PLP synthase before it can be used by Pdx1, it is channelled through a transient hydrophobic tunnel lined by methionine residues in the centre of the Pdx1 \TIM~ barrel to the P1 site where it can be used for PLP biosynthesis \cite{Guedez2012}. The presence of a tunnel within Pdx1 that opens in response to glutamine hydrolysis was first postulated on the basis of the \textit{B. subtilis} PLP synthase structure and has since been confirmed by mutagenesis studies showing that the conserved methionine residues are important for coupling of the glutaminase activity to PLP biosynthesis \cite{Strohmeier2006,Tambasco-Studart2007,Guedez2012}. Several other examples of ammonia channelling from the site of glutamine hydrolysis to a distant active site have been observed in glutamine amidotransferase enzyme; these include carbamoyl phosphate synthetase and imidazole glycerol phosphate synthase \cite{Thoden1997,Douangamath2002,Raushel2003}. 

	\subsection{Catalysis of PLP biosynthesis by PLP Synthase}	
Knowledge of the structure of the enzyme is not, in itself, enough to understand how the protein performs its biological function. \textit{In vitro} reconstitution of PLP biosynthesis by PLP synthase has allowed for some of the details of the reaction mechanism to be elucidated using a combination of X-ray crystallography, UV-Vis spectroscopy, NMR and mass spectrometry\glsadd{NMR}. The biochemical analysis has determined that substrate binding and catalysis proceed in an ordered manner via a series of covalent intermediate states.  
 
Mass spectrometry has been used to identify that R5P binds covalently to \textit{B. subtilis} Pdx1 residue Lys81 (Lys98 in \textit{A. thaliana} Pdx1) \cite{Raschle2007}. This observation has been confirmed in the crystal structures of \textit{Plasmodium berghei} Pdx1 and \textit{Geobacillus stearothermophilus} Pdx1 in complex with R5P \cite{Guedez2012,Smith2015}. Incubating the Pdx1 enzyme with isotopically labelled substrates allowed for the use of NMR to determine that carbon 1 of the open form of R5P binds covalently to the lysine $\varepsilon$-nitrogen \cite{Hanes2008b}. The crystal structures of Pdx1-R5P have shown that the phosphate group of R5P is bound in the P1 site of Pdx1 \cite{Guedez2012,Smith2015}.

Incubating Pdx1-R5P with ammonium sulphate or Pdx2 and glutamine leads to the formation of an intermediate with an absorption maximum $\sim$ 315 nm that has been named I320 \cite{Raschle2007,Hanes2008b}. While the rate of I320 formation is $\sim$ ten times slower when substituting Pdx2 and glutamine with ammonia, the product that is formed has the same absorption spectrum, suggesting that the compounds that are formed are identical \cite{Raschle2007}. The reduction in the rate of the reaction may be due to Pdx1 using ammonia to catalyse I320 formation rather than charged ammonium. In addition, the glutaminase reaction that Pdx2 performs releases energy that may be used to drive conformational changes in Pdx1 required for catalysis. Formation of the Pdx1/Pdx2 complex also ensures that the ammonia is channelled directly to the active site of Pdx1 rather than relying on diffusion to deliver the substrate.   

Mass spectrometry and NMR data show that the I320 intermediate retains the five carbons from R5P, and comparison of the $^{13}$C NMR spectra of I320 reconstituted \textit{in vitro} using $^{14}$N and $^{15}$N ammonium chloride showed that the nitrogen of the ammonia is incorporated at the C2 position \cite{Raschle2007,Hanes2008c}. Incorporation of ammonia is followed by elimination of the R5P phosphate group, while the only oxygen atom remaining from R5P is bound to C3 \cite{Raschle2007,Hanes2008a,Hanes2008b}.          

Once the I320 intermediate has been formed \textit{in vitro} it is stable and remains covalently bound to the protein \cite{Raschle2007}. The NMR analysis suggested that both C1 and C5 of the intermediate were bound to nitrogen atoms \cite{Hanes2008b}. However, this was explained as being a consequence of the denaturing conditions that the experiment was performed under \cite{Hanes2008b}. In the mechanistic proposal produced by Hanes \textit{et al}, the incorporation of ammonia leads to the release of C1 of the intermediate from the lysine $\varepsilon$-nitrogen and the intermediate to covalently rebind a Pdx1 lysine side chain via C5 \cite{Hanes2008b}.

The addition of G3P to Pdx1-I320 leads to a reduction in the absorbance peak at 315 nm and a simultaneous increase in absorbance at 408 nm; absorbance of light by the product, PLP, is responsible for the formation of the 408 nm peak \cite{Hanes2008b}. An absorbance at 408 nm is characteristic for PLP covalently bound to a protein, compared to 388 nm for free PLP \cite{Hanes2008b}. The P2 site of Pdx1 has been identified as the binding site for PLP, whether PLP remains covalently bound to the protein is debated in the literature, the crystal structure of \textit{Saccharomyces cerevisiae} Pdx1-PLP shows a non-covalent interaction while UV-vis spectroscopy and NMR data suggest that PLP is covalently bound to the protein \cite{Hanes2008b,Zhang2010}. 

The first substrate, R5P, is bound in the P1 site while the product is bound in the P2 site; the two sites are separated by $\sim$ 20 \si{\angstrom} and it is currently unknown how the reaction is transferred from one location to the next. An improved understanding of the structure of the I320 intermediate in non-denaturing conditions and determining whether G3P binds in the P1 or P2 site may clarify the mechanism by which the reaction is transferred between active sites.        

\cleardoublepage
\section{Macromolecular Crystallography and Radiation Damage}

To fully understand how a protein performs its function, it is necessary to determine its molecular structure at the angstrom (10\textsuperscript{-10} \si{\metre}) scale. Imaging techniques are limited to a resolution equal to half the wavelength of the light used to probe the sample \cite{Abbe1873}. It is, therefore, necessary to use light with a wavelength in the X-ray range of the electromagnetic spectrum to obtain information about the positions of specific atoms and the lengths of particular bonds within the three-dimensional protein structure. The resolution of X-ray microscopes is currently limited to $\sim$10 nm due to the technical challenge of producing optics capable of precisely focussing X-rays, this resolution is not sufficient to resolve atomic positions and bond lengths \cite{Schropp2012}. 

The most common method for determining the structure of proteins is X-ray crystallography. This technique relies on crystallising the protein; the crystal is then illuminated with X-rays, and the diffraction patterns that are produced can be used to determine the protein structure.  
%and using the diffraction patterns that are produced when the crystal are illuminated with X-rays, to solve the structure. 

The following section describes how diffraction of X-rays by crystals occurs, how X-ray diffraction data can be used to determine a protein structure, and how X-rays can damage protein crystals.
     
\subsection{Single Crystal X-ray Diffraction}
%Refer to Dauter data collection paper, if you can get it.
Protein crystals grow when protein molecules aggregate into an ordered array and form a three-dimensional lattice, a diagram of a two-dimensional lattice is shown in Figure \ref{fig:2D_lattice}. The repeating unit of the crystal is the unit cell (Figure \ref{fig:UnitCell}); each unit cell contains one or multiple copies of the protein, in addition to the solvent molecules that surround the protein.   

The aim of the X-ray diffraction experiment is to reconstruct the electron density map for the unit cell from the information encoded in the diffracted X-rays; it is then possible to build a model for the protein structure by placing atoms into the electron density map.  

\begin{figure}[!htbp]
\centering
\begin{subfigure}{.5\textwidth}
  \centering
  \includegraphics[width=7cm, height=7cm, keepaspectratio]{/Users/matt/Dropbox/ThesisPrep/Thesis/fig/diffraction/2D_lattice/miller2.png}
  \caption{}
  \label{fig:2D_lattice}
\end{subfigure}%
\begin{subfigure}{.5\textwidth}
  \centering
  \includegraphics[width=7cm, height=7cm, keepaspectratio]{/Users/matt/Dropbox/ThesisPrep/Thesis/fig/diffraction/2D_lattice/UnitCell.png}
  \caption{}
  \label{fig:UnitCell}
\end{subfigure}
\caption[Diagram of a two-dimensional lattice]{(a) Diagram of a two-dimensional lattice. Translating from any circle by vector \textbf{a} in the x-axis or \textbf{b} in the y-axis will result in it being superimposed on a neighbouring circle. Each molecule in the lattice is surrounded by an identical environment. Sets of lattice planes (blue) are named according to the distance between the planes along each of the unit cell axes, taken as a fraction of the unit cell length. In this two-dimensional lattice, the blue set of planes intersect \textbf{a} at an interval of 1/2 \textbf{a} and \textbf{b} at an interval of 1/3 \textbf{b}. The planes are referred to by the reciprocal of their spacing along each axis. \textit{h} is the reciprocal of the spacing along \textbf{a}, in this case, \textit{h} = 2. \textit{k} is the reciprocal of the spacing along \textbf{b}, in this case, \textit{k} = 3. In a three-dimensional lattice, \textit{l} is the reciprocal of the spacing along \textbf{c}. Lattice planes that are parallel to an axis of the unit cell have a value of 0 in that direction.\newline
(b) Diagram of a three-dimensional unit-cell, this is the repeating unit of the crystal and contains the protein molecule. The dimensions of the unit cell are defined by lengths of the edges of the unit cell \textbf{a}, \textbf{b} and \textbf{c} along the x, y and z-axes respectively. The angle between \textbf{a} and \textbf{b} is $\gamma$, between \textbf{a} and \textbf{c} is $\beta$ and between \textbf{b}, and \textbf{c} is $\alpha$.}
\end{figure}

\par

Braggs law explains diffraction of X-rays by a lattice (Figure \ref{fig:Bragg}) \cite{Bragg1913}. The planes of the lattice are treated as reflecting planes, and the incoming light is taken to be monochromatic and in phase. If the difference in pathlength between two waves scattered by parallel planes separated by distance \textbf{d$_{hkl}$} is equal to an integer (\textit{n}) of the wavelength ($\lambda$), the waves will interfere constructively (Equation \ref{eq:BraggsLaw}, Figure \ref{fig:Bragg}). 

\begin{equation}\label{eq:BraggsLaw}
n\lambda = 2 \mathbf{d_{hkl}} sin \theta
\end{equation}

In addition to lattice planes that run parallel to the axes of the unit cell, additional planes may also intersect any of the axes. The planes are named by their Miller indices, which are the number of times that they intersect a given axis, for example, the (2,3) plane of a two-dimensional lattice would intersect the \textbf{a} edge of a single unit cell twice and \textbf{b} three times (Figure \ref{fig:2D_lattice}). The Miller indices specify the distance (\textbf{d$_{hkl}$}) between the planes and the direction in which the planes are oriented relative to the unit cell. Any scattering objects separated by the distance (\textbf{d$_{hkl}$}) in the specified direction will contribute to the diffraction of X-rays by the crystal.  
%it is necessary to rotate the crystal relative to the X-ray beam, thereby changing $\theta$ to ensure that reflections from all of the planes are measured. To accurately compute the electron density within the unit cell it is necessary to measure the intensity of the reflection from every lattice plane/%\cite{Dauter}.          
\begin{figure}
  \begin{minipage}{\linewidth}
	\makebox[\linewidth]{
		
	\includegraphics[width=14cm, height=7cm, keepaspectratio]{/Users/matt/Dropbox/ThesisPrep/Thesis/fig/diffraction/2D_lattice/Braggs_Law.png}}	

	\captionof{figure}[Braggs Law]{The two incident X-rays are in phase, the X-rays are scattered by two parallel planes of the lattice. The angle of incidence is $\theta$ and is equal to the angle of reflection. X-ray2 travels 2\textit{x} further than X-ray1. \textit{x} is equal to \textbf{d$_{hkl}$} sin $\theta$. The difference in pathlength for the two X-rays is, therefore, equal to 2 \textbf{d$_{hkl}$} sin $\theta$. If the difference in pathlength between the X-ray source and the detector is equal to an integer (\textit{n}) of the wavelength ($\lambda$) the two scattered waves will interfere  constructively, and Braggs law (Equation \ref{eq:BraggsLaw}) is fulfilled \cite{Bragg1913}.
	   \label{fig:Bragg}}	
\end{minipage} 
\end{figure}

Scattering from each of the lattice planes will contribute to the diffraction from a crystal; the direction that the diffracted X-rays travel in is determined by the geometry of the unit cell while the intensity of each reflection is determined by the composition of the unit cell. A useful construction for understanding which reflections will be detected in a given orientation is the Ewald sphere. The Ewald sphere is a construct in reciprocal space in which each set of lattice planes is defined by a reciprocal lattice vector (\textbf{d}$^{*}_{hkl}$).

\begin{figure}[!htbp]
  \begin{minipage}{\linewidth}
	\makebox[\linewidth]{
		
	\includegraphics[width=14cm, height=7cm, keepaspectratio]{/Users/matt/Dropbox/ThesisPrep/Thesis/fig/diffraction/Ewald/EwaldSphere.png}}	
	\captionof{figure}[The Ewald Sphere]{The radius of the Ewald sphere is equal to $\frac{1}{\lambda}$, when $\mathbf{d^{*}_{hkl}} = \frac{2 \sin \theta}{n \lambda}$ the reciprocal lattice point will lie on the Ewald sphere. The origin of the reciprocal lattice is at point (0,0). Reciprocal lattice points that lie on the Ewald sphere fulfil Braggs Law. In the orientation shown in this example, reciprocal lattice points (-1,1) and (-1,-1) will contribute to the diffraction pattern.
	   \label{fig:Ewald}}	
\end{minipage} 
\end{figure}   

In a two-dimensional lattice, the reciprocal lattice vector is normal to the lattice planes and has a magnitude equal to the reciprocal of the distance between the planes in real space (\textbf{d$_{hkl}$}) (Equation \ref{eq:ReciprocalVector}). The smaller the distance between a set of planes in real space the larger the reciprocal lattice vector. By plotting each of the reciprocal lattice vectors about an origin, a reciprocal lattice is constructed where each of the lattice points represents a set of lattice planes in real space (Figure \ref{fig:Ewald}). 

\begin{equation}\label{eq:ReciprocalVector}
 \mathbf{d^{*}_{hkl}} = \frac{1}{\mathbf{d_{hkl}}} 
\end{equation}
\begin{equation}\label{eq:Ewalds}
 \mathbf{d^{*}_{hkl}} = \frac{2 \sin \theta}{n \lambda} 
\end{equation} 

Braggs Law can be re-arranged as shown in Equation \ref{eq:Ewalds}, so that when the reciprocal lattice vector is equal to 2 $\sin \theta$ divided by the wavelength of the light, the conditions for diffraction to occur are met. The Ewald sphere is a graphical representation of this equation (Figure \ref{fig:Ewald}). Figure \ref{fig:Ewald} shows that only a few of the reciprocal lattice points lie on the Ewald sphere and contribute to diffraction when the crystal is in a given orientation. 

By determining the scattering angle ($\theta$) for each reflection, it is possible to calculate the reciprocal lattice vectors, and from these to compute the unit cell constants. Scattering from every atom in the unit cell contributes to the intensity of each reflection, the intensities of each reflection, therefore, contain information about the contents of the unit cell. 

The wave scattered by each set of lattice planes can be described by three parameters, its direction, phase and amplitude. The wave is represented mathematically by the complex structure factor (\textbf{F$_{\mathbf{hkl}}$}) (Equation \ref{eq:ComplexStructureFactor}), and contains information about the amplitude, direction and phase of the scattered wave. The structure factor amplitude ($F_{hkl}$), is the amplitude of the wave, which is proportional to the square root of the intensity that is measured in the diffraction experiment (Equation \ref{eq:Intensity_Frequency}). 

\begin{equation}\label{eq:Intensity_Frequency}
 (I_{hkl}) \propto (F_{hkl})^{2} 
\end{equation}

\begin{equation}\label{eq:ComplexStructureFactor}
 \mathbf{F_{hkl}} = F_{hkl}e^{2{\pi}i{\phi}_{hkl}} 
\end{equation}

The scattering angle at which the X-rays scattered by a given set of lattice planes will constructively interfere depends on the interplanar distance and orientation of the lattice planes, these are defined by the Miller indices ($hkl$). The phase of each wave is noted as $\phi$; unlike the direction and amplitude of the scattered waves, the phases of the scattered X-rays are cannot be measured directly. The phases for the structure factors can be determined experimentally using the anomalous scattering and isomorphous replacement methods or through the molecular replacement method. A full description of the details of each of these methods is beyond the scope of this introduction; however, once the phases are obtained it is possible to reconstruct the electron density in the unit cell using Equation \ref{eq:ED_Equation}.

\begin{equation}\label{eq:ED_Equation} 
p(x,y,z) = \frac{1}{V} \sum_{hkl}^{} \lvert F_{hkl} \rvert e^{-2{\pi}i{\phi}_{hkl}(hx+ky+lz)}
\end{equation} 

Equation \ref{eq:ED_Equation} describes how a Fourier transform can be used to reconstruct the electron density within the unit cell (\textit{p}(\textit{x,y,z})) from the structure amplitudes (F$_{hkl}$) and phases ($\phi$). To reconstruct the electron density, the sum of all of the complex structure factors (\textbf{F$_{\mathbf{hkl}}$}), which contain both the phase and amplitude information, is taken and multiplied by the reciprocal of the unit cell volume (\textit{V}). We, therefore, need to measure the intensity of the diffraction from all of the lattice planes to reconstruct the electron density map accurately.

It is necessary for all of the reciprocal lattice points to intersect the Ewald sphere to measure the diffraction from each of the lattice planes. This can be achieved by either repositioning the X-ray source, rotating the crystal or measuring the diffraction pattern at multiple wavelengths. Moving the X-ray source has the effect of re-orienting the Ewald sphere relative to the reciprocal lattice. Rotating the crystal causes a rotation of the reciprocal lattice around its origin. Changing the experimental wavelength alters the radius of the Ewald sphere.       

In macromolecular crystallography, the most common method of data collection is to rotate the crystal and measure the intensity of the diffraction of the crystal at a single wavelength. Measuring the diffraction from a crystal in all possible orientations on a single image would result in overlapping reflections. Instead, the crystal is rotated as diffraction images are recorded, usually with a rotation of less than 1$\degree$ per image.

%The absorption of X-rays during data collection while the crystal is rotated in the beam can have damaging effects, as described in Section \ref{sec:damage}.




\clearpage
\subsection{Radiation Damage in Macromolecular Crystallography}\label{sec:damage}
\subsubsection{Global Radiation Damage}
Only $\sim$2\% of the incident X-rays interact with the protein crystal during the diffraction experiment, of these X-rays 8\% are scattered elastically and contribute to the diffraction pattern \cite{Ravelli2006}. The remaining 92\% of interacting X-rays are either scattered inelastically or contribute to the photoelectric effect \cite{Ravelli2006}. Inelastic scattering occurs when the energy of the scattered photon is less than the energy of the incoming photon, the energy difference is deposited with the electron and results in its ejection from the atom \cite{Nave1995}. The photoelectric effect also results in the ejection of an electron from an atom; however, there is no scattered photon, although there may be further fluorescence effects not described here \cite{Nave1995}. \par

The ejection of electrons from atoms within the protein by X-rays during  data collection is referred to as primary radiation damage and has the greatest effect at sites with atoms that have high atomic numbers and therefore photoelectric cross-sections \cite{Holton2009}. In protein crystals, these tend to be sulphur atoms in cysteine and methionine residues and bound metal ions. The photoelectrons generated during crystallographic data collection can travel several hundred nanometres through a crystal causing the ejection of additional low energy electrons from solvent and protein atoms \cite{Sanishvili2011,Cowan2008}. As the liberated electrons pass through the crystals and react with both solvent and protein molecules free radicals are generated \cite{Garman2010}. The production of photoelectrons and free radicals leads to reactions that cause a reduction in the order of the crystal lattice and therefore a loss in diffraction, meaning that eventually data cannot be collected from the crystal, this effect is known as global radiation damage \cite{Garman2010}. It is necessary to measure the intensities for every lattice plane before global radiation damage causes the crystal to stop diffracting, to collect a complete dataset from a single crystal. \par  

It has become common practice to collect diffraction data at 100 \si{\kelvin}; this has the effect of trapping radicals, reducing their ability to migrate through crystal and damage the protein at crystal contact points, which causes the crystal to become disordered \cite{Garman2010}. The amount of damage caused to the crystal is proportional to the amount of energy absorbed per unit mass; X-ray dose is therefore measured in the SI unit Gray which is equivalent to joules per kilogram (\si{\joule\per\kilo\gram}). 

Several metrics can be used to monitor the effect of global radiation damage on the data collected. One metric used measures the change in the average intensity relative to the intensity of the first image. Owen \textit{et al} determined that the average intensity of diffraction from cryo-cooled protein crystals reduces by half after absorbing 43 \si{\mega\gray} \cite{Owen2006}. Cryo-cooling extends the lifetime of protein crystals by a factor of $\sim$70 relative to most room temperature crystals \cite{Nave2005}. This increase in the crystal lifespan allows for one, and in some cases, multiple, datasets to be collected from a single crystal. \par

\subsubsection{Site Specific Radiation Damage}\label{sec:specific_damage}
Site specific damage occurs at lower doses than global radiation damage and is not apparent in the data collection statistics but can affect electron density maps \cite{Ravelli2000}. Particular sites within some proteins contain atoms that have a higher atomic number than the carbon, hydrogen, nitrogen and oxygen atoms that form most of the protein. These include the sulphur atoms of cysteine and methionine residues and metal cofactors that often play a role in catalysis for enzymes. 

As mentioned previously, these atoms have a relatively high photo-electric cross section at X-ray energies and are damaged before the loss of order in the crystal lattice. The effects of site specific radiation damage are visible in the final electron density map as reduction of metal ions \cite{Berglund2002}, elongation and breakage of disulphide bonds, decarboxylation of acidic residues and loss of the hydroxyl group on tyrosine side chains \cite{Berglund2002,Ravelli2000,Burmeister2000}. Due to the importance of acidic residues and metal ions in enzymatic catalysis, it is essential to consider the potential effects of radiation damage on their position, conformation and redox state before drawing conclusions on their role in catalysis.      \par

The fact that metal ions are reduced in the X-ray beam, rather than oxidised, as would be expected if damage occurred solely through the ejection of electrons by the photoelectric effect and inelastic scattering, suggests that the direct interaction between X-rays and high atomic number atoms is not the primary cause of site specific radiation damage. As mentioned previously, electrons are mobile at 100 \si{\kelvin}, electron spin resonance has been used to show that the mobile electrons generated by the photoelectric effect may be transferred along the backbone of the protein until they reach a site with high electron affinity, where they are trapped \cite{Jones1987}. This explains why metal centres and disulphide bonds, which have high electron affinities, are specifically damaged. The mechanism of decarboxylation of acidic residues, which do not have particularly high cross-sections, has also been described using the migration of electrons and electron holes (positions where an electron could exist bound to an atom but is not) \cite{Burmeister2000}. The sensitivity of a given residue to radiation damage is not only dictated by the structure of the amino acid but also the local environment, residues close to atoms with high photoelectric cross sections or with high solvent accessibility are often more susceptible to site-specific radiation damage while residues close to interfaces with nucleic acids show less susceptibility to damage \cite{Gerstel2015,Bury2016}. 

The effects of site specific damage in real space are that particular atoms are shifted and adopt new positions or become disordered. As the atoms move relative the lattice planes of the crystal, the effect of site-specific radiation damage in reciprocal space is that the intensity of the reflections change, some increase and some decrease. 

As the crystal is exposed to X-rays, it absorbs an increasing amount of energy and sequentially passes through a series of damage states. Before exposure to X-rays, the entire crystal is in an undamaged state (A$_1$). As the crystal is exposed to X-rays, site-specific damage takes place and the crystal transitions to the A'$_1$ state; the lattice is not disordered in this state so there is no change in the resolution or the total intensity of diffraction \cite{Sygusch1988}. 


\begin{minipage}[t]{\textwidth}
\begin{equation}
A\textsubscript{1} \xrightarrow{K\textsubscript{0}} A'\textsubscript{1} \xrightarrow{K\textsubscript{1}} A\textsubscript{2} \xrightarrow{K\textsubscript{2}} A\textsubscript{3}
\end{equation}

%\end{minipage}\par
%\begin{minipage}{\textwidth}

The sequential model of radiation damage proposed by Sygusch and Allaire showing the progression from the undamaged state (A$_1$), to damaged at specific sites but still ordered (A'$_1$), to disordered with loss of high resolution diffraction (A$_2$) to amorphous (A$_3$). The rate of transition between states (K$_0$, K$_1$, K$_2$) is dependent on the rate that X-rays are absorbed and susceptibility of the crystal to damage \cite{Sygusch1988}. 
\end{minipage} \par

As data collection proceeds the fraction of the crystal in the undamaged state reduces and more data is collected from regions in the A'$_1$ state. When we use the Fourier transform to reconstruct the electron density, we include observations of reflections collected from across the entire data collection. The final electron density map contains contributions from the crystal in both the A$_1$ and A'$_1$ states. If only a small fraction of the data was collected from the A'$_1$ state, the influence of radiation damage on the final maps may be negligible; however, if the data is mostly collected from the A'$_1$ state, the electron density map will be representative of the damaged state.

Changes in the electron density map due to site specific radiation damage can lead to incorrect modelling of the protein structure and influence the interpretation of how the protein performs its function.%include statement and possibly figure about why microbeams are useful. 

The only way method presently available to obtain a crystal structure that is completely unaffected by radiation damage is to use an X-ray Free Electron Laser which takes advantage of the ``diffraction before destruction" principle \cite{Chapman2014}. Using an X-ray Free Electron Laser, a single diffraction image is collected from a crystal on a femtosecond time scale, faster than radiation damage processes can alter the structure of the molecule of interest. Data from thousands of crystals are merged to produce a damage free dataset.               
\newpage
\subsection{Multi-Crystal X-ray Diffraction Experiments}
\label{sec:multixtal_intro}

It may not always be possible to collect a single complete dataset before the effects of specific or global radiation damage affect the quality of the data collected. However, it is possible to collect data from multiple crystals or several different areas on a large crystal and to merge the data to construct a complete dataset. The requirement that must be fulfilled to merge data from different crystals is that they must be isomorphous, meaning they have same spacegroup, similar unit-cell dimensions and the contents of the unit cell are the same. A commonly used indicator of the degree to which two crystals are isomorphous is the variation in the unit cell dimensions \cite{Foadi2013}. Unit cell parameters are initially calculated at the indexing stage of data processing, programs such as XDS, mosflm and DIALS determine the most likely unit cell based on the positions of the spots on the detector and information about the relative geometry between the incident beam, crystal and detector \cite{Kabsch2010,Waterman2016,Battye2011}. 

In the case that several wedges of data can be collected from a set of isomorphous crystals it is possible to offset the starting angle for each dataset by a given rotation (Figure \ref{fig:multi_offset}), the datasets can then be split during data processing and sweeps of data collected at similar doses merged to produce complete datsets with collected at different doses. The electron density maps of the protein at different doses can be compared by creating an Fo$_{HighDose}$-Fo$_{LowDose}$ electron density map. Negative peaks in the maps inicate that the density of electrons in that region of space has decreased, while the formation of positive peaks indicate that an atom has moved into that space.

\begin{minipage}{\linewidth}
	\makebox[\linewidth]{
		
	\includegraphics[width=14cm, height=7cm, keepaspectratio]{/Users/matt/Dropbox/ThesisPrep/Thesis/fig/multi_xtal_scheme/multi_xtal.png}}	

	\captionof{figure}[Scheme for multi-crystal datset compilation]{Scheme of a multi-crystal data collection protocol, for a crystal that requires a 100 degree rotation for collection of a complete dataset from several crystals. The transition of the crystal from an undamaged to a site-specific damaged state is represented by the blue-red gradient. Processing of the data from any of datasets 1-5 will result in an average of damaged and undamaged states. By off setting the starting position of each dataset by 20 (degrees) and merging the first 20 degrees of each dataset (composite dataset 1) it is possible to form a complete dataset that can be used to produce a low-dose map. Using the following 20 degrees will yield a higher dose dataset that can be compared to the low dose dataset to identify sites of specific radiation damage. Protocol derived from Berglund \textit{et al} \cite{Berglund2002}. \label{fig:multi_offset}}	
\end{minipage} 


\cleardoublepage
\section{Using UV-Vis Spectroscopy to Investigate Protein Structure and Function}

\subsection{UV-Vis Absorption Spectroscopy}

UV-Vis absorption spectroscopy is a technique that has been widely used to investigate the mechanisms of proteins, and enzymes in particular, in solution. In the UV-Vis absorption experiment, the light source within the spectrophotometer illuminates the sample with light between the wavelengths of 200 nm and 1000 nm. The incident photons excite electrons in the molecules within the sample from their ground state to an excited state. Only photons with an energy that is the same as the energy difference between the ground state and the excited state are absorbed \cite{Saleh2001}. 

The energy of the absorbed photon (\textit{E}) is related to it frequency (\textit{v}) by Plancks constant (\textit{h}). The wavelength of the absorbed photon ($\lambda$) is equal to the speed of light (\textit{c}) divided by the frequency. 
\begin{equation}\label{eq:EnergytoFrequency}
\textit{E} = \textit{hv}
\end{equation}

\begin{equation}\label{eq:FrequencytoWavelength}
\lambda = \frac{c}{v}
\end{equation}

\begin{figure}[!hbtp]
\centering
\begin{subfigure}{.5\textwidth}
  \centering
  \includegraphics[width=7cm]{/Users/matt/Dropbox/ThesisPrep/Thesis/fig/spectra/Introduction/RotVibPeak.png}
  \caption{}
  \label{fig:UV_Peak}
\end{subfigure}%
\begin{subfigure}{.5\textwidth}
  \centering
  \includegraphics[width=7cm, height=8cm, keepaspectratio]{/Users/matt/Dropbox/ThesisPrep/Thesis/fig/spectra/Introduction/GaussOfGauss.pdf}
  \caption{}
  \label{fig:GaussOfGauss}
\end{subfigure}
\caption[Composition of UV-Vis absorption peaks]{(a) The absorption of UV-Vis light by molecules causes excitement of an electron to a higher energy state. Molecules exist in equilibrium between a number of vibrational and rotational states that may all be excited to a higher electron state by absorption of a photon. The total difference in the energy between the initial state and the excited state is equal to the energy of the absorbed photon. As there are many possible combinations of initial and excited vibrational and rotational states that a molecule may adopt, photons of many different energies may be absorbed by a molecule. (b) A UV-Vis absorbance peak for a single chromophore (black) is formed from the sum of the absorption of photons causing the transition of the molecule between all initial and final vibrational and rotational states(blue, green, red). In cases where several types of electronic transition are possible, there will often be one peak per type of electronic excitement.}
\end{figure}  

The spectrophotometer does not directly measure absorbance values; it instead measures the intensity of the light that passes that reaches the detector in the absence of the sample (\textit{I$_0$}) and the intensity of light that reaches the detector when the sample is in place (\textit{I}). In the case of measuring the UV-Vis spectrum of a protein, a reference spectrum will usually be taken with a cuvette containing the protein buffer to determine \textit{I$_0$}. A second spectrum is collected from a cuvette containing the protein of interest in the same buffer to determine \textit{I}. The ratio between \textit{I} and \textit{I$_0$} is termed equal to transmittance of the sample (\textit{T}) (Equation \ref{eq:UVTransmittance}). 

\begin{equation}\label{eq:UVTransmittance}
T = \frac{I}{I\textsubscript{0}} 
\end{equation}

\begin{equation}\label{eq:UVAbs}
A = \log \left(\frac{1}{\textit{T}}\right)  
\end{equation}  

The absorbance of a sample (\textit{A}) can be calculated from the transmittance using Equation \ref{eq:UVAbs}. This conversion relies on the assumption that light not absorbed by the sample is transmitted, this is not the case if the sample scatters light. It is convenient to work in terms of absorbance with samples that observe the Beer-Lambert law as the concentration of the chromophore changes linearly with absorbance (Equation \ref{eq:BeerLambert}). By monitoring the change in absorbance of a sample over time it is possible to determine the change in concentration of a chromophore. This principle is often used in enzyme assays to determine the rate at which a substrate is converted to an intermediate or the product, if any of the compounds of interest are chromophoric.          
 
\begin{equation}\label{eq:BeerLambert}
A = \varepsilon \times \textit{C} \times \textit{l}
\end{equation}

The Beer-Lambert law states that the absorption by a molecule in solution is equal to its molecular extinction coefficient ($\varepsilon$) in units of \si{\per\molar\per\centi\meter} multiplied by the molar concentration of the molecule (\textit{C}) multiplied by the pathlength of the sample (\textit{l}) in centimetres. The extinction coefficient of a molecule defines the probability of a molecule absorbing a photon of light at a given wavelength. 

Given that the energy of the absorbed photon must match the energy difference between the initial and excited states it would be expected that the peaks in UV-Vis absorbance spectra would be narrow and well resolved. However, the atoms in each molecule exist in an equilibrium between several vibrational and rotational states, the absorbance of light may cause a change in both the rotational and vibrational state of the molecule in addition to excitation of the electronic state. The energy difference between these states is not the same as the electronic excitation alone, and photons of different energies may therefore by absorbed (Figure \ref{fig:UV_Peak}) \cite{Saleh2001}. The absorption bands for each of the excitations overlap and appear as a single broad peak in the absorption spectrum (Figure \ref{fig:GaussOfGauss}). 

In cases where more than one type of electronic excitation is possible for the absorbing molecule or there are multiple absorbing species in the sample, multiple peaks may be present in the spectrum. Due to the broadness of the peaks in UV-Vis spectra they often overlap, this can make it difficult to quantify the concentration of each absorbing molecule in the sample. Overlapping peaks can make it difficult to compare spectra collected from samples under different conditions or at different time points in an experiment if more than one peak changes, as any change in the observed absorbance at a given wavelength will be caused by changes in the size of several peaks. Section \ref{sec:Methods_Gaussian_Modelling} describes a method that can be used to separate a complex spectrum into the individual peaks that it is formed from.   



%Each individual peak in the spectrum can be defined by three parameters, the wavelength of maximum absorbance (\lwl), the width of the peak at half of the maximum intensity ($\Delta\lambda_{frac{1}{2}}$) and the maximum intensity of the peak (A$_{max}$).  
%
%By measuring spectra of the individual components it is possible to determine the wavelength that each molecule absorbs light most strongly (\lwl), the width of the peak ($\sigma$) and the molecular extinction co-efficient for the molecule ($\varepsilon$). The parameters are determined by fitting the data to a function, typically a gaussian.   
%
%It is possible to determine the concentration of each component in the spectrum if the \lwl, molecular extinction co-efficient and peak width of each of the individual components is known. The typical method for doing so is to model each of the peaks as a gaussian function with a defined    
\newpage 
\subsection{\textit{In Crystallo} Optical Spectroscopy}
It is often necessary to use it in combination with complementary techniques to gain a greater understanding of how a protein may function. Several synchrotrons around the world have developed the capability to collect ultraviolet-visible (UV-Vis) light absorption spectra and Raman spectra from protein crystals both at and away from X-ray beamlines \cite{Pearson2009}.  \par 

UV-Vis spectroscopy is reliant on the feature of interest having an absorbance peak between 200 \nm ~- 900 \nm. All proteins absorb light in the UV-Vis range; the peptide bond has two absorption maxima at 190 nm and 220 nm. The aromatic amino acids phenylalanine, tyrosine and tryptophan, have absorption maxima at 257 nm, 274 nm and 280 nm respectively \cite{Wilson2005}. Metal ions and ligands bound to proteins may also have an absorption peak in the UV-Vis range, these molecules are known as chromophores and are often essential for the function of the protein, $\sim$ 20\% of proteins contain a chromophoric prosthetic group \cite{Dworkowski2015}. UV-Vis spectroscopy can be used to correlate the crystal structure with a state observed in solution and to track enzymatic reactions as they proceed \textit{in crystallo} \cite{Ronda2015}. UV-Vis spectroscopy has also been used to track reactions that occur within the crystal during data collection and may be used to identify cases where the structure of a chromophore is altered by site specific radiation damage \cite{Pearson2009}. 

UV-Vis spectroscopy has been used to identify several proteins where bound metal ions are reduced during X-ray exposure. Correct assignment of the oxidation state of metal ions involved in catalysis is essential in determining the role of the ion in catalysis and it is necessary to know whether the interpretation of electron density maps is influenced by artefacts produced by radiation damage \cite{Antonyuk2011}. UV-Vis spectroscopy is also sensitive to X-ray induced changes in the conformation of organic prosthetic groups that are bound to proteins, as has been identified in the case of retinal in bacteriorhodopsin which isomerises in response to X-ray irradiation \cite{Borshchevskiy2011,Borshchevskiy2014}. UV-Vis spectroscopy can also detect reactions caused by the interaction of X-rays with protein crystals and the surrounding solvent. The most commonly observed phenomenon is the formation of a broad absorption peak with $\lambda_{max} \sim$600 nm due to the generation of solvated electrons by the photoelectric effect as X-rays interact with glycerol, which is often used as a cryoprotectant in protein crystallography experiments at 100 K \cite{McGeehan2009,Owen2012}.

The collection of UV-Vis spectra from protein crystals differs from the collection of spectra from protein in solution in several ways. Due to the regular arrangement of the protein molecules in a crystal, the chromophores tend to be aligned in a particular direction; this can lead to anisotropic absorption of light by large chromophores such as heme, and is not observed in solution where the chromophores are aligned randomly \cite{Wilmot2002}. Aside from the effects of anisotropy, the spectrum of the crystal will change when it is rotated as the thickness of the crystal in the light path changes and because there can be varying amounts of scattering of light by the surfaces of the crystal in different orientations \cite{Wilmot2002,Dworkowski2015}. The anisotropy makes it difficult to interpret changes in the spectra of protein crystals as they are rotated during collection of X-ray diffraction data. Rather than attempting to collect X-ray diffraction and UV-Vis spectroscopy data from a crystal simultaneously, it is often simpler to expose a still crystal to X-rays and collect UV-Vis spectra to ensure that the spectroscopic changes can be assigned as being caused by the absorption of X-rays.                

%\clearpage\null\newpage
  
%Optical spectroscopy has been applied at beamlines as an orthogonal method to track the progress of reactions \textit{in crystallo} and identify cases of specific radiation damage. UV-Vis spectroscopy has been applied to protein crystals to determine the redox state of protein crystals and to track the progress of enzymatic reactions \textit{in crystallo}\cite{Pearson2009}.  

%X-ray diffraction experiments may be used to calculate a protein structure that is spatially averaged across all proteins within the crystal and also averaged over the time taken to collect the diffraction data. 

%It is often desirable to collect spectroscopic data that allows the investigator to assign the structure to a state observed under 
%It is possible to collect ultraviolet-visible (UV-Vis) absorption, Raman and Infrared spectra of single protein crystals at several third generation synchrotrons. These techniques may be used to relate the three dimensional protein structure calculated using X-ray diffraction to a 

