 \chapter{Materials and Methods}\label{ch:Methods}
	\section{Molecular Biology}
		\subsection*{Site-Directed Mutagenesis}
		
The wild type \textit{At}Pdx1.3\glsadd{atpdx1} had previously been cloned into a pET-21a-d(+) vector at the NdeI/XhoI restriction site \cite{Tambasco2005}. Point mutations to \textit{At}Pdx1.3 were introduced using the QuikChange II Site-Directed Mutagenesis Kit (Agilent Technologies). High Purity Salt Free (HPSF) mutagenic primers of 30-40 bases supplied by Eurofins MWG were designed to introduce point mutations at the codon for the residue of interest. The sequences of the primers used are listed in Appendix \ref{App:Primers}.
  
			\subsubsection{Reaction Mix}
				\begin{itemize}
				\item 1 \ul dNTPs\glsadd{dNTPs} (10 \si{\milli\molar} each)
				\item 1.5 \ul Forward Primer (10 \si{\micro\molar})
				\item 1.5 \ul Reverse Primer (10 \si{\micro\molar})
				\item 1 \ul Plasmid DNA (20 \si{\nano\gram\per\micro\litre})
				\item 5 \ul 10x Reaction Buffer
				\item 39 \ul Distilled \ce{H2O} 
				\item 1 \ul \textit{PfuUltra} High Fidelity DNA Polymerase (2.5 U \si{\per\micro\litre})	 
				\end{itemize}
\newpage				
			\subsubsection{PCR Protocol}
			\begin{description}
			\item[(1)] 95 \si{\degreeCelsius} 30 seconds start
			\item[(2)] 95 \si{\degreeCelsius} 30 seconds DNA melting	
			\item[(3)] 55 \si{\degreeCelsius} 60 seconds primer annealing
			\item[(4)] 72 \si{\degreeCelsius} 360 seconds elongation
			\item[(5)] Repeat steps (2) - (4) for total of 16 cycles
			\item[(6)] 72 \si{\degreeCelsius} 300 seconds to complete elongation
			\item[(7)] 4 \si{\degreeCelsius} Hold indefinitely
			\end{description}
			
Template DNA is still present in the mixture after the PCR, this must be removed before the transformation of the plasmid into \ecoli, as it contains the ampicillin resistance marker but is not mutated. The transformation of this plasmid into \ecoli will produce false positive colonies. The template DNA has been purified from DH5$\alpha$ \ecoli, as described in Section \ref{sec:Transformation_Methods}, and has been methylated by the bacteria, in contrast to the unmethylated products of the PCR. Methylated template DNA was digested by incubation of the PCR\glsadd{PCR} product with the restriction enzyme \textit{Dpn1}, which selectively digests methylated DNA, at 37 \si{\degreeCelsius} for one hour. 
		\newpage

		\subsection*{Transformation for Plasmid Amplification}
		\label{sec:Transformation_Methods}
		Plasmids were transformed into chemically competent DH5$\alpha$ \ecoli cells using the heat shock method. DH5$\alpha$ cells were used due to their high transformation efficiency and low non-specific intracellular endonuclease activity \cite{BRL1986}.
			\subsubsection{Materials}
			\begin{itemize}
			\item Competent DH5$\alpha$ \ecoli cells
			\item Low Salt Lysogeny broth (Ampicillin 100 \si{\micro\gram\per\litre}, Sodium Chloride 5 \gl, Tryptone 10 \gl, Yeast Extract 5 \gl, pH 7.0)
			\item LB agar plate (Agar 12 \gl , Ampicillin 100 \ugl , Sodium Chloride 10 \gl , Tryptone 10 \gl , Yeast Extract 5 \gl, pH 7.2)
			\item Plasmid DNA (30 \si{\nano\gram\per\micro\litre} - 100 \si{\nano\gram\per\micro\litre})
			\end{itemize}
			
			\subsubsection{Protocol}
		
			3 \ul of plasmid was mixed with 50 \ul competent DH5$\alpha$ \ecoli, incubated at 4 \degrees for 30 minutes and heat shocked at 42 \degrees in a water bath for 60 seconds. 200 \ul lysogeny broth was added to the cells before shaking at 160 RPM and 37 \degrees for one hour. The cells were spread on an agar plate and incubated at 37 \degrees overnight. 50 \ml of lysogeny broth containing antibiotic was inoculated with a single colony from the agar plate and allowed to grow for eight to twelve hours. The plasmid was extracted from the cells using the standard protocol with a Qiagen MiniPrep kit. 
\par		
Plasmids were stored at -20 \degrees in water. 20\% glycerol stocks of DH5$\alpha$ cells containing each plasmid were stored at -80 \si{\degreeCelsius}.  
		 
	\newpage
	
		\subsection*{Transformation for Protein Expression}\label{sec:Transformation_Methods}
		The purified plasmid was transformed into BL21(DE3) \ecoli cells for protein expression. The BL21 \ecoli strain is deficient in the outer membrane protease \textit{ompT} and serine protease \textit{Ion}, reducing protein degradation.
		\subsubsection{Materials}
			\begin{itemize}
			\item Competent BL21 (DE3) \ecoli cells
			\item Low Salt Lysogeny broth (as per Section \ref{sec:Transformation_Methods})
			\item LB agar plate (as per Section \ref{sec:Transformation_Methods})
			\item Isolated Plasmid DNA (product of Section \ref{sec:Transformation_Methods}) 
			\end{itemize}
		
		\subsubsection{Protocol}
		
			1 \ul of plasmid DNA was mixed with 50 \ul competent BL21 (DE3) \ecoli cells. The cells were incubated at 4 \degrees for 30 minutes, heat shocked at 42 \degrees in a water bath for 60 seconds and recovered in 50 \ul lysogeny broth at 37 \degrees shaking at 160 RPM\glsadd{RPM} for one hour. The cells were then spread on an agar plate containing 100 \ugl ampicillin and incubated at 37 \degrees overnight.
					
		\newpage	
		
	\section{Protein Expression}
	
		\subsubsection{Materials}
			\begin{itemize}
			\item High Salt Lysogeny Broth (Ampicillin 100 \si{\micro\gram\per\litre}, Sodium Chloride 10 \gl, Tryptone 10 \gl, Yeast Extract 5 \gl, pH 7.0)
			\item Lactose 25\% (w/v)
			\item Low Salt Lysogeny Broth (as per Section \ref{sec:Transformation_Methods})
			\end{itemize}
		
		\subsubsection{Protocol}
			The protein of interest was over-expressed in one-litre cultures of BL21 (DE3) \textit{E. coli}. A single colony of BL21 \ecoli cells was picked from an agar plate and used to inoculate 50 \ml low salt lysogeny broth in a 250 \ml flask. The culture was incubated at 37 \degrees and shaken at 160 RPM for a minimum of six hours. 5 \ml - 10 \ml of the starter culture was used to inoculate a two-litre baffled flask containing one litre of autoclaved high salt lysogeny broth (160 RPM, 37 \si{\degreeCelsius}). Protein over-expression was induced by the addition of 60 \ml 25\% lactose once the optical density of the culture had reached 0.6-0.8. After addition of lactose the temperature of the culture was reduced to 30 \si{\degreeCelsius}, shaking at 160 RPM overnight.
	  
	\section{Cell Harvesting}
	\subsubsection{Materials}
			\begin{itemize}
			\item 1x Phosphate Buffered Saline (PBS)\glsadd{PBS} (Disodium Hydrogen Phosphate 10 \si{\milli\molar}, Monopotassium Phosphate 1.8 \si{\milli\molar}, Potassium Chloride 2.7 \si{\milli\molar}, Sodium Chloride 137 \si{\milli\molar}, pH 7.4)   
			\end{itemize}
		
		\subsubsection{Protocol}
The one-litre cultures were centrifuged at 3,993 g (4,000 RPM, Avanti J-20 XPI centrifuge, JLA 8.1000 rotor), 4 \degrees for 20 minutes. The supernatant was discarded, the pellet was re-suspended in 30 \ml- 40 \ml 1x PBS and stored at -20 \degrees until required.  

\newpage  
	\section{Protein Purification}
		\subsection*{Cell Lysis}
		\label{sec:Lysis_Methods}
			\subsubsection{Materials}
				\begin{itemize}
				\item Lysis Buffer (Glycerol 2\%, Imidazole 10 \si{\milli\molar}, Sodium Chloride 500 \si{\milli\molar}, Tris-HCl 50 \si{\milli\molar}, pH 7.5) 
				\item Lysozyme from Chicken Egg White (Sigma-Aldrich, Fluka)
				\end{itemize}
			\subsubsection{Protocol}
			BL21 cell pellets were defrosted by placing the Falcon tube in a beaker of cold water. Each pellet had a volume of $\sim$30 \si{\milli\litre}. The pellet was re-suspended in lysis buffer to a total volume of 50 \si{\milli\litre} to which 20 mg lysozyme was added. The solution was homogenised by vortexing.\par
			The homogenised cell solution was transferred to a sonication vessel, kept on ice and placed within the sonicator (Sonicator Ultrasonic Processor XL, Heat Systems). The sonicator was run with two minutes pulse time, 10-second pulse duration and 20-second pause between pulses, with the intensity set to six.
			
		\subsection*{Ultracentrifugation}
		\subsubsection{Protocol}
		The lysed celled were spun in a Beckman L7-65 ultracentrifuge in a Ti70 rotor pre-cooled to 4 \si{\degreeCelsius}. Pdx1 containing lysate was centrifuged at 55,000 RPM (310523 g). The supernatant was filtered using a 0.2 \si{\micro\meter} Sartorius MiniSart filter. \newpage
		
		\subsection*{Immobilised Metal Ion Affinity Chromatography (IMAC)}
		
		\subsubsection{Materials}
				\begin{itemize}
				\item Lysis Buffer (as per Section \ref{sec:Lysis_Methods})
				\item Wash Buffer (Glycerol 2\% (v/v), Imidazole 50 \si{\milli\molar}, Sodium Chloride 500 \si{\milli\molar}, Tris-HCl 50 \si{\milli\molar}, pH 7.5) 
				\item Elution Buffer (Glycerol 5\% (v/v), Imidazole 500 \si{\milli\molar}, Sodium Chloride 500 \si{\milli\molar}, Tris-HCl 50 \si{\milli\molar}, pH 7.5)
				\item 1 \si{\milli\litre} HisTrap HP Nickel Column (GE Healthcare)
				\end{itemize}
			\subsubsection{Protocol}
			IMAC\glsadd{IMAC} was performed at 4 \degrees to reduce protein degradation; the peristaltic pump was run with a flow rate of one \si{\milli\litre\per\minute} throughout the procedure. All proteins purified had an N-terminal His\textsubscript{6}.\\
			The tubing of the peristaltic pump system was washed with 10 \ml Nanopore filtered and distilled water. The nickel column was then attached and washed with distilled water followed by equilibration with 10 \ml lysis buffer. 50 \si{\milli\litre} - 60 \si{\milli\litre} of filtered cell lysate was then loaded onto the column followed by washing with 10 \ml lysis buffer and 10 \ml wash buffer to remove non-specifically bound proteins. The protein of interest was eluted with 3 \ml of elution buffer. \newpage
			
			
		\subsection*{Size Exclusion Chromatography}
		\label{sec:SEC_Methods}
\subsubsection{Materials}
				\begin{itemize}
				\item Vivaspin 20 Centrifugal Concentrator, MWCO 30,000 Da (Sartorius)
				\item Gel Filtration Buffer (Potassium Chloride 200 \si{\milli\molar}, Tris-HCl 50 \si{\milli\molar}, pH 8.0) 
				\end{itemize}
			\subsubsection{Protocol}
			The nickel column eluate was further purified by size exclusion chromatography using a 26/600 Superdex 200 prep grade gel filtration column (GE Healthcare). Size exclusion chromatography was performed at 4 \degrees with a flow rate of 2 \si{\milli\litre\per\minute}.\\
			The column was equilibrated with one column volume (330 \ml) of gel filtration buffer; the loop was washed with distilled water and gel filtration buffer. The protein sample was injected onto the column and run with a flow rate of 2 \si{\milli\litre\per\minute}. 

A 280 nm UV lamp was used to determine the fractions in which protein was eluting. Fractions showing high protein concentration were concentrated using a Vivaspin 20 Centrifugal Concentrator, MWCO\glsadd{MWCO} 30,000 \si{\dalton} (Sartorius). The final protein concentration was measured using a Nanodrop spectrophotometer.
\newpage
			\subsection*{SDS-PAGE}
			\subsubsection{Materials}
				\begin{itemize}
				\item 12\% SDS-PAGE\glsadd{SDSPAGE} gel				
				\item Anode Buffer (Tris-HCl 200 \si{\milli\molar}, pH 8.9) 
				\item Cathode Buffer (0.1\% Sodium Dodecyl Sulphate,  Tricine  100 \si{\milli\molar},Tris-HCl 100 \si{\milli\molar}, pH 8.2)
				\item 4x Reducing Sample Buffer ($\beta$-mercaptoethanol 1.2 \si{\molar}, Bromophenol Blue 0.004\% (w/v), Glycerol (20\%), Tris-HCl 100 \si{\milli\molar}, pH 6.8) 
				\item Fixing Buffer (Acetic Acid 10\% (v/v),  Ethanol 40\% (v/v))
				\item Coomassie stain (Coomassie Brilliant Blue G250 (90 \si{\milli\gram\per\litre}), Hydrochloric Acid \\0.555\%(v/v))  
				\end{itemize}
			\subsubsection{Protocol}Samples were taken at the cell lysis, ultracentrifugation, IMAC, size exclusion chromatography and protein concentration steps of protein purification. SDS-PAGE analysis of these samples enabled identification of steps in the purification where the protein was lost, degraded or contaminated with other proteins.\par

15 \ul of protein containing sample was mixed with 5 \ul  4x sample buffer. Depending on the expected protein concentration 5 \ul - 10 \ul of each sample was loaded onto the gel. The gel was run at 160 \si{\volt} for 40 minutes and placed in fixing buffer for 10 minutes. The gel was then transferred to the Coomassie stain, microwaved for 25 seconds and imaged after 30 minutes.
			
\newpage
	\section{Pdx1 Activity Assays}\label{sec:Assay_Methods}
	Activity assays were performed to determine the rate at which wild-type \atpdx~ and selected mutants catalysed the formation of the chromophoric I320 intermediate and the product, PLP. In addition to the assays spectra of the protein were collected before the enzyme assays, after the I320 assay and after the PLP assay. Due to aggregation of some of the samples during the assays, the amount of background scattering increased making interpretation of the spectra difficult. A background correction was made to ease interpretation of the spectra, as described in Section \ref{sec:bg}.  
		\subsection*{Pdx1 I320 Assay}	 \label{sec:320_assay_methods}
		\subsubsection{Materials}
				\begin{itemize}
				\item Gel Filtration Buffer (as per Section \ref{sec:SEC_Methods}) 
				\item \atpdx
				\item Ribose 5-phosphate 100 \si{\milli\molar}
				\item Ammonium Sulphate 4 \si{\molar}, diluted to 1 \si{\molar} in Gel Filtration Buffer
				\end{itemize}
		\subsubsection{Protocol}I320 assays were performed in a quartz cuvette with a path length of 1 \si{\centi\metre}. The total reaction volume was 300 \si{\micro\litre}. The final concentration of Pdx1 was 20 \si{\micro\molar}, Pdx1 was incubated with a final concentration of 10 \si{\milli\molar} R5P for 15 minutes before the addition of 30 \si{\micro\litre} 1 \si{\molar} ammonium sulphate (final concentration 100 \si{\milli\molar}). This protocol is a modified version of the protocol first described by Raschle \textit{et al} for reconstitution of the I320 intermediate in Pdx1 \cite{Raschle2007}. \par
Absorbance at 315 \si{\nano\metre} was measured at one-second intervals for one hour using a Hitachi U-3010 Spectrophotometer. 

A straight line was fitted to the absorbance at 315 nm for the first 50 seconds of the experiment using the MATLAB script in Appendix \ref{App:Enzyme_Activity}. The gradient of the fitted line is equal to the absorbance change per second; this was multiplied by 60 to determine to absorbance change per minute. The absorbance change per minute was multiplied by the extinction co-efficient for I320 ($\varepsilon$ = 16,200 \cite{Raschle2007}) to calculate the change in I320 concentration per minute (\si{\molar\per\minute}). The number of moles of I320 produced by the protein per minute \si{\mole\per\minute} is determined by multiplying the change in I320 concentration per minute by the assay volume in units of litres.  

The molar protein concentration multiplied by the volume of the assay in litres is equal to the number of moles of protein in the assay. The mass of the protein present in the assay is determined by multiplying the number of moles of protein by the molecular weight of the protein.

Dividing the moles of I320 produced per minute by the mass of protein in the assay gives the specific activity of the protein in units of moles of I320 per milligram of protein per minute (\si{\mol\per\milli\gram\per\minute}).      
	
\newpage		
		\subsection*{Pdx1 PLP Assay}
		\subsubsection{Materials}
				\begin{itemize}
				\item 300 \ul I320 assay (as per Section \ref{sec:320_assay_methods})
				\item G3P 100 \si{\milli\molar}
				\end{itemize}
		\subsubsection{Protocol} 
75 \ul of G3P was added to the 300 \ul I320 assay (Section \ref{sec:320_assay_methods}) to a final G3P concentration of 20 \si{\milli\molar}. The addition of G3P to Pdx1-I320 has previously been observed to cause a reduction in the absorbance at 320 nm and an increase in absorbance 415 nm signalling formation of PLP in active Pdx1 enzymes \cite{Raschle2007}. \par
		Absorbance at 415 \si{\nano\metre} was measured at one-second intervals for one hour using a Hitachi U-3010 Spectrophotometer.
		
The specific activity of \atpdx ~for PLP biosynthesis was calculated using the same procedure as previously described for the Pdx1 I320 assay; with the exception that the extinction co-efficient for PLP is 5,380, and that the fit was made to the absorbance between 200 and 300 seconds to exclude the lag phase \cite{Raschle2007}.
\clearpage		
		\subsection{Background Correction of \atpdx ~UV-Vis Spectra}\label{sec:bg}
 Previous studies have identified that the dominant source of background scattering in dilute protein samples is Rayleigh scattering \cite{Leach1960, Porterfield2010}. The spectrum measured (A\textsubscript{Measured}) is formed by a combination of two contributions, the first part is due to actual absorption of light by the sample (A\textsubscript{Corrected}) and the second is caused by light scattering effects (A\textsubscript{LS}), of which the Rayleigh effect is dominant \cite{Porterfield2010} (Equation \ref{eq:rayleigh_contribution}).


\begin{equation}\label{eq:rayleigh_contribution}
A\textsubscript{Measured} = A\textsubscript{Corrected} + A\textsubscript{LS}
\end{equation}

Rayleigh scattering decreases proportionally to the fourth power of the wavelength; it is possible to estimate the contribution of light scattering to the spectrum by fitting equation \ref{eq:rayleigh_lightscatter} to a region of the spectrum where no absorbance is expected from the protein sample.   

\begin{equation}\label{eq:rayleigh_lightscatter}
A\textsubscript{LS} = c_{1}(\lambda^{-4}) + c_{2}
\end{equation}

Equation \ref{eq:rayleigh_lightscatter} was fitted to a region of the spectrum where there is no absorbance is expected from the protein or any of the chromophoric intermediates, in this case between 500 \nm ~and 700 \nm . This allowed for calculation of the $c_{1}$ and $c_{2}$ coefficients, once these were determined the equation was evaluated across the entire  data range (240 \nm ~- 900 \nm ). This provided the estimated contribution to the measured absorbance from light scattering effects at all wavelengths where absorbance was measured. Subtracting this set of values ($A\textsubscript{LS}$) from the measured absorbance ($A\textsubscript{Measured}$) values provides a baseline corrected set of absorbance values ($A\textsubscript{Corrected}$) (Figure \ref{eq:rayleigh_lightscatter}). All data manipulations were performed in MATLAB, see Appendix \ref{App:MATLABSPEC_RAYLEIGH} for the MATLAB script used to perform the background correction. 

\begin{minipage}{\linewidth}
	\makebox[\linewidth]{
	\includegraphics[width=8cm, height=8cm, keepaspectratio]{/Users/matt/Dropbox/ThesisPrep/Thesis/fig/spectra/wl/WT_Native_RayleighSub/WT_Native_RayleighSub.pdf}}	

	\captionof{figure}[Subtraction of Background Scattering from \atpdx ~Spectra]{Spectra of \atpdx ~before addition of substrates (blue), equation \ref{eq:rayleigh_lightscatter} is fitted to the data in the 500 \nm ~- 700\nm ~range and then evaluated across the entire range (240 \nm ~- 900 \nm ) (red dashed line). Subtraction of the estimated contribution from light scattering from the initially measured spectrum results in a baseline corrected spectrum (blue dashed line).\label{fig:RayleighSub_demo}}	
\end{minipage}

The procedure described does not provide a perfect correction for background scattering, as can be observed by the negative absorbance that results in the 300 \nm ~- 310 \nm ~region. However, it does make the comparison of the spectra and the changes in the 320 \nm ~and 415 \nm ~features after the addition of each set of substrates easier (Figure \ref{fig:Pdx1WT_RayleighSub}).
		
		\newpage
	\section{Single Crystal X-ray Diffraction Experiments with \textit{At}Pdx1}
	\label{sec:Crystallisation_Methods}
	Crystallisation was performed using the vapour diffusion method. A drop of concentrated protein is mixed with a drop of precipitant solution and incubated in a sealed environment. The precipitant solution typically has a higher solute concentration than the concentrated protein solution, and therefore the protein/precipitant droplet. The water in the protein droplet diffuses into the reservoir over time, reducing the drop volume and raising the effective protein concentration.  \par
	The droplet begins in an under-saturated state with a protein concentration too low to favour aggregation into an ordered structure; as is required for crystal growth.

As the water diffuses out of the protein/precipitant droplet into the reservoir the effective protein and precipitant concentrations rise, and the solution becomes super-saturated, arriving in the nucleation phase. In the nucleation phase the protein molecules aggregate into many small seeds, this leads to a reduction in the effective concentration of the protein free in solution. As the effective protein concentration drops into the metastable phase, the formation of crystal seeds stops and crystal growth occurs.
 
Concentrated protein samples were initially screened against a variety of crystallisation conditions in the 96 condition pre-formulated sparse matrices JCSG+, Morpheus and Pact Premier Screens \cite{Newman2005,Gorrec2009}. Initial screening used sitting drop vapour diffusion with the screen and protein dispensed with a minimum drop size of 100 \si{\nano\litre} + 100 \si{\nano\litre} into a 96 well MRC plate by a Crystal Gryphon robot (Art Robbins Instruments).

Optimisation screens were created to produce larger crystals. These screens used the information gained from initial hits to select buffers and precipitants that had previously yielded crystals while varying pH and precipitant concentration. 

Customised 96 well optimisation screens were prepared using an Alchemist DT Liquid Handling System (Rigaku). Larger crystallisation droplets were used for optimisation, up to a maximum drop size of 2 \ul + 2 \si{\micro\litre}. 

The 96 well CrystalQuik X: Microplate supplied by Greiner was used for growth of crystals to be used for \textit{in situ} data collection; 100 \nl + 100 \nl drops were dispensed by the Crystal Gryphon robot. 
 
\subsection{Crystallisation of \textit{At}Pdx1.3, soaking experiments and X-ray diffraction data collection}
Crystallisation of \textit{At}Pdx1.3 was optimised in 96-well sitting drop and 24-well hanging drop crystallisation plates. Both wild-type \textit{At}Pdx1.3 and point mutants crystallised reproducibly at concentrations between 10 \mgml - 40 \si{\milli\gram\per\milli\litre}, in Tris pH 7.0 - pH 9.0, sodium acetate 0.1 M - 0.4 M and polyethylene glycol 4000 (PEG 4000). Optimisation screens varied the concentration of PEG 4000 between 4 \% (w/v) and 15 \% (w/v).
		
\subsubsection{Crystallisation of the wild type \textit{At}Pdx1.3-R5P complex}
Crystallisation of the Pdx1-R5P complex was performed by Stefan Weber. \atpdx~ was crystallised in 500 \si{\milli\molar} sodium citrate buffered with 100 \si{\milli\molar} HEPES\glsadd{HEPES} at pH 7.5 in a 24 well hanging drop plate \cite{Weber2008}. 0.5 \ul of 2.5 \mM R5P dissolved in 500 \si{\milli\molar} sodium citrate buffered with 100 \si{\milli\molar} HEPES at pH 7.5 was added to the 2 \ul crystallisation drop and incubated for five minutes before addition of the cryoprotectant solution (500 \si{\milli\molar} sodium citrate, 100 \si{\milli\molar} HEPES pH 7.5, 20\% glycerol (v/v)) and flash cooling of the crystals in preparation for data collection. 

Dataset sw5-4 was collected at ESRF beamline ID14-1 by Dr Ivo Tews on June 24\textsuperscript{th} 2008. %300 images were collected with 0.4\degree oscillation per image, 3 seconds exposure per image.    
%%%%%%% Fill in data processing section%%%%%%%%%%%%%%%%%
		\subsubsection{Crystallisation of the  wild type \textit{At}Pdx1.3-I320 complex}\label{sec:I320crystallisation}
		
\atpdx ~was crystallised in 300 \si{\milli\molar} ~sodium acetate pH 5.5, 100 \si{\milli\molar} ~Tris-HCl pH 8.4,  8.6\% (w/v) PEG4000\glsadd{PEG} at a concentration of 40 \si{\milli\gram\per\milli\litre}, with a 1 \ul : 1 \ul mix of protein to buffer solution in a sitting drop vapour diffusion crystallisation plate. Pdx1 crystals were soaked with 0.5 \ul of R5P soaking buffer (300 \si{\milli\molar} sodium acetate pH 5.5, 100 \si{\milli\molar} ~Tris-HCl pH 8.5, 12.3\% (w/v) PEG4000, R5P 10 \si{\milli\molar}) for a minimum of 15 minutes (final R5P concentration $\sim$2.5 \si{\milli\molar}). The crystal was then soaked with buffer solution containing ammonium sulphate (sodium acetate pH 5.5, 100 \si{\milli\molar} ~Tris-HCl pH 8.5,  12.3\% (w/v) PEG4000, ammonium sulphate 100 \si{\milli\molar}) to a final ammonium sulphate concentration of $\sim$25 \si{\milli\molar}) and incubated for two days. Cryoprotectant (300 \si{\milli\molar} ~sodium acetate pH 5.5, 100 \si{\milli\molar} ~Tris-HCl pH 8.4,  8.6\% (w/v) PEG4000 and 20\% glycerol (v/v)) was added to the crystallisation drop before cryocooling in liquid nitrogen. The highest resolution \atpdx -I320 dataset was collected from crystal ma4-1, which was produced during an undergraduate project preceding this project.    

Dataset ma4-1 was collected at DLS beamline I04-1 by Dr Ivo Tews on January 30\textsuperscript{th} 2012. %Diffraction images were indexed and integrated using DIALS, AIMLESS was used to scale and merge the data, molecular replacement was performed using MOLREP, model building and refinement were performed iteratively using COOT and REFMAC \cite{Waterman2013,Evans2013,Vagin1997,Emsley2010,Murshudov1997}. Library files defining the geometry of the I320 intermediate were generated using JLigand \cite{Lebedev2012}.  

\subsubsection{Crystallisation of the wild type \textit{At}Pdx1.3-I320/G3P complex}
The crystallisation of \atpdx, soaking experiments and X-ray diffraction data collection for the \textit{At}Pdx1-I320/G3P complex were performed by Dr Volker Windeisen and are described in detail in his PhD thesis \cite{Windeisen2013}. Pdx1 crystals were grown and soaked to form the I320 intermediate using the protocol described above for the \atpdx -I320 intermediate. After a four day incubation with ammonium sulphate the crystals were transferred to a drop containing the same buffer constituents as the reservoir solution with the addition on 10 \si{\milli\molar} G3P for 2 - 3 minutes, before being transferred to a drop containing the reservoir buffer constituents, 10 \si{\milli\molar} G3P and 20\% glycerol (v/v) for a few seconds prior to cryocooling \cite{Windeisen2013}.    

X-ray diffraction dataset vw47-2 was collected at ESRF beamline ID23-1 on September 5\textsuperscript{th} 2009. %Diffraction images were indexed and integrated using XDS, AIMLESS was used to scale and merge the data, molecular replacement was performed using MOLREP, model building and refinement were performed iteratively using COOT and REFMAC \cite{Kabsch2010,Evans2013,Vagin1997,Emsley2010,Murshudov1997}. Library files defining the geometry of the I320/G3P intermediate were generated using JLigand \cite{Lebedev2012}.  

		\subsubsection{Crystallisation of the wild type \textit{At}Pdx1.3-PLP complex}\label{sec:PLP_soak}
%%%%%%%%%%% Check soaking conditions%%%%%%%%%%%%%
\atpdx~ was crystallised at a concentration of 40 \mgml in 15\% PEG 4000 (w/v), 100\mM sodium acetate pH 5.5, 100\mM Tris pH 8.4 in a sitting drop vapour diffusion crystallisation plate with a 1 \ul : 1 \ul mix of protein to buffer solution. PLP was mixed with the reservoir solution to a concentration of $\sim$ 10 \si{\milli\molar}, 0.5 \ul of the PLP containing reservoir solution was added to the 2 \ul crystallisation drop before addition of the cryoprotectant solution (15\%PEG 4000 (w/v), 100 \mM sodium acetate pH 5.5, 100 \mM Tris\glsadd{Tris} pH 8.4, 20\% glycerol (w/v)) and cryocooling of the crystal. The highest resolution \atpdx -PLP dataset was collected from crystal ma17-3, which was produced during an undergraduate project preceding this project. 

X-ray diffraction dataset ma17-3 was collected at ESRF beamline ID14-1 on September 4\textsuperscript{th} 2011. %Diffraction images were indexed and integrated using DIALS, AIMLESS was used to scale and merge the data, molecular replacement was performed using MOLREP, model building and refinement were performed iteratively using COOT and REFMAC \cite{Waterman2013,Evans2013,Vagin1997,Emsley2010,Murshudov1997}.  


%The Pdx1-PLP complex was formed by soaking Pdx1 crystals with 2.5 \mM - 10 \mM pyridoxal 5'-phosphate (Sigma Aldrich). \atpdx ~was crystallised in 8\% PEG8000 buffered at pH 8.25 with 100 \si{\milli\molar} ~Tris-HCl . 
 	
\subsubsection{Crystallisation of \textit{At}Pdx1.3 K166R in the native state complex}
\atpdx~ K166R was crystallised at a concentration of 6.5 \mgml in 28.1\% PEG 1000 (w/v) and 100 \mM HEPES pH 7.0 in a sitting drop vapour diffusion crystallisation plate with a 1 \ul : 0.5 \ul mix of protein to buffer solution. Crystals were cryoprotected in the buffer solution with the addition of 20\% glycerol. 

Dataset ma46-1 was collected at ESRF beamline ID23-1 on June 26\textsuperscript{th} 2013. %Diffraction images were indexed and integrated using DIALS, AIMLESS was used to scale and merge the data, molecular replacement was performed using MOLREP, model building and refinement were performed iteratively using COOT and REFMAC\cite{Waterman2013,Evans2013,Vagin1997,Emsley2010,Murshudov1997}.   

\subsubsection{Crystallisation of \textit{At}Pdx1.3 K166R in the pre-I320 state}
\atpdx~ K166R was crystallised at a concentration of 6.5 \mgml in 28.1\% PEG 1000 (w/v) and 100 \mM HEPES pH 7.5 in a sitting drop vapour diffusion crystallisation plate with a 1 \ul : 0.5 \ul mix of protein to buffer solution. \atpdx~ K166R crystals were soaked with R5P and ammonium chloride in a single buffer (28.6\% PEG 1000 (w/v), 100 \mM HEPES pH 7.5, R5P 10 \si{\milli\molar}, ammonium chloride 800 \si{\milli\molar}). 0.6 \ul of soaking buffer was added to the 1.5 \ul crystallisation drop. Crystals were cryoprotected in 28.6\% PEG 1000 (w/v), 100 \mM HEPES pH 7.5 and 20\% glycerol. 

The collection of dataset ma54-1 was performed at ESRF beamline ID23-1 on June 27\textsuperscript{th} 2013. %Diffraction images were indexed and integrated using DIALS, AIMLESS was used to scale and merge the data, molecular replacement was performed using MOLREP, model building and refinement were performed iteratively using COOT and REFMAC\cite{Waterman2013,Evans2013,Vagin1997,Emsley2010,Murshudov1997}.  

\subsubsection{Crystallisation of \textit{At}Pdx1.3 D41N in the native state}
\atpdx~ D41N was crystallised at a concentration of 19 \mgml in 6.5\% PEG 4000 (w/v), 400 \mM sodium acetate pH 5.5 and 100 \mM Tris pH 7.8 in a hanging drop vapour diffusion crystallisation plate with a 1 \ul : 1 \ul mix of protein to buffer solution. Crystals were transferred to a drop containing the buffer solution with the addition of 20\% glycerol for cryoprotection before cryocooling.

The collection of dataset ma237-2 was performed at DLS beamline I02 on July 13\textsuperscript{th} 2015. %Diffraction images were indexed and integrated using DIALS, AIMLESS was used to scale and merge the data, molecular replacement was performed using MOLREP, model building and refinement were performed iteratively using COOT and REFMAC\cite{Waterman2013,Evans2013,Vagin1997,Emsley2010,Murshudov1997}.  

\subsubsection{Crystallisation of the \textit{At}Pdx1.3 D41N R5P complex}
\atpdx~ D41N was crystallised at a concentration of 19 \mgml in 4.5\% PEG 4000 (w/v), 400 \mM sodium acetate pH 5.5 and 100 \mM Tris pH 7.8 in a hanging drop vapour diffusion crystallisation plate with a 1 \ul : 1 \ul mix of protein to buffer solution. Crystals were soaked with 1 \ul of the crystallisation buffer with the addition of 10 \mM R5P. Crystals were transferred to a drop containing the buffer solution with the addition of 20\% glycerol for cryoprotection before cryocooling. 

The collection of dataset ma240-1 was performed at DLS beamline I02 on July 13\textsuperscript{th} 2015. %Diffraction images were indexed and integrated using DIALS, AIMLESS was used to scale and merge the data, molecular replacement was performed using MOLREP, model building and refinement were performed iteratively using COOT and REFMAC\cite{Waterman2013,Evans2013,Vagin1997,Emsley2010,Murshudov1997}.

		%\subsection{Single Crystal X-Ray Diffraction Experiments}
		%X-ray diffraction experiments were performed at Diamond Light Source (DLS)\glsadd{DLS} and the \\European Synchrotron Radiation Facility (ESRF)\glsadd{ESRF}. 
		%All data were collected at 100 \si{\kelvin} from crystals mounted in loops. Data were indexed and integrated using XDS \cite{Kabsch2010} or DIALS\cite{Waterman2013}, data reduction was performed using AIMLESS \cite{Evans2013}, Molrep was used for molecular replacement \cite{Vagin1997}, model building and refinement were performed iteratively using REFMAC\cite{Murshudov1997} and COOT\cite{Emsley2010}.     
		%\subsection{Multi-Crystal X-ray Diffraction Experiments}
		%Multi-crystal experiments used the same data collection protocol as described in \ref{sec:multixtal_intro}. RADDOSE 3D was used to calculate the diffraction weighted dose for the each wedge of data collected\cite{Zeldin2013}. The diffraction weighted dose for the wedge was divided by the number of images in the wedge to obtain the average Diffraction Weighted Dose per image. The number of images below a threshold dose for each crystal was then known and could be integrated using XDS or DIALS, the integrated files were then merged and scaled using BLEND\cite{Waterman2013,Kabsch2010,Foadi2013}. Molecular replacement and refinement were then performed to obtain a low dose crystal structure.  
\newpage
\subsection{Processing of Single Crystal X-ray Diffraction Data}

Indexing and integration of X-ray diffraction datasets sw5-4 (wild type \atpdx -R5P) and vw47-2 (wild type \atpdx -I320/G3P) was performed using XDS \cite{Kabsch2010}. All other datasets were indexed and integrated using DIALS \cite{Waterman2013}. During the indexing step in processing the dimensions of the unit cell are determined, and a space group is tentatively assigned to each dataset. All of the wild-type and mutant \atpdx ~crystals were in the R3 space group with similar unit cell dimensions, approximately \textbf{a} = 178 \si{\angstrom}, b = 178 \si{\angstrom}, c = 116 \si{\angstrom}. The integration software counts the intensity of each Bragg spot and lists the intensities together with the \textit{hkl} of each spot and the estimated uncertainty of the measured intensity. Due to the collection of fine-sliced data with an oscillation per image of less than 0.5\degree ~a single reflection can often be split across several images; these are known as partial reflections. Both DIALS and XDS can perform three-dimensional integration so that these partial reflections are combined at the integration stage \cite{Kabsch2010,Waterman2013}.  

The CCP4 program AIMLESS was used for data reduction with all datasets \cite{Evans2013}. Over the course of the diffraction experiment, the intensity observed for a given reflection can change due to variability in the flux of the X-ray source, the volume of the crystal in the beam and the effects of global radiation damage. AIMLESS uses a scaling algorithm to attempt to compensate for the effects of changing experimental conditions on the data. AIMLESS also merges multiple observations of the same reflection and symmetry-related reflections. Once the data are merged, AIMLESS calls another program, CTRUNCATE, to convert the observed intensities into amplitudes \cite{Evans2013}.

Molecular replacement was performed with MOLREP for all datasets \cite{Vagin1997}. The sw5-4 (wild type \atpdx ~R5P) dataset was solved by Dr Ivo Tews after molecular replacement with the \textit{B. subtilis} Pdx1 protein (PDB code: 2NV2) and iterative model building with REFMAC and COOT \cite{Strohmeier2006,Murshudov1997,Emsley2010}. The other structures were solved by molecular replacement with the refined sw5-4 model with the ligand and solvent atoms removed from the PDB, further model building and refinement were performed using COOT and REFMAC \cite{Emsley2010,Murshudov1997}. Several of the complexes contain covalently bound ligands, the dictionary files defining the geometry of these ligands were generated in JLIGAND and input to COOT and REFMAC during model building and refinement \cite{Lebedev2012}. Dataset vw47-2 was refined with the assistance of Dr Yang Zhang (Cornell University) using PHENIX \cite{Adams2011}. Datasets were refined with isotropic B-factors, and, in some cases Translation Libration Screw (TLS) refinement as described in Table \ref{tab:Refmac}.     

\begin{table}[!h]
  \centering
\begin{tabular}{|P{4.5cm}|P{1.6cm}|P{1.6cm}|P{2cm}|P{2.9cm}|}
\hline
 \multicolumn{5}{|c|}{Refinement Parameters} \\
\hline
Structure&Dataset Name&Resolution ~(\si{\angstrom})&Program&B-factors\\
\hline
\atpdx -R5P&sw5-4&1.90&REFMAC&Isotropic \& TLS\\%10
\atpdx -I320&ma4-1&1.73&REFMAC&Isotropic \& TLS\\%5
\atpdx -I320/G3P&vw47-2&1.90&PHENIX&Isotropic \& TLS\\
\atpdx -PLP&ma17-3&1.61&REFMAC&Isotropic \& TLS\\
\atpdx ~D41N&ma237-2&2.24&REFMAC&Isotropic\\
\atpdx ~D41N R5P&ma240-1&2.18&REFMAC&Isotropic\\
\atpdx ~K166R&ma46-1&2.40&REFMAC&Isotropic\\
\atpdx ~K166R pre-I320&ma54-1&2.23&REFMAC&Isotropic \& TLS\\%10
\hline
\end{tabular}
  \caption[Refinement parameters for single crystal \atpdx ~data]{Refinement parameters for single crystal \atpdx ~datasets.\label{tab:Refmac}}
\end{table}

     
\clearpage 
\section{Calculating the X-ray Dose Absorbed by Crystals}

As described in Section \ref{sec:damage}, the X-rays used in diffraction experiments have a damaging effect on protein crystals. The amount of damage that occurs is dependent on the amount of X-ray energy absorbed by the crystal per unit mass. In cases where a complementary technique such as UV-Vis absorption or Raman spectroscopy is used to track site-specific radiation damage, it is necessary to quantify the amount of X-ray energy absorbed by the sample with the changes in the spectrum to understand how sensitive the feature of interest is to X-rays. Once the dose that the spectroscopic changes occur at is known it is possible to devise a data collection strategy that allows the investigator to solve the structure of the protein before the feature of interest is significantly damaged.    

The doses of X-rays absorbed by the crystals used in this project were calculated using the program RADDOSE-3D \cite{Zeldin2013}. This program requires input of details concerning the crystal (crystal size, unit cell parameters, number of residues per protein monomer, number of monomers per unit cell, solvent fraction, number of heavy atoms in protein/solvent), the  beamline (flux, beam profile, beam size, energy of incident X-rays) and the data collection strategy (exposure time per image, oscillation, number of images) \cite{Zeldin2013}. 

The crystal parameters allow RADDOSE-3D to determine the proportion of X-rays that will be absorbed by the crystal. A crystal of a protein that contains many atoms with large photoelectric cross-sections such as bound metal ions or cysteine and methionine residues will absorb more a higher percentage of the incident X-rays than a similar crystal with fewer heavy atoms. The beam parameters allow RADDOSE-3D to determine the number of incident photons over the period of the exposure time, which is defined in the data collection parameters.  

RADDOSE-3D outputs the dose absorbed by the crystal using a few different metrics. In the case of a crystal that is completely and uniformly exposed to X-rays, diffraction from all parts of the crystal will contribute to the measured diffraction patterns and will also absorb X-rays at the same rate. In this case, it may be appropriate to use the Average Dose absorbed by the Whole Crystal (ADWC) or the Average Dose absorbed by the Exposed Area (ADEA), which should be equal. It should be noted that these metrics provide the dose absorbed by the crystal by the end of the X-ray exposure; this may not be a truly represent the dose that the data has been measured at, as the data is collected over the duration of the X-ray exposure rather than at the end.     

%In the case of a crystal that is smaller than the beam and is uniformly and completely illuminated by the X-ray beam it is relatively easily to define the dose absorbed by the crystal as the average dose absorbed by the whole crystal (ADWC).

The reality of data collection at third generation synchrotrons is that beam sizes tend to be smaller than the crystal and beams often have a Gaussian profile with a greater flux in the centre of the incident beam than at the edges. The advantage of this type of beam is that data collection is possible from relatively small or weakly diffracting crystals. The disadvantage is that when crystals are larger than the beam, the region of the crystal that is exposed to X-rays absorbs a high dose while other parts of the crystal are not exposed to X-rays and do not contribute to the diffraction pattern. In addition, some regions of the crystal rotate in and out of the beam. This results in a crystal that is inhomogeneously exposed to X-rays. In this case, it is not appropriate to refer to the ADWC as regions of the crystal that have not been exposed to X-rays absorb no X-rays and lower the overall value but also make no contribution to the diffraction data collected and are therefore irrelevant when determining the actual dose that the structure was solved at. The ADEA is an improvement on the ADWC, as only regions of the crystal that have been exposed to X-rays are included in the dose calculation; however, even within the exposed area some regions of the crystal remain in the X-ray beam for longer and also make a greater contribution to the diffraction data.   

\par

Rather than measuring dose in terms of the ADWC or ADEA, a newer metric has been suggested, diffraction weighted dose (DWD)\glsadd{DWD} \cite{Zeldin2013b}. Diffraction weighted dose is calculated by dividing the crystal into a number of cubes, termed voxels. The dose absorbed by each voxel at each time point in the experiment is calculated and weighted by the contribution that each voxel is making to the diffraction by the crystal at each time point. RADDOSE-3D takes the average dose absorbed by each voxel in the crystal, weighted by its contribution to the diffraction over the course of the experiment to calculated the dose. Using this procedure, DWD takes into account that images collected at the start of the data collection will be relatively unaffected by radiation damage and that regions of the crystal remaining in the X-ray beam the longest also contribute more to the diffraction pattern, unless global radiation damage is taking place.      

%Figure \ref{fig:Raddose_Beam_Profile} (left), shows the simple case of a crystal completely and evenly illuminated by X-rays. In this case the Diffraction Weighted Dose (9.50 \mgy) is approximately half of the Average Dose Absorbed by the Whole Crystal (18.80 \mgy). This is because the ADWC is a measure of the dose absorbed by the whole crystal at the end point of the experiment, while the DWD provides the average dose that all of the images are collected at over the course of the X-ray exposure. 

This is an important distinction to make, as the intensity measurements that are made over the course of the experiment are collected during the X-ray exposure rather than at the endpoint. The final electron density map that we use to build our model of the protein structure is not only averaged across all of the unit cells in the crystal but also over the time course of the experiment.

%The dose simulations displayed in Figure \ref{fig:Raddose_Beam_Profile} predict that two identical crystals would absorb similar amounts of energy across the whole crystal, however the crystal on the right has surpassed a diffraction weighted dose of 30 \mgy , suggesting that it would no longer be producing useful diffraction patterns \cite{Owen2006}.

%\begin{minipage}{\linewidth}
%	\makebox[\linewidth]{
%	\includegraphics[width=15cm, height=8cm, keepaspectratio]{/Users/matt/Dropbox/ThesisPrep/Thesis/fig/RADDOSE/Testsimulations}}	
%	\captionof{figure}[Comparison of Dose Distributions in Crystals Exposed to Different Beam Profiles]{Dose simulations for two identical crystals rotated in X-ray beams with different profiles. Left: Dose simulation of a 100 \si{\micro\metre}$^3$  crystal illuminated by a 200 \si{\micro\metre}$^2$ top hat X-ray beam. Average dose whole crystal 18.80 \mgy , diffraction weighted dose 9.50 \mgy . Right: Dose simulation of a 100 \si{\micro\metre}$^3$ crystal exposed to an X-ray beam with a gaussian beam profile (Full Width Half Maximum 10 \um)\glsadd{FWHM}. Average dose whole crystal 18.28 \mgy , diffraction weighted dose 114.86 \mgy . The areas shaded in light blue absorbed 0.1 \mgy - 20 \mgy , purple 20 \mgy ~- 30 \mgy , red $>$ 30 \mgy .}	\label{fig:Raddose_Beam_Profile}	
%\end{minipage}  
%		
\clearpage
\section{Collection and Processing of Multi-Crystal Data}	
The protocols for collection and processing of the multi-crystal datasets for both lysozyme, which was used as a test case, and \atpdx -I320 were derived from the protocol used by Berglund \textit{et al} to investigate site-specific damage of the horseradish peroxidase enzyme \cite{Berglund2002}. The protocol for the construction of the isomorphous difference density maps was used by Southworth-Davies \textit{et al} in an investigation into the effects of site-specific radiation damage to lysozyme crystals at room temperature and cryo-temperatures \cite{Southworth-Davies2007}.

The protocols were first tested with Hen Egg White Lysozyme, referred to as lysozyme from here onwards, which has been well characterised in radiation damage experiments and is expected to show signs of site-specific radiation damage at around the sulphur atoms of the disulphide bonds.
  
\subsection{Lysozyme Multi-Crystal Experiment}
\subsubsection*{Lysozyme Crystallisation and Data Collection}	
Lysozyme was purchased from Sigma-Aldrich as a lyophilised powder. The protein was dissolved in distilled water at a concentration of 100 \si{\milli\gram\per\milli\litre} and crystallised in a 96 well sitting drop vapour diffusion plate. The 96 well plate varied sodium chloride concentration 300 \si{\milli\molar} - 1 \si{\molar} in 100 \si{\milli\molar} increments and PEG monomethylether 5000 12 \% - 34\% (w/v) in 2\% increments with a constant concentration of 50 \si{\milli\molar} ammonium acetate pH 4.5. Cryoprotectant contained the reservoir solution with an additional 20 \% glycerol, single crystals were mounted in loops and cryocooled to 100 K in liquid nitrogen. 

Data collection was performed on beamline I04 (DLS) on January 21\textsuperscript{st}, 2016. The X-ray beam has a Gaussian profile with at Full Width Half Maximum of 31.7 \um~ x 20.0 \um~ (Horizontal x Vertical). The flux was measured to be 1.318 x 10$^{11}$ photons\si{\per\second} at the end of the beamtime. Several datasets were collected from large crystals; the crystal was translated by a minimum of 80 \um~ between datasets to ensure that each dataset was collected from a previously unexposed region of the crystal. The starting angle between datasets collected on a single crystal was offset by 30\degree ~relative to the previous data collection. 3600 images were collected in each dataset with an oscillation of 0.1\degree ~per image and a total oscillation of 360\degree. 16 datasets were collected from five lysozyme crystals; the datasets were collected without the use of test shots to determine the optimum starting orientation, as any test shots would affect the dose absorbed by the crystal before the data collection. The protocol of collecting data without determining the crystal orientation is termed the `American Method' \cite{Rossmann1983}.

\subsubsection*{Lysozyme Data Processing}\label{sec:lysozyme_processing}	

The dimensions of the crystals were measured using the on-axis camera on beamline I04 (DLS), the dimensions of each crystal and number of datasets collected from each crystal are listed in Appendix \ref{App:lys_xtal_dimensions}. Input parameters for calculation of the dose absorbed by each crystal are presented in Table \ref{table:RADINP_LYS}. The average DWD for all of the datasets is 705 kGy. Each dataset was split into 36 sweeps of 100 images; each 100 image sweep was collected at a DWD of ~20 \kgy, calculated using RADDOSE-3D \cite{Zeldin2013}.

\begin{table}[!ht]
 \centering
%\begin{tabular}{|p{6cm}|p{4cm}|}
\begin{tabular}{|l|c|}
 \hline
 \multicolumn{2}{|c|}{\textbf{Input Parameters for Lysozyme Dose Calculations}} \\
 \hline
 \multicolumn{2}{|c|}{Crystal Parameters}\\
 \hline
 Crystal Shape & Cuboid\\
 Crystal Size & Variable\\
 Unit cell \textit{a, b, c} ($\alpha$,$\beta$,$\gamma$)&77\si{\angstrom}, 77\si{\angstrom}, 37\si{\angstrom} (90, 90, 90)\\
 Monomers per Unit Cell    & 8\\
 Residues per Monomer&129\\
 Number of Protein Heavy Atoms& 11 Sulphur atoms\\
 Solvent Fraction&0.3705\\
 \hline
 \multicolumn{2}{|c|}{Beam Parameters}\\
 \hline
 Beam Type   & Gaussian\\
 Full-Width Half Maximum (H x V)    &60 \si{\micro\metre} x 50 \si{\micro\metre}\\
 Rectangular Collimation (H x V)  &31.7 \si{\micro\metre} x 20.0 \si{\micro\metre}\\
 Flux (photons \si{\per\second})    &1.318 x 10$^{11}$\\
 Incident Photon Energy (\si{\kilo\electronvolt})   &12.658 \\
 \hline
 \multicolumn{2}{|c|}{Data Collection Strategy} \\
 \hline
 Number of Images   & 3600 \\
 Oscillation per Image   & 0.1\degree \\
 Total Oscillation & 360\degree \\
 Exposure Time per Image (\si{\second}) & 0.04 \\
 Total Exposure Time (\si{\second})& 144.0 \\
 Transmission & 69.83\% \\
 \hline 
\end{tabular}
\caption[Input parameters for calculation of the dose absorbed by lysozyme crystals]{Parameters for calculation of X-ray dose absorbed by lysozyme crystals. The dimensions of the crystals were also input to RADDOSE-3D (Appendix \ref{App:lys_xtal_dimensions}). The solvent fraction of the lysozyme crystals was calculated using the Matthews Coefficient program in CCP4 \cite{Matthews1968}. The flux was measured at 100\% transmission at the end of the beamtime and is assumed to have remained constant during the beamtime.}\label{table:RADINP_LYS} 
\end{table}
    
Individual sweeps were then indexed and integrated independently using XDS, the INTEGRATE.HKL output file from XDS contains the intensity values for each reflection \cite{Kabsch2010}. XDS produced 16 INTEGRATE.HKL files (one per dataset) for each sweep, all of the INTEGRATE.HKL files for sweep number one of each dataset were input to BLEND in analysis mode to determine the degree of isomorphism between the sweeps. BLEND was then run in synthesis mode using POINTLESS and AIMLESS to merge and scale different combinations of datasets for each sweep together, starting with combinations of two or three highly isomorphous sweeps and increasing the number of sweeps, including less isomorphous datasets \cite{Evans2013,Foadi2013}. The scaling and merging statistics for each of the combinations of sweeps were listed in the output log files from BLEND. Increasing the number of sweeps improves the completeness of the composite datasets, but the inclusion of less isomorphous data can also worsen the merging statistics. The combination of files that had the lowest R$_{pim}$ while also having a completeness above 98\% was chosen as the composite dataset for each dose.  

The intensities in each of the merged composite files were converted into amplitudes using the CCP4 program TRUNCATE \cite{French1978}. Molecular replacement was performed with MOLREP using the lysozyme structure 2VB1 from the PDB against the lowest dose composite dataset, sweep 1 \cite{Wang2007,Vagin1997}. Manual model building in COOT and refinement using REFMAC were performed iteratively to fit the model to the electron density for sweep 1, placement of water molecules was performed in COOT \cite{Emsley2010,Murshudov1997}. The REFMAC refinement protocol used 10 cycles of Translation Libration Screw (TLS)\glsadd{TLS} refinement followed by 10 cycles of restrained refinement. The sulphur atoms were not restrained to form disulphide bonds to allow the geometry of any disulphide bonds to refine without restraints, consistent with the method described by Weik \textit{et al} \cite{Weik2002}.     

The structure factor amplitudes (F) and the standard deviations of the amplitudes (SIGF) for each composite dataset were concatenated into a single multi-column mtz file, together with the phases and figure of merit (FOM)\glsadd{FOM} for each amplitude from the refined sweep 1 mtz, using the CCP4 program CAD. The amplitudes for each composite dataset were all scaled against sweep 1 using the CCP4 program SCALEIT \cite{Howell1992}.   

Isomorphous difference density maps were produced using FFT \cite{Ten1973}. FFT was used to subtract the structure factors from the lowest dose composite dataset (Fo$_1$) from the structure factors for the subsequent sweeps (Fo$_n$). FFT then performs a Fourier transform on the Fo$_n$-Fo$_1$ structure factors with the phases from the lowest dose composite dataset to produce an isomorphous difference Fo$_n$-Fo$_1$ electron density map.    

%Isomorphous difference density maps were produced using FFT\cite{Ten1973}. FFT converts the structure factors and phases from the lowest dose composite file into an electron density map (Fo$_1$), it then converts the structure factors for the subsequent sweeps into electron density maps (Fo$_n$) using the phases from the lowest dose sweep. The program then subtracts the Fo$_n$ map from the Fo$_1$ map to produce the isomorphous difference density map (Fo$_1$-Fo$_n$).  

\subsection{\atpdx -I320 Multi-Crystal Experiment}
\subsubsection*{\atpdx -I320 Crystallisation and Data Collection}	
Protein from two separate protein purifications was crystallised for use in the \atpdx -I320 multi-crystal experiment. Crystals were grown both in hanging drop and sitting drop crystallisation plates at concentrations between 22 \mgml and 55 \si{\milli\gram\per\milli\litre}. The crystallisation buffers contained 100 mM Tris pH 8.0, 200 mM - 400 mM sodium acetate and 5\% - 15\% PEG 4000 (w/v).
The exact crystallisation conditions used to produce the crystals for the \atpdx -I320 multi-crystal experiment are listed in the Appendix Table \ref{table:I320multi-crystallisation}. The protocol used to generate in the I320 intermediate \textit{in crystallo} and to cryocool the crystal is described in Section \ref{sec:I320crystallisation}.  
 

27 datasets were collected from 22 \atpdx -I320 crystals mounted in loops at 100 \si{\kelvin} on December 15\textsuperscript{th}, 2013 at ESRF beamline ID23-1. No test shots were collected before collection of each 450 image dataset with an exposure time of 0.4 \si{\second} per image, 0.2\degree~ oscillation per image and transmission of 15\%. For five of the datasets only 300 images were collected, the other data collection parameters remained the same. The flux was constantly measured during the experiment and varied between 4.0 x 10$^{10}$ photons\si{\per\second} and 1.4 x 10$^{11}$ photons\si{\per\second}. As the flux was recorded for each dataset, the dose for each dataset was calculated using the flux measured at the time of data collection. The beam size at the sample was 45 \si{\micro\metre} x 30 \si{\micro\metre} (Horizontal x Vertical), with a Gaussian profile FWHM 40 \si{\micro\metre} x 30 \si{\micro\metre} (Horizontal x Vertical). As with the lysozyme data collection, no test shots were taken to avoid absorption of X-rays before the start of data collection.

\subsubsection*{\atpdx -I320 Multi-Crystal Data Processing}	

The dose absorbed by each crystal during data collection was calculated using RADDOSE-3D, crystal dimensions were measured at the time of crystal mounting \cite{Zeldin2013}. The input parameters for the dose calculation are listed in Table \ref{table:RADINP_320}; the diffraction weighted dose for each dataset is listed in Appendix Table \ref{App:I320Dose}.   

\begin{table}[!ht]
 \centering
%\begin{tabular}{|p{6cm}|p{4cm}|}
\begin{tabular}{|l|c|}
 \hline
 \multicolumn{2}{|c|}{\textbf{Input Parameters for \atpdx -I320 Dose Calculations}} \\
 \hline
 \multicolumn{2}{|c|}{Crystal Parameters}\\
 \hline
 Crystal Size & Variable (See Appendix \ref{App:I320Dose} )\\
 Crystal Shape & Cuboid\\
 Unit cell \textit{a, b, c} ($\alpha$,$\beta$,$\gamma$)&178\si{\angstrom}, 178\si{\angstrom}, 116\si{\angstrom} (90, 90, 120)\\
 Monomers per Unit Cell    & 36\\
 Residues per Monomer&315\\
 Number of Protein Heavy Atoms& 19 Sulphur atoms\\
 Solvent Fraction&0.5878\\
 \hline
 \multicolumn{2}{|c|}{Beam Parameters}\\
 \hline
 Beam Type   & Gaussian\\
 Full-Width Half Maximum (H x V)    &40 \si{\micro\metre} x 30 \si{\micro\metre}\\
 Rectangular Collimation (H x V)  &45 \si{\micro\metre} x 30 \si{\micro\metre}\\
 Flux (photons \si{\per\second})    & Variable\\
 Incident Photon Energy (\si{\kilo\electronvolt})   &12.7 \\
 \hline
 \multicolumn{2}{|c|}{Data Collection Strategy} \\
 \hline
 Number of Images   &300 or 450 \\
 Oscillation per Image   & 0.2\degree \\
 Total Oscillation & 90\degree \\
 Exposure Time per Image (\si{\second}) & 0.4 \\
 Total Exposure Time (\si{\second})& 180.0 \\
 Transmission & 15.03\% \\
 \hline 
\end{tabular}
\caption[Input parameters for calculation of the dose absorbed by \atpdx -I320 crystals]{Parameters for calculation of X-ray dose absorbed by \atpdx -I320 crystals. The dimensions of the crystals and flux were also input to RADDOSE-3D. The solvent fraction of the \atpdx -I320 crystals was calculated using the Matthews Coefficient program in CCP4 \cite{Matthews1968}.}\label{table:RADINP_320} 
\end{table}   
Each dataset was indexed using XDS; integration was then performed individually on 15 sweeps of 30 images, the output INTEGRATE.HKL file from each sweep was output to a separate folder; the script used to integrate the data is included in Appendix \ref{App:Multi-XtalScripts} and provides a more detailed description of the processing protocol \cite{Kabsch2010}. The diffraction weighted dose for each dataset was divided by the number of images and multiplied by 30 to determine the diffraction weighted dose for a thirty image sweep in each dataset, 131.8 kGy. The mean 30 image DWD for all of the datasets was multiplied by the sweep number to obtain the dose for each of the first ten sweeps. The mean 30 image DWD for the 450 image datasets (118.7 kGy) was multiplied by the sweep number to calculate the dose for sweeps 11-15.      

BLEND was then used to merge the integrated files for each of the sweeps of equivalent dose, the script used to merge the data is in Appendix \ref{App:Multi_XtalBlend} \cite{Foadi2013}. BLEND uses the CCP4 program POINTLESS to merge the intensities and outputs an unscaled mtz. The unscaled mtz was input to AIMLESS to scale the data and convert the intensities to amplitudes. The R3 space group has an internal symmetry with the \textbf{a} and \textbf{b} axes of the unit cell having an equal length. It is therefore equally to valid to index the data in two different ways; however, it is important that the dataset is internally consistent in which index is chosen. To ensure that all sweeps in each multi-crystal dataset were indexed the same the output mtz from the refinement of the Pdx1-R5P complex was used as a reference to which all input data would be matched to \cite{Evans2013}.      
 
Molecular replacement was performed with MOLREP using the single crystal \atpdx -I320 structure with and ligand atoms removed and the lowest dose composite dataset, sweep 1 (132 kGy) \cite{Vagin1997}. Manual model building in COOT and refinement using REFMAC were performed iteratively to fit the model to the electron density for sweep 1, placement of water molecules was carried out in COOT \cite{Emsley2010,Murshudov1997}. The REFMAC refinement protocol used eight cycles of Translation Libration Screw (TLS) refinement followed by five cycles of restrained refinement. Library files defining the geometry of the I320 intermediate in REFMAC and COOT were generated using JLigand \cite{Lebedev2012}. 

Isomorphous difference electron density maps were produced using the methods described in Section \ref{sec:lysozyme_processing}. 
\newpage
\subsection{Analysis of Radiation Induced Density Loss}
A quantitative analysis of the effects of site-specific radiation damage on the electron density maps for lysozyme and \atpdx -I320 was performed using the Radiation Induced Density Loss (RIDL) scripts developed by Bury \textit{et al} \cite{Bury2016}.  
  
The RIDL scripts produce a series of isomorphous difference density (Fo$_n$-Fo$_1$) maps using the MTZs, produced by TRUNCATE and AIMLESS for the lysozyme and \atpdx -I320 data respectively, which contain the structure factor amplitudes and associated standard deviations. The protocol used to generate the maps is the same as previously described for the lysozyme data using CAD, SCALEIT and FFT \cite{Bury2016,Howell1992,Ten1973}. 

The structure of the protein, refined against the lowest dose dataset, is also input to the RIDL scripts, which remove hydrogen atoms and alternate conformations. The unit cell is divided into a number of equally sized cubes, named voxels; each voxel is assigned to the atoms closest to it. The difference density maps are masked to the asymmetric unit, and a list is made of the change in the electron density in each of the voxels associated with each atom in the asymmetric unit. The most negative difference density value for the voxels associated with a given atom is listed as the D$_{loss}$ for that atom, in units of electrons per cubic {\AA}ngstrom ($e^{-}$\si{\angstrom}$^3$). 

Repeating the process of assigning the difference density peaks to atoms for the difference density maps at each dose makes it possible to plot the radiation induced density loss for each atom against dose. Atoms that are displaced by site-specific radiation damage would be expected to show negative difference density peaks at their original position, with a positive correlation between absorbed dose and D$_{loss}$.

D$_{loss}$ was plotted against diffraction weighted dose for a selection of atoms in lysozyme and \atpdx -I320 that were suspected of being susceptible to site-specific radiation damage. A straight line was fitted to the data in MATLAB. A positive gradient indicates that there is a positive correlation between dose and D$_{loss}$. The quality of the fit of the straight line to the data was determined by calculating the R-square value for each fit. An R-square value close to zero suggests that there is no correlation between dose and radiation induced density loss, while an R-square close to 1 suggests that the relationship between dose and density loss is modelled well by the straight line.      
          
 \clearpage
\section{Time-Resolved \textit{In Crystallo} UV-Vis Spectroscopy}		
\textit{In Crystallo} UV-Vis spectroscopy was performed on crystals and thin films of solvents mounted in loops. Spectra were collected offline at the ESRF Cryobench (ID29S) and online at ESRF beamline BM30; all \textit{In Crystallo} UV-Vis spectroscopy was conducted at 100 K. The spectrophotometer was aligned so that the X-ray beam passed through the focal spot of the spectrophotometer. A full description of the ESRF spectrophotometer has been provided by von Stetten \textit{et al} \cite{vonStetten2015}.    

Spectra were collected from thin films of the solvents surrounding \atpdx-I320 crystals during exposure to X-rays as well as \atpdx-I320 crystals in loops. A cryoloop was dipped into a droplet of the solution; the cryoloop was then manually mounted onto the BM30 goniometer under the cryostream at 100 K (Figure \ref{fig:BM30spec}). The loop was centred in a position ensuring that the sample was completely within the 300 \si{\micro\meter}$^2$ X-ray beam and oriented to provide the best possible spectrum. The optimal orientation of the loop was usually with the plane of the loop parallel to the direction of the X-ray beam (Figure \ref{fig:InCrystalloSchematic}). The thickness of the cryo-cooled buffer solutions exceeded the depth of the loop and was measured to be between 125 \si{\micro\meter} and 150 \si{\micro\meter} for most samples, using the on-axis camera on the beamline.   

The constituents of each buffer solution tested are listed in Table \ref{table:BufferSpec}.


\begin{table}[!ht]
 \centering
%\begin{tabular}{|p{6cm}|p{4cm}|}
\begin{tabular}{|l|c|}
 \hline
 \multicolumn{2}{|c|}{\textbf{Input Parameters for Buffer Solution Dose Calculations}} \\
 \hline
 \multicolumn{2}{|c|}{Sample Parameters}\\
 \hline
 Sample Shape & Cuboid\\
 Sample Size & 150 \si{\micro\meter} x 300 \si{\micro\meter} x 300 \si{\micro\meter}\\
 Unit cell \textit{a, b, c} ($\alpha$,$\beta$,$\gamma$)&100\si{\angstrom}, 100\si{\angstrom}, 100\si{\angstrom} (90, 90, 90)\\
 Monomers per Unit Cell    & 1\\
 Residues per Monomer&1\\
 Number of Solvent Heavy Atoms& Variable\\
 Solvent Fraction&1.0\\
 \hline
 \multicolumn{2}{|c|}{Beam Parameters}\\
 \hline
 Beam Type   & Top Hat\\
 Rectangular Collimation (H x V)  &300 \si{\micro\metre} x 300 \si{\micro\metre}\\
 Flux (photons \si{\per\second})    & Variable (Table \ref{table:BufferSpec})\\
 Incident Photon Energy (\si{\kilo\electronvolt})   &12.65 \\
 \hline
 \multicolumn{2}{|c|}{Data Collection Strategy} \\
 \hline
 Number of Images   & 1\\
 Oscillation per Image   & 0.0\degree \\
 Total Oscillation & 0.0\degree \\
 Exposure Time per Image (\si{\second}) & 1\\
 Total Exposure Time (\si{\second})& 1.0 \\
 Transmission & 100.00\% \\
 \hline 
\end{tabular}
\caption[Input parameters for calculation of the dose absorbed by the buffer solutions]{Parameters for calculation of X-ray dose absorbed by buffer solutions per second. The dimensions of the crystals and flux were also input to RADDOSE-3D, The unit cell size was arbitrarily set to 100 \si{\micro\meter}$^3$. The concentrations of elements in the buffers with a greater mass than oxygen were listed under the Number of Solvent Heavy Atoms keyword; these included the protein sulphur atoms from buffers containing protein their concentration was determined by multiplying the protein concentration by the number of heavy atoms per protein.}\label{table:RADINP_Buffer} 
\end{table}   

Non-essential lighting in the experimental hutch was turned off to minimise the amount of stray light that entered the detector optics. The sample was translated out of the light path of the spectrophotometer so that a reference spectrum could be taken; the crystal was then translated back into position and collection of spectra was initiated. The hutch was interlocked before the X-ray shutter was opened. The spectroscopy experiments were performed in two beamtimes, a flux of the X-ray beam was measured to be 3.2 x 10$^{10}$ photons\si{\per\second} at the first beamtime and 2.60 x 10$^{10}$ photons\si{\per\second} at the second beamtime when the beam current was 200 \si{\milli\ampere}. The flux was measured by Dr Antoine Royant (ESRF) at the start of each beamtime using a silicon pin diode, as described by Owen \textit{et al} \cite{Owen2009}. The beam current was noted at the beginning of each X-ray exposure so that the flux and the dose could be calculated for each sample; the flux at the sample position scales linearly with the current in the synchrotron storage ring at ESRF beamlines using bending magnets.   
%https://books.google.co.uk/books?id=9RR0AwAAQBAJ&pg=RA7-PA20&lpg=RA7-PA20&dq=is+flux+of+beamline+linearly+proportional+to+the+storage+ring+current&source=bl&ots=QFHtuQybXD&sig=SNvk9uLCh5wGg2BBq4g6qI9NQ0k&hl=en&sa=X&ved=0ahUKEwjG_8KMu47OAhUC5xoKHR0KAPMQ6AEIRDAI#v=onepage&q=is%20flux%20of%20beamline%20linearly%20proportional%20to%20the%20storage%20ring%20current&f=false
%\begin{minipage}{\linewidth}
%	\makebox[\linewidth]{
%	\includegraphics[width=6cm, height=6cm, keepaspectratio]{/Users/matt/Dropbox/ThesisPrep/Thesis/fig/spectra/Methods/BM30setup.png}}	
%
%	\captionof{figure}[Experimental Setup for Collection of Online UV-Vis Spectra]{The ESRF UV-Vis spectrophotometer was mounted on beamline BM30 (ESRF), crystals and films of various buffer soliutions were mounted in cryoloops directly on to the goniometer and under the cryostream (100 K).\label{fig:BM30spec}}	
%\end{minipage}
%


\begin{figure}[!htbp]
\centering
\begin{subfigure}{.5\textwidth}
  \centering
  \includegraphics[width=7cm, height=7cm]{/Users/matt/Dropbox/ThesisPrep/Thesis/fig/spectra/Methods/InCrystalloSpec.png}
  \caption{}
  \label{fig:InCrystalloSchematic}
\end{subfigure}%
\begin{subfigure}{.5\textwidth}
  \centering
  \includegraphics[width=7cm, height=7cm]{/Users/matt/Dropbox/ThesisPrep/Thesis/fig/spectra/Methods/BM30setup.png}
  \caption{}
  \label{fig:BM30spec}
\end{subfigure}
\caption[Experimental Setup for Collection of Online UV-Vis Spectra]{(a) Schematic of the experimental setup for simultaneous X-ray exposure and collection of UV-Vis spectra. X-ray beam size 300 \si{\micro\meter} x 300 \si{\micro\meter}. UV-Vis focal spot size $\sim$100 \si{\micro\meter} in diameter (b) The ESRF UV-Vis spectrophotometer was mounted on beamline BM30 (ESRF), crystals and films of various buffer solutions were mounted in cryoloops directly onto the goniometer and under the cryostream (100 K).}
\end{figure}


Spectra were collected with an integration time of 100 \si{\milli\second}; five spectra were averaged to reduce noise with a total acquisition time of 500 \si{\milli\second}. Spectra were collected continuously while the samples were exposed to X-rays, for a total of 15 minutes. The goniometer was not rotated during the X-ray exposure. RADDOSE-3D was used to determine the dose absorbed by the buffer solution during the entire exposure, and divided by the number of seconds that the sample was exposed to X-rays to calculate the dose rate in units of \si{\gray\per\second}.

The samples tested were thin films of distilled water, 100\% glycerol, 40\% glycerol, 100\% MPD, 40\% MPD and the glycerol-containing cryobuffer that was used to cryoprotect \atpdx ~crystals used, as well as \atpdx ~ K98A crystals and wild-type \atpdx -I320 crystals. The composition of each sample and the corresponding dose rates are listed in Appendix \ref{App:buffer_comp}.

      
\clearpage
\subsection{Processing of Time-Resolved UV-Vis Spectroscopy Data}\label{sec:SPEC_PROC}
The spectra were saved as individual text files, all spectra from a single time series were imported into MATLAB as a single three-dimensional array (Appendix \ref{App:MATLABSPEC_IMPORT}). The first column of the array contains the Wavelengths that absorbance was measured at while the second column contains the associated Absorbance values. Each page in the array contains the Wavelength and Absorbance values for a single spectrum. 

Each spectrum in the array was smoothed using a Savitzky-Golay filter, the role of the filter is to reduce the amount of high-frequency noise in the spectra \cite{Savitzky1964}. The Savitzky-Golay filter uses a similar concept to a moving average. To calculate the smoothed absorbance values for the spectrum, it takes a window with a given number of data points, fits a polynomial to the data using a Least Squares Minimisation procedure and evaluates the polynomial at the central data point in the window to provide the smoothed value \cite{Savitzky1964}. The window then shifts by one data point to a longer wavelength to calculate the smoothed value for the next absorbance value. For the time-resolved spectroscopy data, a window of 20 data points (15.18 nm) was used with a second order polynomial (Appendix \ref{App:MATLABSPEC_SGOLAY}). The smoothing procedure was implemented in MATLAB; smoothed data values are written into the 3D array in column 3 of each page, see Appendix \ref{App:MATLABSPEC_SGOLAY} for an example of a script used to smooth all of the spectra in a single time-resolved experiment.

To account for any drift in the baseline of the spectra over the course of the experiment each smoothed spectrum was normalised to set the absorbance at 900 nm, where no signal is expected, to 0. The normalised spectra were saved in column four of the array (Appendix \ref{App:MATLABSPEC_NORM}).

To observe how the spectra changed over time difference spectra were produced, the first smoothed normalised spectrum, collected before the start of the X-ray exposure was subtracted from every subsequent spectrum (Appendix \ref{App:MATLABSPEC_DIFFSSPEC}). Using difference spectra removes any features that do not change over the course of the experiment from the spectra. The difference spectra were written in column 5 of the array (Appendix \ref{App:MATLABSPEC_DIFFSSPEC}).

All of the difference spectra in a single time series were plotted on a single three-dimensional plot to allow visualisation of the evolution of the difference spectra against time/dose.  

\newpage
\subsubsection{Modelling of Peaks in UV-Vis Difference Spectra as Gaussians}\label{sec:Methods_Gaussian_Modelling}
One of the problems with UV-Vis absorbance spectroscopy is that several peaks often overlap. The observed spectrum is the product of the sum of the absorbance by each peak at each wavelength, and it can be difficult to determine the characteristics of each peak, such as the width, amplitude and $\lambda_{max}$. This is also true of the difference spectra produced in Section \ref{sec:SPEC_PROC}. To understand the rate at which each of the absorbing species that are produced when the samples are irradiated it is necessary to separate the spectrum into each of its pure components.

This is relatively simple when the composition of the sample is known, and spectra are available each of the isolated components. In this case, a function can be fitted to describe the absorbance by a known concentration of each of the isolated components; in UV-Vis spectroscopy Gaussian functions are usually used to model peaks \cite{Antonov2000}.  

\begin{equation}\label{eq:Gaussian_Equation}
f (x) = A_{max} e^{-\frac{(x-\lambda_{max})^2}{2\sigma^2}}
\end{equation} 

The Gaussian function is defined in Equation \ref{eq:Gaussian_Equation} where \textit{A$_{max}$} is the maximum height of the peak, $\lambda_{max}$ is the wavelength of maximum absorbance, and \textit{$\sigma$} is related to the Full-Width Half Maximum of the peak ($\Delta\lambda_{\frac{1}{2}}$) by the equation $ \Delta\lambda_{\frac{1}{2}} = 2 \sqrt{2\ln2}\sigma$. Once the extinction coefficient ($\varepsilon$), the wavelength of maximum absorption (\lwl) and the width of the peak at half of the maximum absorbance ($\Delta\lambda_{\frac{1}{2}}$) for each of the components is known the concentration of each component can be fitted as a variable using a Least Squares fitting procedure \cite{Gonen2009}.          

In the case of irradiation of \atpdx~ crystals and the individual buffer components the products of irradiation are unknown and it is therefore not possible to collect spectra of the isolated components of the sample. A Least Squares Minimisation algorithm has been used to fit multiple Gaussian functions to complex difference spectra. An example of a MATLAB script used to decompose a time-resolved series of difference spectra is given in Appendix \ref{App:MATLABSPEC_LSQ_GAUSS}. 

It is assumed that difference spectra have been produced and stored as a three-dimensional array of time-resolved spectra as described in Section \ref{sec:SPEC_PROC}. The user must visually inspect the spectrum to determine the number of Gaussians that should be modelled and input that as variable \textit{k} in the processing script. The script uses a least squares minimisation algorithm to place \textit{k} Gaussians on the spectrum, each with a $\lambda_{max}$, \textit{$\sigma$} and \textit{A$_{max}$}. The script can be used to model every spectrum in the time series to monitor how the size of each peak changes over the course of the experiment. Working with difference spectra simplifies the processing as fewer peak are present, for example, with spectra of \atpdx -I320 crystals the 280 nm protein peak and 320 nm intermediate absorbance peak are absent, assuming that they do not change in response to irradiation.

UV-Vis absorbance peaks are only truly symmetrical on an energy scale and should theoretically be modelled on a frequency scale, which is directly proportional to energy (Equation \ref{eq:EnergytoFrequency}). The use of a wavelength scale, which is inversely proportional to frequency, leads to asymmetric peaks \cite{Antonov1997}. Fitting of peaks using the Least Squares Minimisation method was trialled on both the wavelength and wavenumber scale and showed similar results, as the wavelength scale is conventionally used for presentation of UV-Vis spectra it was used for this analysis. 

Each time series of difference spectra was visually inspected to determine the number of absorbance peaks that formed during the experiments. While there are more sophisticated methods for determining the number of peaks that there are in a spectrum, such as using derivative spectroscopy which is more sensitive to less obvious peaks, visual inspection is the simplest method \cite{Antonov2000}. 

In cases where peaks in the spectra appeared transiently, the Gaussian that was modelling that peak often shifted to other peaks. To prevent this from having an adverse effect on the observed change in size for each of the peaks over the course of the series the $\lambda_{max}$ of these peaks was restrained.
 
By integrating each of the Gaussians for each spectrum, it is possible to plot the change in size for each of the peaks in response to irradiation.  

  
%The number of peaks to find was input to the processing script as variable \textit{k} (Appendix \ref{App:MATLAB_GMM}).      

%Each difference spectrum is converted to a one-dimensional array by multiplying the absorbance values at each wavelength by 1000 so that they are all integers and compiling a list (\textit{D1}) where each wavelength at which the absorbance was sampled is added to the list a number of times proportional to the absorbance at that wavelength. The script then places \textit{k} gaussian peaks at evenly spaced intervals across the wavelength range. At this point each of the gaussians has a \textit{$\sigma$} equal to the square root of the variance of \textit{D1} and an area equal to 1 divided by \textit{k}.    



   

%Each peak that is visible in the spectra has an absorption maximum ($\lambda_{max}$) at a wavelength that is 
%wavenumber conversion and GMM modelling 2000 paper in Endnote  